{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pymed import PubMed\n",
    "pubmed = PubMed(tool=\"PubMedSearcher\", email=\"pg49846@alunos.uminho.pt\")\n",
    "\n",
    "## PUT YOUR SEARCH TERM HERE ##\n",
    "search_term = \"TCF7L2\"\n",
    "results = pubmed.query(search_term, max_results=20)\n",
    "articleList = []\n",
    "articleInfo = []\n",
    "\n",
    "# for i in results:\n",
    "#     print(i)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "title: Genomic profiling and network-level understanding uncover the potential genes and the pathways in hepatocellular carcinoma.\n",
      "\n",
      "title: Genomic Editing and Diabetes.\n",
      "\n",
      "title: Bioinformatic analysis and clinical diagnostic value of hsa_circ_0004099 in acute ischemic stroke.\n",
      "\n",
      "title: Identification of odontogenic ameloblast associated as a novel target gene of the Wnt/β-catenin signaling pathway.\n",
      "\n",
      "title: Advances in multi-omics study of biomarkers of glycolipid metabolism disorder.\n",
      "\n",
      "title: Association between TCF7L2 polymorphism and type 2 diabetes mellitus susceptibility: a case-control study among the Bangladeshi population.\n",
      "\n",
      "title: Indolepropionic Acid, a Gut Bacteria-Produced Tryptophan Metabolite and the Risk of Type 2 Diabetes and Non-Alcoholic Fatty Liver Disease.\n",
      "\n",
      "title: Decoding Diabetes Biomarkers and Related Molecular Mechanisms by Using Machine Learning, Text Mining, and Gene Expression Analysis.\n",
      "\n",
      "title: FGF9 Promotes Expression of HAS2 in Palatal Elevation via the Wnt/β-Catenin/TCF7L2 Pathway.\n",
      "\n",
      "title: CHRNB2 represses pancreatic cancer migration and invasion via inhibiting β-catenin pathway.\n",
      "\n",
      "title: Clinical signs of type 1 diabetes are associated with type 2 diabetes marker transcription factor 7-like 2 polymorphism.\n",
      "\n",
      "title: ITGA2 Gene Polymorphism Is Associated with Type 2 Diabetes Mellitus in the Kazakhstan Population.\n",
      "\n",
      "title: Milder loss of insulin-containing islets in individuals with type 1 diabetes and type 2 diabetes-associated TCF7L2 genetic variants.\n",
      "\n",
      "title: \n",
      "\n",
      "title: Regulation of the THRA gene, encoding the thyroid hormone nuclear receptor TRα1, in intestinal lesions.\n",
      "\n",
      "title: Integrated analysis of microRNA expression profiles and function network in mice testes after low dose lead exposure from early puberty.\n",
      "\n",
      "title: Leveraging family history in genetic association analyses of binary traits.\n",
      "\n",
      "title: Genetic risk score is associated with T2DM and diabetes complications risks.\n",
      "\n",
      "title: Development and validation of a model that predicts the risk of diabetic retinopathy in type 2 diabetes mellitus patients.\n",
      "\n",
      "title: Effect of TCF7L2 on the relationship between lifestyle factors and glycemic parameters: a systematic review.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for article in results:\n",
    "# Print the type of object we've found (can be either PubMedBookArticle or PubMedArticle).\n",
    "# We need to convert it to dictionary with available function\n",
    "    articleDict = article.toDict()\n",
    "    articleList.append(articleDict)\n",
    "\n",
    "for article in articleList:\n",
    "    print(\"title:\", article['title'])\n",
    "    # print('authors:', article['authors'])\n",
    "    # print('doi:', article['doi'])\n",
    "    print()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'pubmed_id': '36479247', 'title': 'Genomic profiling and network-level understanding uncover the potential genes and the pathways in hepatocellular carcinoma.', 'keywords': ['HCV and HCC', 'biomarkers', 'co-expression', 'gene expression/mutational profiling', 'network-level understanding'], 'journal': 'Frontiers in genetics', 'abstract': 'Data integration with phenotypes such as gene expression, pathways or function, and protein-protein interactions data has proven to be a highly promising technique for improving human complex diseases, particularly cancer patient outcome prediction. Hepatocellular carcinoma is one of the most prevalent cancers, and the most common cause is chronic HBV and HCV infection, which is linked to the majority of cases, and HBV and HCV play a role in multistep carcinogenesis progression. We examined the list of known hepatocellular carcinoma biomarkers with the publicly available expression profile dataset of hepatocellular carcinoma infected with HCV from day 1 to day 10 in this study. The study covers an overexpression pattern for the selected biomarkers in clinical hepatocellular carcinoma patients, a combined investigation of these biomarkers with the gathered temporal dataset, temporal expression profiling changes, and temporal pathway enrichment following HCV infection. Following a temporal analysis, it was discovered that the early stages of HCV infection tend to be more harmful in terms of expression shifting patterns, and that there is no significant change after that, followed by a set of genes that are consistently altered. PI3K, cAMP, TGF, TNF, Rap1, NF-kB, Apoptosis, Longevity regulating pathway, signaling pathways regulating pluripotency of stem cells, Cytokine-cytokine receptor interaction, p53 signaling, Wnt signaling, Toll-like receptor signaling, and Hippo signaling pathways are just a few of the most commonly enriched pathways. The majority of these pathways are well-known for their roles in the immune system, infection and inflammation, and human illnesses like cancer. We also find that ADCY8, MYC, PTK2, CTNNB1, TP53, RB1, PRKCA, TCF7L2, PAK1, ITPR2, CYP3A4, UGT1A6, GCK, and FGFR2/3 appear to be among the prominent genes based on the networks of genes and pathways based on the copy number alterations, mutations, and structural variants study.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': 'Copyright © 2022 El-Kafrawy, El-Daly, Bajrai, Alandijany, Faizo, Mobashir, Ahmed, Ahmed, Alam, Jeet, Kamal, Anwer, Khan, Tashkandi, Rizvi and Azhar.', 'doi': '10.3389/fgene.2022.880440\\n10.1016/j.ajpath.2012.08.005\\n10.1186/1479-5876-10-41\\n10.3389/fonc.2022.914032\\n10.1101/gr.087528.108\\n10.3390/ijms231911024\\n10.1016/j.dnarep.2019.06.005\\n10.1038/s41598-021-98031-7\\n10.3389/fcimb.2021.707905\\n10.1038/nrg1272\\n10.1093/nar/gkp942\\n10.1074/jbc.M010271200\\n10.1093/sysbio/syu081\\n10.1038/nrg3424\\n10.1245/s10434-013-2985-7\\n10.1038/nature07385\\n10.1098/rsif.2011.0390\\n10.1038/nbt.2203\\n10.1038/nature08987\\n10.1038/ng.3221\\n10.1038/msb4100200\\n10.1093/annonc/mdt419\\n10.1056/NEJMra1001683\\n10.3389/fendo.2021.641888\\n10.1038/nature06758\\n10.1002/wsbm.134\\n10.1093/nar/29.2.449\\n10.1038/nrc1299\\n10.1038/nrg3813\\n10.1038/nmeth.2562\\n10.1182/blood-2008-05-158865\\n10.21873/anticanres.11078\\n10.1038/nrg.2016.87\\n10.3390/biomedicines10051076\\n10.1016/j.dnarep.2007.11.007\\n10.1002/ijc.22605\\n10.1186/s13059-014-0436-9\\n10.1146/annurev-immunol-032712-100001\\n10.36013/jimsa.v1i1.3\\n10.1093/nar/gkm882\\n10.1093/nar/gkp896\\n10.1038/s41598-022-11143-6\\n10.1038/s41598-022-11143-6\\n10.1371/journal.pone.0062113\\n10.3389/fonc.2012.00202\\n10.1038/nature11412\\n10.1016/j.imu.2020.100422\\n10.1016/j.imu.2020.100422\\n10.1073/pnas.0304146101\\n10.1186/s13059-014-0473-4\\n10.1038/ng1956\\n10.1038/nrclinonc.2015.103\\n10.1053/j.gastro.2019.04.003\\n10.1053/j.gastro.2019.04.003\\n10.1093/nar/gkv1194\\n10.1038/nbt.3027\\n10.1016/j.ajpath.2013.02.026\\n10.1126/scitranslmed.aaa0763\\n10.1073/pnas.0609635104\\n10.1186/s13059-014-0439-6\\n10.3390/diseases3040294\\n10.1093/hmg/ddq079\\n10.1093/bioinformatics/bts448\\n10.1038/nrc3816\\n10.1146/annurev.med.45.1.297\\n10.1016/j.diabres.2018.03.002\\n10.1038/nrclinonc.2014.5\\n10.1101/gr.1239303\\n10.1186/1475-2840-11-15\\n10.1073/pnas.0506580102\\n10.1093/annonc/mdu005\\n10.1016/j.ccr.2010.05.026\\n10.1146/annurev.biophys.33.110502.132654\\n10.1074/mcp.M500279-MCP200\\n10.1126/science.aax9198\\n10.1126/science.aan2507\\n10.1146/annurev-pharmtox-010611-134628\\n10.1016/j.ccr.2005.10.001\\n10.2174/1381612826666201106093148\\n10.1038/nrclinonc.2014.6\\n10.1126/science.aaw0726\\n10.1038/nrg3871\\n10.1186/s40246-015-0056-9', 'publication_date': datetime.date(2022, 12, 9), 'authors': [{'lastname': 'El-Kafrawy', 'firstname': 'Sherif A', 'initials': 'SA', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'El-Daly', 'firstname': 'Mai M', 'initials': 'MM', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Bajrai', 'firstname': 'Leena H', 'initials': 'LH', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nBiochemistry Department, Faculty of Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Alandijany', 'firstname': 'Thamir A', 'initials': 'TA', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Faizo', 'firstname': 'Arwa A', 'initials': 'AA', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Mobashir', 'firstname': 'Mohammad', 'initials': 'M', 'affiliation': 'Department of Microbiology, Tumor and Cell Biology (MTC), Karolinska Institute, Stockholm, Sweden.\\nGenome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Ahmed', 'firstname': 'Sunbul S', 'initials': 'SS', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Ahmed', 'firstname': 'Sarfraz', 'initials': 'S', 'affiliation': 'Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Alam', 'firstname': 'Shoaib', 'initials': 'S', 'affiliation': 'Department of Biotechnology, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Jeet', 'firstname': 'Raja', 'initials': 'R', 'affiliation': 'Botany Department, Ganesh Dutt College, Begusarai, Bihar, India.'}, {'lastname': 'Kamal', 'firstname': 'Mohammad Amjad', 'initials': 'MA', 'affiliation': 'Institutes for Systems Genetics, Frontiers Science Center for Disease-related Molecular Network, West China Hospital, Sichuan University, Chengdu, China.\\nKing Fahd Medical Research Center, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Pharmacy, Faculty of Allied Health Sciences, Daffodil International University, Dhaka, Bangladesh.\\nEnzymoics, Hebersham, NSW, Australia.\\nNovel Global Community Educational Foundation, Hebersham, NSW, Australia.'}, {'lastname': 'Anwer', 'firstname': 'Syed Tauqeer', 'initials': 'ST', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Khan', 'firstname': 'Bushra', 'initials': 'B', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Tashkandi', 'firstname': 'Manal', 'initials': 'M', 'affiliation': 'Department of Biochemistry, College of Science, University of Jeddah, Jeddah, Saudi Arabia.'}, {'lastname': 'Rizvi', 'firstname': 'Moshahid A', 'initials': 'MA', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Azhar', 'firstname': 'Esam Ibraheem', 'initials': 'EI', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}]}\n",
      "\n",
      "{'pubmed_id': '36454469', 'title': 'Genomic Editing and Diabetes.', 'keywords': ['CRISPR/Cas9', 'Diabetes', 'Genetic editing', 'Long noncoding RNA', 'MicroRNA'], 'journal': 'Advances in experimental medicine and biology', 'abstract': 'The diabetes types and its complications have varying developmental and metabolic pathways. There is an interplay of nongenetic and genetic components in pathogenesis of diabetes and its complications. There are several established genes such as ABCC8, TCF7L2, SLC2A2, and CAPN10 which are known to influence blood insulin and glucose levels. Current management of diabetes types may include lifetime burdensome use of insulin, insulin sensitizers, insulin secretagogues, etc. There has been increasing interest in improving genetic editing tools such as CRISPR/Cas9 and using genetically edited stem cells to alter diabetes disease course or possibly cure it. Current research on microRNAs and long noncoding RNAs may provide insights into the pathways involved in development of diabetes and its complications. Consequently, developing further understanding of genetics and its messenger pathways in diabetes would enhance our ability to develop precise and accurate genetic editing tools which can translate into clinically useful therapeutics.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2023. The Author(s), under exclusive license to Springer Nature Singapore Pte Ltd.', 'doi': '10.1007/978-981-19-5642-3_14\\n10.1016/S0140-6736(16)00618-8\\n10.34172/hpp.2020.18\\n10.1016/j.ymgme.2003.10.005\\n10.2337/db17-0318\\n10.1016/j.ijbiomac.2020.04.240\\n10.1186/s12881-018-0553-5\\n10.1055/a-1008-9223\\n10.1007/s42000-018-0047-z\\n10.1016/j.molmet.2019.03.003\\n10.1172/jci.insight.136136\\n10.1002/humu.23995\\n10.1007/978-3-030-25905-1_18\\n10.1111/1753-0407.13144\\n10.1007/s00125-018-4759-z\\n10.1038/ng.3632\\n10.1530/JME-19-0253\\n10.1155/2017/3937893\\n10.1007/s00404-016-4140-8\\n10.1007/5584_2018_305\\n10.1080/21655979.2016.1189039\\n10.1371/journal.pbio.3000603\\n10.1016/j.bbrc.2019.09.001\\n10.1186/s12933-019-0824-2\\n10.1530/EJE-19-0206\\n10.1161/CIRCRESAHA.119.314898\\n10.3389/fimmu.2018.02711\\n10.1080/09674845.2017.1402404\\n10.1038/s41467-019-13587-3\\n10.1007/s11033-018-4429-8\\n10.1073/pnas.1914353117\\n10.3390/ijms21114020\\n10.1016/j.atherosclerosis.2018.08.031\\n10.7150/ijms.34200', 'publication_date': datetime.date(2022, 12, 2), 'authors': [{'lastname': 'Shah', 'firstname': 'Parth', 'initials': 'P', 'affiliation': 'ObvioHealth, New York, NY, USA. prshah06@gmail.com.'}]}\n",
      "\n",
      "{'pubmed_id': '36399471', 'title': 'Bioinformatic analysis and clinical diagnostic value of hsa_circ_0004099 in acute ischemic stroke.', 'keywords': [], 'journal': 'PloS one', 'abstract': 'This study investigates the expression and effect of hsa_circ_0004099 in acute ischemic stroke (AIS). We conducted a case-controlled study that included 40 patients with AIS within 24 hours and 40 healthy subjects during the same period as a control group. Differentially expressed circular RNAs (circRNAs) were obtained using GEO2R, and the expression of hsa_circ_0004099 was verified using RT-PCR. Correlation analysis of the National Institutes of Health Stroke Scale (NIHSS) disease severity score and ischemic time with hsa_circ_0004099 expression levels was also performed. The receiver operating characteristic (ROC) curve of hsa_circ_0004099 was constructed, and bioinformatic analysis of hsa_circ_0004099 was performed. NIHSS scores negatively correlated with hsa_circ_0004099 levels (P<0.001, r = -0.7053), whereas infarct time was negatively correlated with hsa_circ_0004099 levels (P<0.001, r = -0.5130); hsa_circ_0004099 could benefit clinical diagnosis (area under the curve [AUC]: 0.923 [95% confidence interval [CI]: 0.8680-0.9904]). Kyoto encyclopedia of genes and genomes (KEGG) analysis showed that hsa_circ_0004099 was enriched in several cancer pathways, which were collectively enriched in four genes namely TCF7L2, NRAS, CTNNB1, and KRAS. Eight core proteins were screened using a protein-protein interaction (PPI) network namely SMAD4, HIF1A, CTNNB1, CDKN1B, CDK6, FOXO3, KRAS, and NRAS. hsa_circ_0004099 is a potential clinical diagnostic marker. In addition, the possible role of hsa_circ_0004099 in the pathogenesis of AIS was analyzed using bioinformatics, which provided a new potential molecular target for AIS treatment.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': 'Copyright: © 2022 Zheng et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.', 'doi': '10.1371/journal.pone.0277832\\n10.1016/S0140-6736(18)31874-9\\n10.1001/jama.2015.3058\\n10.29271/jcpsp.2019.09.898\\n10.6061/clinics/2021/e2958\\n10.1002/wrna.1294\\n10.2174/1567205017666201111114928\\n10.1007/s00018-017-2688-5\\n10.1016/j.gene.2019.03.053\\n10.18632/aging.103424\\n10.1002/jcp.29165\\n10.1038/s41598-020-66990-y\\n10.3389/fnins.2020.00081\\n10.1016/j.ajhg.2016.10.006\\n10.1016/j.atherosclerosis.2018.01.028\\n10.1161/JAHA.114.001119\\n10.1371/journal.pone.0141214\\n10.4037/aacnacc2018483\\n10.1002/jcla.22954\\n10.7754/Clin.Lab.2020.191266\\n10.1161/STROKEAHA.119.027348\\n10.1016/j.brainres.2020.146835\\n10.3892/mmr.2019.10317\\n10.1007/s12975-020-00851-8\\n10.1007/s10528-022-10225-4\\n10.1155/2021/5585206\\n10.1093/jnen/nlab106\\n10.1016/j.gene.2020.145022\\n10.1159/000514543\\n10.1590/1806-9282.20210509\\n10.1016/j.diabres.2013.12.051\\n10.1007/s00432-021-03842-w\\n10.1002/ana.26059\\n10.1155/2021/6745282\\n10.1002/ana.23883\\n10.1186/s12885-022-09186-z\\n10.1002/acn3.51154\\n10.1007/s11010-022-04428-x\\n10.3389/fnagi.2022.830494\\n10.1155/2020/2507074\\n10.17116/jnevro201611612210-13\\n10.3389/fonc.2021.757650\\n10.1161/STROKEAHA.111.000504\\n10.1016/j.jns.2015.05.036\\n10.3390/metabo12030250\\n10.1093/gerona/glac016\\n10.3390/ijms222011131\\n10.1080/00207454.2018.1564290\\n10.1016/j.gep.2021.119218', 'publication_date': datetime.date(2022, 11, 19), 'authors': [{'lastname': 'Zheng', 'firstname': 'Jiqing', 'initials': 'J', 'affiliation': 'Deparment of Rehabilitation, The Second Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi, China.'}, {'lastname': 'Luo', 'firstname': 'Shuiming', 'initials': 'S', 'affiliation': 'Deparment of Rehabilitation, The Second Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi, China.'}, {'lastname': 'Long', 'firstname': 'Yaobin', 'initials': 'Y', 'affiliation': 'Deparment of Rehabilitation, The Second Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi, China.'}]}\n",
      "\n",
      "{'pubmed_id': '36382598', 'title': 'Identification of odontogenic ameloblast associated as a novel target gene of the Wnt/β-catenin signaling pathway.', 'keywords': ['ODAM', 'Wnt signaling', 'cell cycle', 'colorectal cancer', 'hepatoblastoma'], 'journal': 'Cancer science', 'abstract': 'The Wnt/β-catenin signaling pathway plays a key role in development and carcinogenesis. Although some target genes of this signaling have been identified in various tissues and neoplasms, the comprehensive understanding of the target genes and their roles in the development of human cancer, including hepatoma and colorectal cancer remain to be fully elucidated. In this study, we searched for genes regulated by the Wnt signaling in liver cancer using HuH-7 hepatoma cells. A comparison of the expression profiles between cells expressing an active form of mutant β-catenin and cells expressing enhanced green fluorescent protein (EGFP) identified seven genes upregulated by the mutant β-catenin gene (CTNNB1). Among the seven genes, we focused in this study on ODAM, odontogenic,\\xa0ameloblast associated, as a novel target gene. Interestingly, its expression was frequently upregulated in hepatocellular carcinoma, colorectal adenocarcinoma, and hepatoblastoma. We additionally identified a distant enhancer region that was associated with the β-catenin/TCF7L2 complex. Further analyses revealed that ODAM plays an important role in the regulation of the cell cycle, DNA synthesis, and cell proliferation. These data may be useful for clarification of the main molecular mechanism(s) underlying these cancers.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2022 The Authors. Cancer Science published by John Wiley & Sons Australia, Ltd on behalf of Japanese Cancer Association.', 'doi': '10.1111/cas.15657', 'publication_date': datetime.date(2022, 11, 17), 'authors': [{'lastname': 'Yamaguchi', 'firstname': 'Kiyoshi', 'initials': 'K', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Horie', 'firstname': 'Chiaki', 'initials': 'C', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Takane', 'firstname': 'Kiyoko', 'initials': 'K', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Ikenoue', 'firstname': 'Tsuneo', 'initials': 'T', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Nakagawa', 'firstname': 'Saya', 'initials': 'S', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Isobe', 'firstname': 'Yumiko', 'initials': 'Y', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Ota', 'firstname': 'Yasunori', 'initials': 'Y', 'affiliation': 'Department of Pathology, Research Hospital, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Ushiku', 'firstname': 'Tetsuo', 'initials': 'T', 'affiliation': 'Department of Pathology, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Tanaka', 'firstname': 'Mariko', 'initials': 'M', 'affiliation': 'Department of Pathology, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Fujishiro', 'firstname': 'Jun', 'initials': 'J', 'affiliation': 'Department of Pediatric Surgery, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Hoshino', 'firstname': 'Noriko', 'initials': 'N', 'affiliation': 'Department of Pediatric Surgery, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Arisue', 'firstname': 'Atsuhiro', 'initials': 'A', 'affiliation': 'Department of Surgery, Iwate Medical University School of Medicine, Yahaba, Japan.'}, {'lastname': 'Nishizuka', 'firstname': 'Satoshi', 'initials': 'S', 'affiliation': 'Division of Biomedical Research and Development, Iwate Medical University Institute for Biomedical Sciences, Yahaba, Japan.'}, {'lastname': 'Aikou', 'firstname': 'Susumu', 'initials': 'S', 'affiliation': 'Division of Frontier Surgery, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Shida', 'firstname': 'Dai', 'initials': 'D', 'affiliation': 'Division of Frontier Surgery, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Furukawa', 'firstname': 'Yoichi', 'initials': 'Y', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}]}\n",
      "\n",
      "{'pubmed_id': '36382190', 'title': 'Advances in multi-omics study of biomarkers of glycolipid metabolism disorder.', 'keywords': ['1,5-AG, 1,5-anhydroglucitol', '2-AAA, 2-aminoadipic acid', '2-DGE, two-dimensional gel electrophoresis', 'ABCC8, ATP binding cassette subfamily C member 8', 'ADA, American Diabetes Association', 'AhR, aromatic hydrocarbon receptor', 'BA, bile acid', 'BCAA, branched-chain amino acid', 'BCKAs, branched chain keto acids', 'Biomarkers', 'CE, cholesterol ester', 'CFL1, cofilin-1', 'CRP, C reactive protein', 'CYP450, cytochrome P450', 'DAG, diacylglycerol', 'DPP-4, dipeptyl peptidase 4', 'EASD, European Association for the Study of Diabetes', 'FA, fatty acid', 'FFA, free fatty acid', 'FMT, fecal microbiota transplantation', 'FTO, fat mass and obesity-associated', 'GAS5, growth arrest-specific transcript 5', 'GC–MS/MS, gas chromatography-tandem mass spectrometry', 'GLP-1, glucagon-like peptide 1', 'GLP-1R, glucagon-like peptide 1 receptor', 'GSIS, glucose-stimulated insulin secretion', 'GWAS, genome-wide association study', 'Glycolipid metabolism disorder', 'HGF, hepatocyte growth factor', 'HMG-CoA, hydroxymethylglutaryl-coenzyme A', 'HMGCR, 3-hydroxy-methylglutaryl coenzyme A reductase', 'HPLC, high performance liquid chromatography', 'HbA1C, glycylated hemoglobin', 'IL-1ra, interleukin-1 receptor antagonist', 'IMP, imidazole propionate', 'IR, insulin resistance', 'JNK, c-Jun-N-terminal-kinase', 'KCNJ11, potassium inwardly rectifying channel subfamily J member 11', 'L-GPC, linoleoyl-glycerophosphocholine', 'LC-MS/MS, liquid chromatography-tandem mass spectrometry', 'LPC, lysophosphatidylcholine', 'LPE, lysophosphatidylethanolamines', 'LPS, lipopolysaccharide', 'MAPK, mitogen-activated protein kinase', 'MASP, mannose-binding lectin-associated serine protease 1', 'MATE1, multidrug and toxin extrusion protein 1', 'MDR1, multidrug resistance mutation 1', 'MGT, magnesium transporter', 'MS, Mass Spectrometry', 'Mechanism study', 'Multi-omics', 'NK, natural killer', 'NMR, nuclear magnetic resonance', 'OCT, organic cationic transporter', 'OGTT, oral glucose tolerance test', 'PC-PL, phosphatidylcholine-plasmalogen', 'PE, phosphatidylethanolamines', 'PNPLA3, patatin-like phospholipase domain-containing protein 3', 'PPAR, peroxisome proliferator-activated receptor', 'PTBP1, polypyrimidine tract-binding protein 1', 'PTP, protein tyrosine phosphatase', 'RNS, reactive nitrogen species', 'ROS, reactive oxygen species', 'SCFA, short-chain fatty acid', 'SLC30A8, solute carrier family 30 member 8', 'SLC47A1, solute carrier family 47 member 1', 'SLC5A2, solute carrier family 5 member 2', 'SM, sphingomyelin', 'SNP, single-nucleotide polymorphism', 'SSPG, steady-state plasma glucose', 'SUR1, sulfonylurea receptor 1', 'T2DM, type 2 diabetes mellitus', 'TAG, triacylglycerol', 'TCA, tricarboxylic acid', 'TCF7L2, transcription factor 7-like 2', 'TF, transcription factor', 'TMA, trimethylamine', 'TMAO, trimethylamine oxide', 'TNF-α, tumor necrosis factor alpha', 'TOF-MS, time-of-flight mass spectrum', 'TUG1, taurine upregulated gene 1', 'USP20, ubiquitin-specificpeptidase 20', 'WBC, white blood cell', 'circRNA, circular RNA', 'hADCS, human adipose-derived stem cells', 'hsCRP, high-sensitivity C-reactive protein', 'lnc-BATE1, brown adipose tissue enriched long non-coding RNA 1', 'lncRNA SHGL, lncRNA suppressor of hepatic gluconeogenesis and lipogenesis', 'lncRNA, long non-coding RNA', 'mRNA, messenger RNAs', 'mTORC1, mechanistic target of rapamycin complex 1', 'miRNA, micro RNA', 'ncRNA, non-coding RNA', 't-PA, tissue plasminogen activator', 'α-HB, α-hydroxybutyrate'], 'journal': 'Computational and structural biotechnology journal', 'abstract': 'Glycolipid metabolism disorder are major threats to human health and life. Genetic, environmental, psychological, cellular, and molecular factors contribute to their pathogenesis. Several studies demonstrated that neuroendocrine axis dysfunction, insulin resistance, oxidative stress, chronic inflammatory response, and gut microbiota dysbiosis are core pathological links associated with it. However, the underlying molecular mechanisms and therapeutic targets of glycolipid metabolism disorder remain to be elucidated. Progress in high-throughput technologies has helped clarify the pathophysiology of glycolipid metabolism disorder. In the present review, we explored the ways and means by which genomics, transcriptomics, proteomics, metabolomics, and gut microbiomics could help identify novel candidate biomarkers for the clinical management of glycolipid metabolism disorder. We also discuss the limitations and recommended future research directions of multi-omics studies on these diseases.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2022 The Authors. Published by Elsevier B.V. on behalf of Research Network of Computational and Structural Biotechnology.', 'doi': '10.1016/j.csbj.2022.10.030\\n10.1038/nrendo.2017.151\\n10.3961/jpmph.19.100\\n10.1038/s41586-020-2338-1\\n10.1155/2019/2958920\\n10.1016/j.jff.2021.104556\\n10.3390/nu12123652\\n10.1007/s00125-019-4938-6\\n10.1016/j.appet.2018.01.015\\n10.1016/j.molmet.2018.12.006\\n10.2337/db11-0785\\n10.1016/j.cmet.2013.12.015\\n10.1016/j.cmet.2015.04.008\\n10.1016/j.metabol.2021.154892\\n10.1038/s41580-021-00390-6\\n10.1038/s41586-019-1797-8\\n10.1016/j.redox.2021.101887\\n10.1038/nrendo.2015.189\\n10.1016/j.celrep.2021.109161\\n10.1007/s13238-020-00814-7\\n10.3390/microorganisms8040527\\n10.1016/j.arr.2021.101346\\n10.1038/s41576-020-0216-1\\n10.1038/nature19806\\n10.1074/jbc.AAC118.006541\\n10.1038/ng1732\\n10.1186/s12881-018-0553-5\\n10.1016/j.numecd.2017.10.012\\n10.1016/j.diabet.2016.05.003\\n10.1186/s12881-018-0614-9\\n10.2337/diabetes.47.3.490\\n10.3390/ijms19072124\\n10.1038/nature05616\\n10.1126/science.1142382\\n10.2337/db08-0047\\n10.2337/db08-0877\\n10.1126/science.1151710\\n10.2147/DMSO.S154978\\n10.1016/s0014-5793(97)00488-2\\n10.1126/science.270.5239.1166\\n10.1007/s00125-013-3031-9\\n10.2337/diab.45.6.825\\n10.1038/s41467-018-04951-w\\n10.1038/nature18642\\n10.1038/s41586-020-2263-3\\n10.1038/nature14177\\n10.1210/er.2013-1084\\n10.1016/j.cmet.2019.03.009\\n10.1016/j.diabres.2015.05.003\\n10.1007/s40291-014-0126-z\\n10.1016/j.diabres.2006.10.021\\n10.2337/dc07-2248\\n10.1016/j.ejim.2011.10.018\\n10.1155/2015/176949\\n10.1210/en.2011-0286\\n10.1007/s40262-015-0307-x\\n10.1097/FPC.0000000000000268\\n10.1007/s00125-018-4675-2\\n10.1016/j.amjcard.2003.08.078\\n10.1001/jama.291.23.2821\\n10.1016/s0021-9150(03)00094-7\\n10.1016/s0021-9150(01)00410-5\\n10.1038/nsmb.2921\\n10.1186/s40246-018-0173-3\\n10.1016/j.cmet.2012.08.010\\n10.1159/000373999\\n10.1016/j.ijcard.2016.11.182\\n10.1159/000484191\\n10.2337/db17-0799\\n10.1073/pnas.0804549105\\n10.2337/dbi19-0016\\n10.1038/nature03076\\n10.1007/s00109-015-1296-9\\n10.1038/nm.3862\\n10.2337/db17-0816\\n10.1038/s41574-019-0260-0\\n10.1038/s41467-020-15529-w\\n10.3390/cells9030659\\n10.1038/nature11928\\n10.1038/srep12453\\n10.1007/s00592-016-0943-0\\n10.3892/ijmm.2018.3625\\n10.3390/cells9061473\\n10.1038/s42255-019-0078-z\\n10.1016/j.yexcr.2019.111753\\n10.2152/jmi.54.289\\n10.1016/j.jpba.2009.11.016\\n10.1016/j.atherosclerosis.2013.02.035\\n10.2337/db15-0881\\n10.1007/s10654-018-0475-8\\n10.1016/j.mce.2014.11.021\\n10.5483/bmbrep.2012.45.12.235\\n10.1016/j.jprot.2013.10.005\\n10.3389/fphar.2021.735410\\n10.3390/cells10112832\\n10.2337/dc15-2251\\n10.1111/1753-0407.12645\\n10.3803/EnM.2019.34.3.234\\n10.1007/s00125-017-4222-6\\n10.1074/jbc.M106703200\\n10.1210/jc.2014-2357\\n10.2337/dc17-1642\\n10.1172/JCI64801\\n10.1056/NEJMc1412427\\n10.2337/dc13-0800\\n10.1007/s00125-016-4069-2\\n10.2337/dc18-0840\\n10.1373/clinchem.2014.228965\\n10.1021/pr300523x\\n10.1016/j.dsx.2018.09.009\\n10.1016/j.arcmed.2017.11.003\\n10.1136/gutjnl-2020-323071\\n10.1038/s41574-019-0156-z\\n10.1038/sj.emboj.7601049\\n10.3389/fphar.2018.01382\\n10.3945/ajcn.117.157107\\n10.1111/obr.12843\\n10.1038/s41575-019-0258-z\\n10.1016/j.cmet.2018.07.001\\n10.1016/j.cell.2018.09.055\\n10.1007/s12020-012-9760-6\\n10.1007/s00125-016-4041-1\\n10.2337/db12-0707\\n10.1038/1881110a0\\n10.3390/ijms22136862\\n10.1007/s11306-019-1484-7\\n10.3389/fnut.2019.00021\\n10.3748/wjg.v21.i29.8787\\n10.1017/S0007114516001045\\n10.5582/ddt.2018.01036\\n10.1038/tpj.2012.43\\n10.1038/nature12198\\n10.1016/j.diabet.2016.04.004\\n10.1186/s12876-015-0330-2\\n10.1136/gutjnl-2014-308778\\n10.1038/nature05414\\n10.1038/nm.4358\\n10.1016/j.chom.2019.07.004\\n10.1038/s41467-019-10189-x\\n10.1007/s00018-019-03434-5\\n10.1038/s41586-020-2928-y\\n10.1016/j.jprot.2020.103823\\n10.1038/nature18646\\n10.1038/s41586-019-1236-x\\n10.1038/s41591-019-0414-6\\n10.1038/nrd.2016.32\\n10.3389/fphys.2011.00094\\n10.1111/jdi.12673\\n10.1016/j.nut.2011.08.013', 'publication_date': datetime.date(2022, 11, 17), 'authors': [{'lastname': 'Fang', 'firstname': 'Xinyi', 'initials': 'X', 'affiliation': \"Institute of Metabolic Diseases, Guang'anmen Hospital, China Academy of Chinese Medical Sciences, Beijing 100053, China.\\nGraduate College, Beijing University of Chinese Medicine, Beijing 100029, China.\"}, {'lastname': 'Miao', 'firstname': 'Runyu', 'initials': 'R', 'affiliation': \"Institute of Metabolic Diseases, Guang'anmen Hospital, China Academy of Chinese Medical Sciences, Beijing 100053, China.\\nGraduate College, Beijing University of Chinese Medicine, Beijing 100029, China.\"}, {'lastname': 'Wei', 'firstname': 'Jiahua', 'initials': 'J', 'affiliation': 'Changchun University of Chinese Medicine, Jilin 130117, China.'}, {'lastname': 'Wu', 'firstname': 'Haoran', 'initials': 'H', 'affiliation': 'Graduate College, Beijing University of Chinese Medicine, Beijing 100029, China.'}, {'lastname': 'Tian', 'firstname': 'Jiaxing', 'initials': 'J', 'affiliation': \"Institute of Metabolic Diseases, Guang'anmen Hospital, China Academy of Chinese Medical Sciences, Beijing 100053, China.\"}]}\n",
      "\n",
      "{'pubmed_id': '36369331', 'title': 'Association between TCF7L2 polymorphism and type 2 diabetes mellitus susceptibility: a case-control study among the Bangladeshi population.', 'keywords': ['Bangladesh', 'Chittagong', 'Comorbidities', 'Diabetes', 'Polymorphism', 'TCF7L2'], 'journal': 'Molecular biology reports', 'abstract': 'Diabetes is a severe health burden for Bangladesh. Genetic polymorphism has been reported to be one of the major risk factors for diabetes in various studies. TCF7L2 (transcription factor 7 like 2) transcripts in the human β-cell have effects on β-cell survival, function, and Wnt signaling activation. This study aimed to evaluate the frequency and association of various polymorphisms namely TCF7L2 rs12255372 and rs7903146 among Bangladeshi patients with T2DM (Type 2 Diabetes Mellitus).\\nThis case-control study included 300 patients with T2DM and 234 healthy individuals from two health facilities in the Chattogram Division of Bangladesh. Anthropometric measurements were assessed using a self-reported, structured, eight-item questionnaire. The polymorphisms were identified by PCR-RFLP and sequencing method.\\nA strong association of T2DM\\xa0with polymorphisms was observed, including rs12255372 (p\\u2009=\\u20090.0004) and rs7903146 (p\\u2009=\\u20090.005). It was observed that the risk genotype at rs12255372 was associated with age (p\\u2009=\\u20090.009), a family history of diabetes (p\\u2009<\\u20090.0001), and HbA1C (p\\u2009<\\u20090.0001). Furthermore, it was found that rs12255372 was substantially associated with hypertension (p\\u2009=\\u20090.03), eye problems (p\\u2009=\\u20090.01), and neurological abnormalities (p\\u2009=\\u20090.02).\\nThis study postulates that TCF7L2 genetic polymorphism is associated with the risk of T2DM among the\\xa0studied Bangladeshi population. The findings should be replicated through more studies with a large number of samples and in different populations.', 'conclusions': 'This study postulates that TCF7L2 genetic polymorphism is associated with the risk of T2DM among the\\xa0studied Bangladeshi population. The findings should be replicated through more studies with a large number of samples and in different populations.', 'methods': None, 'results': 'A strong association of T2DM\\xa0with polymorphisms was observed, including rs12255372 (p\\u2009=\\u20090.0004) and rs7903146 (p\\u2009=\\u20090.005). It was observed that the risk genotype at rs12255372 was associated with age (p\\u2009=\\u20090.009), a family history of diabetes (p\\u2009<\\u20090.0001), and HbA1C (p\\u2009<\\u20090.0001). Furthermore, it was found that rs12255372 was substantially associated with hypertension (p\\u2009=\\u20090.03), eye problems (p\\u2009=\\u20090.01), and neurological abnormalities (p\\u2009=\\u20090.02).', 'copyrights': '© 2022. The Author(s), under exclusive licence to Springer Nature B.V.', 'doi': '10.1007/s11033-022-08081-x\\n10.1016/j.metabol.2019.153952\\n10.1038/nrdp.2015.19\\n10.1159/000356747\\n10.1007/s00125-006-0506-y\\n10.3390/ijms21176275\\n10.2471/BLT.13.128371\\n10.1038/nature05616\\n10.4161/isl.23345\\n10.1172/JCI30706\\n10.1007/s00109-007-0203-4\\n10.1007/s00125-006-0502-2\\n10.2337/db09-1386\\n10.2337/db06-0341\\n10.2337/db06-0643\\n10.1186/1471-2350-9-106\\n10.1007/s00125-007-0618-z\\n10.1038/sj.ejhg.5201773\\n10.1016/j.banm.2021.09.001\\n10.1590/S1415-47572010005000056\\n10.1590/S1415-47572012005000029\\n10.1046/j.1365-2125.2001.00346.x\\n10.4103/2321-0656.193997\\n10.4093/dmj.2015.39.5.395\\n10.7754/clin.lab.2013.130809\\n10.1016/j.metabol.2007.04.012\\n10.1016/j.jgeb.2016.10.004\\n10.1038/ng1960\\n10.2337/db07-0012\\n10.1097/MD.0000000000014288\\n10.1186/1471-2350-9-72\\n10.1186/1471-2350-10-15\\n10.2337/db08-0906\\n10.1086/520599\\n10.1038/ng2043\\n10.1126/science.1142364\\n10.1186/s40200-015-0148-z\\n10.4093/dmj.2015.39.6.512\\n10.2337/db07-0421\\n10.1016/j.gene.2012.09.034\\n10.1186/s13104-021-05598-5\\n10.1186/1472-6823-10-9\\n10.1331/JAPhA.2009.09077\\n10.1016/j.diabres.2010.01.024\\n10.1155/2013/409083\\n10.1016/j.phrs.2015.05.013\\n10.3305/nh.2013.28.sup2.6714\\n10.1111/j.1469-1809.2008.00443.x\\n10.1371/journal.pone.0188382\\n10.1590/0004-2730000003510', 'publication_date': datetime.date(2022, 11, 13), 'authors': [{'lastname': 'Salauddin', 'firstname': 'Asma', 'initials': 'A', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Chakma', 'firstname': 'Kallyan', 'initials': 'K', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Hasan', 'firstname': 'Md Mahbub', 'initials': 'MM', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Akter', 'firstname': 'Farhana', 'initials': 'F', 'affiliation': 'Department of Endocrinology, Chittagong Medical College, Chattogram, 4203, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Chowdhury', 'firstname': 'Nowshad Asgar', 'initials': 'NA', 'affiliation': 'Chittagong Diabetic General Hospital, Chattogram, Bangladesh.'}, {'lastname': 'Chowdhury', 'firstname': 'Sumon Rahman', 'initials': 'SR', 'affiliation': 'Chittagong Diabetic General Hospital, Chattogram, Bangladesh.'}, {'lastname': 'Mannan', 'firstname': 'Adnan', 'initials': 'A', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh. adnan.mannan@cu.ac.bd.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh. adnan.mannan@cu.ac.bd.'}]}\n",
      "\n",
      "{'pubmed_id': '36364957', 'title': 'Indolepropionic Acid, a Gut Bacteria-Produced Tryptophan Metabolite and the Risk of Type 2 Diabetes and Non-Alcoholic Fatty Liver Disease.', 'keywords': ['NAFLD', 'diet', 'gut microbiota', 'indolepropionic acid', 'insulin', 'type 2 diabetes'], 'journal': 'Nutrients', 'abstract': 'An intricate relationship between gut microbiota, diet, and the human body has recently been extensively investigated. Gut microbiota and gut-derived metabolites, especially, tryptophan derivatives, modulate metabolic and immune functions in health and disease. One of the tryptophan derivatives, indolepropionic acid (IPA), is increasingly being studied as a marker for the onset and development of metabolic disorders, including type 2 diabetes (T2D) and non-alcoholic fatty liver disease (NAFLD). The IPA levels heavily depend on the diet, particularly dietary fiber, and show huge variations among individuals. We suggest that these variations could partially be explained using genetic variants known to be associated with specific diseases such as T2D. In this narrative review, we elaborate on the beneficial effects of IPA in the mitigation of T2D and NAFLD, and further study the putative interactions between IPA and well-known genetic variants (', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': None, 'doi': '10.3390/nu14214695\\n10.1042/bj0170907\\n10.1007/s12031-002-0036-0\\n10.3390/ijms23031222\\n10.1080/19490976.2019.1586038\\n10.1007/s13668-020-00307-3\\n10.1210/endocr/bqaa235\\n10.1038/srep46337\\n10.1038/s41387-018-0046-9\\n10.3389/fendo.2022.841703\\n10.3389/fphar.2019.00877\\n10.1016/S0140-6736(69)90757-0\\n10.1017/S0954422417000117\\n10.1016/S2468-1253(19)30257-2\\n10.1007/s00125-006-0198-3\\n10.1080/13813455.2017.1398262\\n10.1038/nature24661\\n10.3389/fmicb.2020.575586\\n10.1038/s41586-022-04884-x\\n10.3390/nu13103509\\n10.1017/S0007114500001070\\n10.1155/2016/8952520\\n10.1007/s00535-018-1521-8\\n10.1038/s41467-018-05470-4\\n10.3390/ijms222212435\\n10.1128/JB.187.6.2084-2092.2005\\n10.1016/j.chom.2018.05.003\\n10.3390/ijms22189904\\n10.1186/s40168-020-00837-6\\n10.1038/s41467-019-13498-3\\n10.1016/j.immuni.2013.08.003\\n10.1073/pnas.0812874106\\n10.1016/j.immuni.2014.06.014\\n10.1111/nmo.13178\\n10.1038/s41467-021-23460-x\\n10.1016/j.ajpath.2018.01.011\\n10.1080/19490976.2020.1788899\\n10.1186/s40779-022-00397-w\\n10.3390/metabo10050181\\n10.1038/s41467-020-14676-4\\n10.1016/j.xcrm.2021.100246\\n10.3389/fimmu.2021.653208\\n10.1038/s42003-020-01383-5\\n10.3389/fmed.2021.648259\\n10.1136/gutjnl-2021-324053\\n10.1016/j.ebiom.2019.11.051\\n10.1007/s00253-019-10156-y\\n10.1016/j.nutres.2020.03.005\\n10.1128/mSystems.00313-20\\n10.1002/mnfr.201900580\\n10.1172/jci.insight.145207\\n10.2337/dc21-0099\\n10.3390/microorganisms8010111\\n10.1093/jn/nxy053\\n10.1007/s11306-011-0320-5\\n10.1093/cdn/nzz040.P20-003-19\\n10.1002/mnfr.202100349\\n10.1111/febs.16470\\n10.1002/oby.22870\\n10.1093/cdn/nzaa062_016\\n10.1186/s13073-020-00806-6\\n10.2337/dc21-1705\\n10.1093/gerona/glt152\\n10.1371/journal.pone.0198156\\n10.2337/dc19-2533\\n10.3390/metabo12010069\\n10.3390/metabo11040228\\n10.1080/13813455.2021.2024577\\n10.1152/ajpgi.00105.2020\\n10.1016/j.bmcl.2006.05.007\\n10.3390/nu11030591\\n10.1038/ng.2982\\n10.1007/s00125-007-0656-6\\n10.1038/oby.2008.618\\n10.2337/diabetes.51.8.2581\\n10.1007/s00125-012-2752-5\\n10.2337/db08-1607\\n10.1038/ng.2383\\n10.1038/s41598-020-65051-8\\n10.1016/j.molmet.2020.101092\\n10.1111/joim.12892\\n10.3390/nu14112187\\n10.1002/hep.26342\\n10.1111/ijpo.270\\n10.1161/CIRCRESAHA.122.321253\\n10.1016/j.jvs.2017.09.029\\n10.1016/S0041-1345(02)03616-3\\n10.1038/s12276-019-0304-5\\n10.1002/1097-4644(20010601)81:3<507::AID-JCB1064>3.0.CO;2-M\\n10.3390/nu12103100\\n10.3390/cancers12092411\\n10.1039/D1FO02181H\\n10.1136/gutjnl-2019-319664\\n10.1096/fj.14-259515\\n10.1007/s11356-022-21075-3\\n10.1016/j.tox.2021.152996\\n10.1002/smll.201905233\\n10.18999/nagjms.79.4.477', 'publication_date': datetime.date(2022, 11, 12), 'authors': [{'lastname': 'Sehgal', 'firstname': 'Ratika', 'initials': 'R', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.'}, {'lastname': 'de Mello', 'firstname': 'Vanessa D', 'initials': 'VD', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.'}, {'lastname': 'Männistö', 'firstname': 'Ville', 'initials': 'V', 'affiliation': 'Department of Medicine, University of Eastern Finland, Kuopio University Hospital, 70211 Kuopio, Finland.'}, {'lastname': 'Lindström', 'firstname': 'Jaana', 'initials': 'J', 'affiliation': 'Public Health Prevention Unit, Finnish Institute for Health and Welfare, 00271 Helsinki, Finland.'}, {'lastname': 'Tuomilehto', 'firstname': 'Jaakko', 'initials': 'J', 'affiliation': 'Public Health Prevention Unit, Finnish Institute for Health and Welfare, 00271 Helsinki, Finland.\\nDepartment of Public Health, University of Helsinki, 00014 Helsinki, Finland.\\nDiabetes Research Group, King Abdulaziz University, Jeddah 21589, Saudi Arabia.'}, {'lastname': 'Pihlajamäki', 'firstname': 'Jussi', 'initials': 'J', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.\\nDepartment of Medicine, Endocrinology and Clinical Nutrition, Kuopio University Hospital, 70210 Kuopio, Finland.'}, {'lastname': 'Uusitupa', 'firstname': 'Matti', 'initials': 'M', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.'}]}\n",
      "\n",
      "{'pubmed_id': '36360783', 'title': 'Decoding Diabetes Biomarkers and Related Molecular Mechanisms by Using Machine Learning, Text Mining, and Gene Expression Analysis.', 'keywords': ['bioinformatics', 'diabetes', 'gene expression', 'protein–protein interaction network', 'text mining'], 'journal': 'International journal of environmental research and public health', 'abstract': 'The molecular basis of diabetes mellitus is yet to be fully elucidated. We aimed to identify the most frequently reported and differential expressed genes (DEGs) in diabetes by using bioinformatics approaches. Text mining was used to screen 40,225 article abstracts from diabetes literature. These studies highlighted 5939 diabetes-related genes spread across 22 human chromosomes, with 112 genes mentioned in more than 50 studies. Among these genes, ', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': None, 'doi': '10.3390/ijerph192113890\\n10.1038/414782a\\n10.2522/ptj.20080008\\n10.1371/journal.pone.0243270\\n10.3389/fgene.2020.00734\\n10.3390/cancers13215268\\n10.1097/MD.0000000000020338\\n10.1007/s11011-019-00465-6\\n10.1007/s12551-018-0490-8\\n10.1097/MD.0000000000017504\\n10.1111/brv.12457\\n10.1186/s12874-019-0681-4\\n10.1016/j.cmpb.2019.04.008\\n10.3390/e23010018\\n10.1016/j.ijmedinf.2021.104453\\n10.1093/nar/gks1193\\n10.1093/bioinformatics/bty721\\n10.1093/bioinformatics/btz931\\n10.1093/bioinformatics/btr261\\n10.1093/nar/gkq973\\n10.36462/H.BioSci.20195\\n10.1093/bioinformatics/btp616\\n10.1172/JCI200216857\\n10.1172/JCI200522365\\n10.1186/s13098-020-00557-9\\n10.1093/eurheartj/ehab724.3196\\n10.1007/s12013-007-0025-6\\n10.1074/jbc.R109.040543\\n10.1016/j.jgg.2017.11.007\\n10.1016/j.molmet.2014.12.007\\n10.1016/j.biopha.2017.03.056\\n10.1016/j.biopha.2017.03.102\\n10.2337/db09-1187\\n10.1530/JOE-17-0002\\n10.1038/ng2068\\n10.1038/s41598-019-54545-9\\n10.1194/jlr.M080382\\n10.1038/s41419-020-03312-0\\n10.1016/j.reprotox.2021.10.008\\n10.3390/ijms22136924\\n10.1016/j.str.2008.10.018\\n10.1038/ncomms3863\\n10.1016/j.cellsig.2015.09.016\\n10.1002/jcb.20343\\n10.1186/s12916-020-1503-6\\n10.2147/DMSO.S345081\\n10.1016/j.ejphar.2022.174956\\n10.1210/jc.2009-0384\\n10.1007/s001250051573\\n10.1159/000295775\\n10.3389/fimmu.2015.00493\\n10.1039/b914182k\\n10.1681/ASN.2005080859\\n10.3390/cells8070751\\n10.2337/db20-0038\\n10.1210/jc.2018-02094\\n10.1371/journal.pone.0193684\\n10.1002/humu.23639\\n10.1093/hmg/ddx262\\n10.1371/journal.pone.0234132\\n10.1159/000524849\\n10.3390/cells9030730\\n10.1016/j.jid.2017.11.038\\n10.3390/cells8020178\\n10.1038/nm1185', 'publication_date': datetime.date(2022, 11, 12), 'authors': [{'lastname': 'Elsherbini', 'firstname': 'Amira M', 'initials': 'AM', 'affiliation': 'Department of Oral Biology, Faculty of Dentistry, Mansoura University, Mansoura 35116, Egypt.'}, {'lastname': 'Alsamman', 'firstname': 'Alsamman M', 'initials': 'AM', 'affiliation': 'Agricultural Genetic Engineering Research Institute, Agricultural Research Center, Giza 12619, Egypt.'}, {'lastname': 'Elsherbiny', 'firstname': 'Nehal M', 'initials': 'NM', 'affiliation': 'Department of Pharmaceutical Chemistry, Faculty of Pharmacy, University of Tabuk, Tabuk 71491, Saudi Arabia.\\nDepartment of Biochemistry, Faculty of Pharmacy, Mansoura University, Mansoura 35116, Egypt.'}, {'lastname': 'El-Sherbiny', 'firstname': 'Mohamed', 'initials': 'M', 'affiliation': 'Department of Basic Medical Sciences, College of Medicine, AlMaarefa University, Riyadh 71666, Saudi Arabia.\\nDepartment of Anatomy, Mansoura Faculty of Medicine, Mansoura University, Mansoura 35116, Egypt.'}, {'lastname': 'Ahmed', 'firstname': 'Rehab', 'initials': 'R', 'affiliation': 'Department of Natural Products and Alternative Medicine, Faculty of Pharmacy, University of Tabuk, Tabuk 71491, Saudi Arabia.\\nDepartment of Pharmaceutics, Faculty of Pharmacy, University of Khartoum, Khartoum 11111, Sudan.'}, {'lastname': 'Ebrahim', 'firstname': 'Hasnaa Ali', 'initials': 'HA', 'affiliation': 'Department of Basic Medical Sciences, College of Medicine, Princess Nourah bint Abdulrahman University, Riyadh 11671, Saudi Arabia.'}, {'lastname': 'Bakkach', 'firstname': 'Joaira', 'initials': 'J', 'affiliation': 'Biomedical Genomics and Oncogenetics Research Laboratory, Faculty of Sciences and Techniques of Tangier, Abdelmalek Essaâdi University Morocco, Tétouan 93000, Morocco.'}]}\n",
      "\n",
      "{'pubmed_id': '36358989', 'title': 'FGF9 Promotes Expression of HAS2 in Palatal Elevation via the Wnt/β-Catenin/TCF7L2 Pathway.', 'keywords': ['FGF9', 'HAS2', 'TCF7L2', 'Wnt/β-catenin pathway', 'hyaluronic acid'], 'journal': 'Biomolecules', 'abstract': 'Using Ddx4-Cre mice, we generated the \\nFGF9 promotes HAS2 expression via Wnt/β-catenin/TCF7L2 pathway with TCF7L2 activating transcription of ', 'conclusions': None, 'methods': None, 'results': '', 'copyrights': None, 'doi': '10.3390/biom12111639\\n10.3389/fphys.2016.00067\\n10.1002/ajmg.a.31965\\n10.3109/08977199509036882\\n10.1038/ng.316\\n10.1002/(SICI)1097-0177(199909)216:1<72::AID-DVDY9>3.0.CO;2-9\\n10.1177/0022034518785336\\n10.1242/dev.128.11.2095\\n10.3389/fphys.2021.653040\\n10.1177/0022034517703580\\n10.1038/s41368-021-00122-8\\n10.1016/j.ydbio.2019.09.004\\n10.1182/blood-2014-07-590513\\n10.1080/15476278.2020.1735239\\n10.3389/fphys.2013.00010\\n10.1002/ar.24263\\n10.3389/fcell.2019.00340\\n10.1177/0022034519872478\\n10.3390/biom10111525\\n10.1530/REP-16-0240\\n10.3389/fcvm.2021.701279\\n10.1371/journal.pone.0241281\\n10.1016/j.biopha.2018.04.026\\n10.1002/bdrc.20180\\n10.1242/dev.134577\\n10.1002/lio2.214\\n10.3390/cells9030608\\n10.1016/j.mce.2016.05.023\\n10.1016/j.ydbio.2016.04.005\\n10.1007/BF00428098\\n10.1016/j.ydbio.2016.01.008\\n10.1042/bj3480029\\n10.1074/jbc.274.35.25085\\n10.3389/fcell.2020.00264\\n10.1242/dmm.037051\\n10.1242/dev.158048\\n10.1126/science.aat1178\\n10.1016/j.cell.2012.05.012\\n10.1038/s41598-018-20641-5\\n10.1126/scisignal.aat9519\\n10.1016/j.devcel.2008.04.015\\n10.1016/j.cellsig.2017.08.008\\n10.15252/embj.201798873\\n10.1186/s13098-019-0449-3\\n10.1210/me.2003-0225\\n10.1016/j.semcdb.2009.09.002\\n10.3892/etm.2019.8308\\n10.1016/j.chemosphere.2020.126889\\n10.1002/anie.200462552', 'publication_date': datetime.date(2022, 11, 12), 'authors': [{'lastname': 'Sun', 'firstname': 'Yidan', 'initials': 'Y', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Ying', 'firstname': 'Xiyu', 'initials': 'X', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Li', 'firstname': 'Ruomei', 'initials': 'R', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Weng', 'firstname': 'Mengjia', 'initials': 'M', 'affiliation': 'Department of Stomatology, Huadong Hospital, Fudan University, Shanghai 200040, China.'}, {'lastname': 'Shi', 'firstname': 'Jiajun', 'initials': 'J', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Chen', 'firstname': 'Zhenqi', 'initials': 'Z', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}]}\n",
      "\n",
      "{'pubmed_id': '36344976', 'title': 'CHRNB2 represses pancreatic cancer migration and invasion via inhibiting β-catenin pathway.', 'keywords': ['CHRNB2', 'Lymph node ratio', 'Pancreatic cancer', 'Weighted gene co-expression network analysis', 'β-catenin pathway'], 'journal': 'Cancer cell international', 'abstract': 'Pancreatic cancer is one of the most lethal disease with highly fatal and aggressive properties. Lymph node ratio (LNR), the ratio of the number of metastatic lymph nodes to the total number of examined lymph nodes, is an important index to assess lymphatic metastasis and predict prognosis, but the molecular mechanism underlying high LNR was unclear.\\nGene expression and clinical information data of pancreatic cancer were obtained from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO). Patients in TCGA were averagely divided into low and high LNR groups. Then, Weighted Gene Co-expression Network Analysis (WGCNA) was utilized to build co-expression network to explore LNR-related modules and hub genes. GO and KEGG analysis was performed to find key pathways related to lymph node metastasis. Next, GSE101448 and the overall survival data in TCGA was employed to further select significant genes from hub genes. Considering the key role of CHRNB2 in LNR and survival, gene set enrichment analysis (GSEA) was applied to find pathways related to CHRNB2 expression in pancreatic cancer. The contribution of CHRNB2 to migrative and invasive ability of pancreatic cancer cells was confirmed by Transwell assays. We finally explored the role of CHRNB2 in EMT and β-catenin pathway via Western Blot.\\nHigh LNR was significantly related to high T stages and poor prognosis. In WGCNA, 14 hub genes (COL5A1, FN1, THBS2, etc.) were positively related to high LNR, 104 hub genes (FFAR1, SCG5, TMEM63C, etc.) were negatively related to high LNR. After taking the intersection with GSE101448, 13 genes (CDK5R2, SYT7, CACNA2D2, etc.) which might prevent lymph node metastasis were further selected. Among them, CHRNB2 showed the strongest relationship with long survival. Moreover, CHRNB2 also negatively related to the T stages and LNR. Next, knockdown of CHRNB2 expression could acetylcholine (ACh)-independently increase the migration and invasion of pancreatic cancer cells, while CHRNB2 overexpression ACh-independently decrease the migration and invasion of pancreatic cancer cells. For exploring the underlying mechanism, CHRNB2 downregulated β-catenin pathway might through controlling its upstream regulators such as SOX6, SRY, SOX17, and TCF7L2.\\nCHRNB2 negatively relates to lymph node metastasis in pancreatic cancer patients. CHRNB2 could inhibit β-catenin pathway, EMT, migration and invasion of pancreatic cancer cells via ACh-independent mechanism.', 'conclusions': None, 'methods': None, 'results': 'High LNR was significantly related to high T stages and poor prognosis. In WGCNA, 14 hub genes (COL5A1, FN1, THBS2, etc.) were positively related to high LNR, 104 hub genes (FFAR1, SCG5, TMEM63C, etc.) were negatively related to high LNR. After taking the intersection with GSE101448, 13 genes (CDK5R2, SYT7, CACNA2D2, etc.) which might prevent lymph node metastasis were further selected. Among them, CHRNB2 showed the strongest relationship with long survival. Moreover, CHRNB2 also negatively related to the T stages and LNR. Next, knockdown of CHRNB2 expression could acetylcholine (ACh)-independently increase the migration and invasion of pancreatic cancer cells, while CHRNB2 overexpression ACh-independently decrease the migration and invasion of pancreatic cancer cells. For exploring the underlying mechanism, CHRNB2 downregulated β-catenin pathway might through controlling its upstream regulators such as SOX6, SRY, SOX17, and TCF7L2.', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1186/s12935-022-02768-8\\n10.3390/ijms23031231\\n10.1172/JCI146431\\n10.3390/ijms20174242\\n10.1038/s41573-022-00410-w', 'publication_date': datetime.date(2022, 11, 8), 'authors': [{'lastname': 'Qin', 'firstname': 'Cheng', 'initials': 'C', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Li', 'firstname': 'Tianhao', 'initials': 'T', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.\\nDivision of Plastic Surgery, Department of Surgery, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Wang', 'firstname': 'Yuanyang', 'initials': 'Y', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Zhao', 'firstname': 'Bangbo', 'initials': 'B', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Li', 'firstname': 'Zeru', 'initials': 'Z', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Li', 'firstname': 'Tianyu', 'initials': 'T', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Yang', 'firstname': 'Xiaoying', 'initials': 'X', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Zhao', 'firstname': 'Yutong', 'initials': 'Y', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Wang', 'firstname': 'Weibin', 'initials': 'W', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China. wwb_xh@163.com.'}]}\n",
      "\n",
      "{'pubmed_id': '36300877', 'title': 'Clinical signs of type 1 diabetes are associated with type 2 diabetes marker transcription factor 7-like 2 polymorphism.', 'keywords': ['Anti-islet autoantibodies', 'Transcription factor 7-like 2 gene', 'Type 1 diabetes'], 'journal': 'Journal of diabetes investigation', 'abstract': 'We aimed to assess the distribution of transcription factor 7-like 2 gene TCF7L2 (rs7903146) polymorphism and to find possible associations between TCF7L2 and the characteristics of type 1 diabetes.\\nWe studied 190 newly diagnosed type 1 diabetes patients (median age 12.7\\u2009years, range 2.0-72.5) and 246 controls (median age 23.8\\u2009years, range 1.4-81.5) for TCF7L2 single nucleotide polymorphism. We determined anti-islet autoantibodies, random C-peptide levels, diabetes associated HLA DR/DQ haplotypes and genotypes in all patients.\\nThere were no differences in the distribution of TCF7L2 single nucleotide polymorphism between patients and controls. However, patients with in type 1 diabetes, after adjusting for age and sex, subjects carrying C allele were at risk for a C-peptide level lower than 0.5\\u2009nmol/L (OR 5.65 [95% CI: 1.14-27.92]) and for zinc transporter 8 autoantibody positivity (5.22 [1.34-20.24]). Participants without T allele were associated with a higher level of islet antigen-2 autoantibodies (3.51 [1.49-8.27]) and zinc transporter 8 autoantibodies (2.39 [1.14-4.99]).\\nThe connection of TCF7L2 polymorphism with zinc transporter 8 and islet antigen-2 autoantibodies and C-peptide levels in patients supports the viewpoint that TCF7L2 is associated with the clinical signs and autoimmune characteristics of type 1 diabetes. The mechanisms of the interaction between the TCF7L2 risk genotype and anti-islet autoantibodies need to be studied further.', 'conclusions': None, 'methods': None, 'results': 'There were no differences in the distribution of TCF7L2 single nucleotide polymorphism between patients and controls. However, patients with in type 1 diabetes, after adjusting for age and sex, subjects carrying C allele were at risk for a C-peptide level lower than 0.5\\u2009nmol/L (OR 5.65 [95% CI: 1.14-27.92]) and for zinc transporter 8 autoantibody positivity (5.22 [1.34-20.24]). Participants without T allele were associated with a higher level of islet antigen-2 autoantibodies (3.51 [1.49-8.27]) and zinc transporter 8 autoantibodies (2.39 [1.14-4.99]).', 'copyrights': '© 2022 The Authors. Journal of Diabetes Investigation published by Asian Association for the Study of Diabetes (AASD) and John Wiley & Sons Australia, Ltd.', 'doi': '10.1111/jdi.13933', 'publication_date': datetime.date(2022, 10, 28), 'authors': [{'lastname': 'Ergür', 'firstname': 'Efe', 'initials': 'E', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Ergür', 'firstname': 'Ege', 'initials': 'E', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Alnek', 'firstname': 'Kristi', 'initials': 'K', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Metsküla', 'firstname': 'Kaja', 'initials': 'K', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Peet', 'firstname': 'Aleksandr', 'initials': 'A', 'affiliation': \"Department of Pediatrics, Institute of Clinical Medicine, University of Tartu, Tartu, Estonia.\\nChildren's Clinic of Tartu University Hospital, Tartu, Estonia.\"}, {'lastname': 'Lubi', 'firstname': 'Maire', 'initials': 'M', 'affiliation': 'Department of Internal Medicine, Institute of Clinical Medicine, University of Tartu, Tartu, Estonia.\\nInternal Medicine Clinic of Tartu University Hospital, Tartu, Estonia.'}, {'lastname': 'Heilman', 'firstname': 'Kaire', 'initials': 'K', 'affiliation': \"Tallinn Children's Hospital, Tallinn, Estonia.\"}, {'lastname': 'Uibo', 'firstname': 'Raivo', 'initials': 'R', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}]}\n",
      "\n",
      "{'pubmed_id': '36295578', 'title': 'ITGA2 Gene Polymorphism Is Associated with Type 2 Diabetes Mellitus in the Kazakhstan Population.', 'keywords': ['BglII polymorphism in ITGA2 gene', 'diabetic retinopathy', 'type 2 diabetes mellitus', 'α2β1 integrin receptor'], 'journal': 'Medicina (Kaunas, Lithuania)', 'abstract': '', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': None, 'doi': '10.3390/medicina58101416\\n10.1038/nrendo.2017.151\\n10.1172/jci.insight.93751\\n10.1016/j.visres.2017.03.006\\n10.1016/j.preteyeres.2013.02.001\\n10.1038/nrdp.2016.12\\n10.1016/j.pop.2015.05.005\\n10.2337/diab.46.11.1829\\n10.1046/j.1464-5491.2002.00820.x\\n10.1038/s41581-020-0278-5\\n10.21037/aes.2018.08.04\\n10.1093/ndt/gfl641\\n10.1097/IIO.0b013e31819fd5d7\\n10.1155/2010/861291\\n10.12809/hkmj164869\\n10.1371/journal.pone.0094160\\n10.3109/02713683.2014.932386\\n10.1007/s00125-015-3569-9\\n10.3109/02713683.2015.1084642\\n10.1155/2016/5305282\\n10.2337/diabetes.50.6.1505\\n10.15171/jnp.2018.54\\n10.1182/blood.V92.7.2382\\n10.3132/dvdr.2005.001\\n10.1186/s12933-018-0763-3\\n10.1055/s-0035-1556731\\n10.5858/2002-126-0305-IOPCRP\\n10.1182/blood.V95.5.1560.005k43_1560_1564\\n10.1007/s10038-003-0060-0\\n10.1080/08820538.2019.1632354\\n10.2147/TACG.S31979\\n10.1182/blood.V89.6.1939\\n10.1182/blood.V97.6.1721\\n10.17513/spno.29283\\n10.1177/1099800420977685\\n10.4172/2155-6156.1000223', 'publication_date': datetime.date(2022, 10, 28), 'authors': [{'lastname': 'Magazova', 'firstname': 'Aizhan', 'initials': 'A', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.\\nDepartment of Biochemistry, Asfendiyarov Kazakh National Medical University, Almaty 050012, Kazakhstan.'}, {'lastname': 'Ashirbekov', 'firstname': 'Yeldar', 'initials': 'Y', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.'}, {'lastname': 'Abaildayev', 'firstname': 'Arman', 'initials': 'A', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.'}, {'lastname': 'Satken', 'firstname': 'Kantemir', 'initials': 'K', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.'}, {'lastname': 'Balmukhanova', 'firstname': 'Altynay', 'initials': 'A', 'affiliation': 'Department of Health Policy and Organization, Al-Farabi Kazakh National University, Almaty 050040, Kazakhstan.'}, {'lastname': 'Akanov', 'firstname': 'Zhanay', 'initials': 'Z', 'affiliation': 'City Center of Diabetes AAA Medical Clinic, Almaty 050040, Kazakhstan.'}, {'lastname': 'Jainakbayev', 'firstname': 'Nurlan', 'initials': 'N', 'affiliation': 'Department of Public Health, Kazakh-Russian Medical University, Almaty 050004, Kazakhstan.'}, {'lastname': 'Balmukhanova', 'firstname': 'Aigul', 'initials': 'A', 'affiliation': 'International Medical School, Caspian University, Almaty 050000, Kazakhstan.'}, {'lastname': 'Sharipov', 'firstname': 'Kamalidin', 'initials': 'K', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.\\nDepartment of Biochemistry, Asfendiyarov Kazakh National Medical University, Almaty 050012, Kazakhstan.'}]}\n",
      "\n",
      "{'pubmed_id': '36282337', 'title': 'Milder loss of insulin-containing islets in individuals with type 1 diabetes and type 2 diabetes-associated TCF7L2 genetic variants.', 'keywords': ['African American', 'Genetics', 'Heterogeneity', 'Insulin-containing cells', 'Pancreas', 'Phenotype', 'Precision medicine', 'Race', 'TCF7L2', 'Type 1 diabetes'], 'journal': 'Diabetologia', 'abstract': 'TCF7L2 variants are the strongest genetic risk factor for type 2 diabetes. In individuals with type 1 diabetes, these variants are associated with a higher C-peptide AUC, a lower glucose AUC during an OGTT, single autoantibody positivity near diagnosis, particularly in individuals older than 12 years of age, and a lower frequency of type 1 diabetes-associated HLA genotypes. Based on initial observations from clinical cohorts, we tested the hypothesis that type 2 diabetes-predisposing TCF7L2 genetic variants are associated with a higher percentage of residual insulin-containing cells (ICI%) in pancreases of donors with type 1 diabetes, by examining genomic data and pancreatic tissue samples from the Network for Pancreatic Organ donors with Diabetes (nPOD) programme.\\nWe analysed nPOD donors with type 1 diabetes (n=110; mean±SD age at type 1 diabetes onset 12.2±7.9 years, mean±SD diabetes duration 15.3±13.7 years, 53% male, 80% non-Hispanic White, 12.7% African American, 7.3% Hispanic) using data pertaining to residual beta cell number; quantified islets containing insulin-positive beta cells in pancreatic tissue sections; and expressed these values as a percentage of the total number of islets from each donor (mean ± SD ICI% 9.8±21.5, range 0-92.2).\\nDonors with a high ICI% (≥5) (n=30; 27%) vs a low ICI% (<5) (n=80; 73%) were older at onset (15.3±6.9 vs 11.1±8 years, p=0.013), had a shorter diabetes duration at donor tissue procurement (7.0±7.4 vs 18.5±14.3 years, p<0.001), a higher African ancestry score (0.2±0.3 vs 0.1±0.2, p=0.043) and a lower European ancestry score (0.7±0.3 vs 0.9±0.3, p=0.023). After adjustment for age of onset (p=0.105), diabetes duration (p<0.001), BMI z score (p=0.145), sex (p=0.351) and African American race (p=0.053), donors with the TCF7L2 rs7903146 T allele (TC or TT, 45.5%) were 2.93 times (95% CI 1.02, 8.47) more likely to have a high ICI% than those without it (CC) (p=0.047).\\nOverall, these data support the presence of a type 1 diabetes endotype associated with a genetic factor that predisposes to type 2 diabetes, with donors in this category exhibiting less severe beta cell loss. It is possible that in these individuals the disease pathogenesis may include mechanisms associated with type 2 diabetes and thus this may provide an explanation for the poor response to immunotherapies to prevent type 1 diabetes or its progression in a subset of individuals. If so, strategies that target both type 1 diabetes and type 2 diabetes-associated factors when they are present may increase the success of prevention and treatment in these individuals.', 'conclusions': None, 'methods': None, 'results': 'Donors with a high ICI% (≥5) (n=30; 27%) vs a low ICI% (<5) (n=80; 73%) were older at onset (15.3±6.9 vs 11.1±8 years, p=0.013), had a shorter diabetes duration at donor tissue procurement (7.0±7.4 vs 18.5±14.3 years, p<0.001), a higher African ancestry score (0.2±0.3 vs 0.1±0.2, p=0.043) and a lower European ancestry score (0.7±0.3 vs 0.9±0.3, p=0.023). After adjustment for age of onset (p=0.105), diabetes duration (p<0.001), BMI z score (p=0.145), sex (p=0.351) and African American race (p=0.053), donors with the TCF7L2 rs7903146 T allele (TC or TT, 45.5%) were 2.93 times (95% CI 1.02, 8.47) more likely to have a high ICI% than those without it (CC) (p=0.047).', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1007/s00125-022-05818-y', 'publication_date': datetime.date(2022, 10, 26), 'authors': [{'lastname': 'Redondo', 'firstname': 'Maria J', 'initials': 'MJ', 'affiliation': \"Department of Pediatrics, Baylor College of Medicine, Texas Children's Hospital, Houston, TX, USA. redondo@bcm.edu.\"}, {'lastname': 'Richardson', 'firstname': 'Sarah J', 'initials': 'SJ', 'affiliation': 'Islet Biology Group (IBEx), Exeter Centre of Excellence for Diabetes Research (EXCEED), University of Exeter College of Medicine and Health, Exeter, UK. s.richardson@exeter.ac.uk.'}, {'lastname': 'Perry', 'firstname': 'Daniel', 'initials': 'D', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}, {'lastname': 'Minard', 'firstname': 'Charles G', 'initials': 'CG', 'affiliation': 'Institute for Clinical and Translational Research, Baylor College of Medicine, Houston, TX, USA.'}, {'lastname': 'Carr', 'firstname': 'Alice L J', 'initials': 'ALJ', 'affiliation': 'Islet Biology Group (IBEx), Exeter Centre of Excellence for Diabetes Research (EXCEED), University of Exeter College of Medicine and Health, Exeter, UK.'}, {'lastname': 'Brusko', 'firstname': 'Todd', 'initials': 'T', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}, {'lastname': 'Kusmartseva', 'firstname': 'Irina', 'initials': 'I', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}, {'lastname': 'Pugliese', 'firstname': 'Alberto', 'initials': 'A', 'affiliation': 'Diabetes Research Institute, Department of Medicine, Division of Endocrinology, Department of Microbiology and Immunology, Miller School of Medicine, University of Miami, Miami, FL, USA.'}, {'lastname': 'Atkinson', 'firstname': 'Mark A', 'initials': 'MA', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}]}\n",
      "\n",
      "{'pubmed_id': '36263318', 'title': '', 'keywords': ['TCF7L2 gene', 'adipose tissue', 'postprandial hypertriglyceridemia', 'prediabetes', 'type 2 diabetes mellitus'], 'journal': 'Frontiers in endocrinology', 'abstract': 'Postprandial lipid responses to a standardized fat challenge test were performed in 620 Asian Indian subjects (310 with NGT and 310 with T2DM/prediabetes) and compared between the risk and wild genotypes of the rs7903146 \\nT allele of rs7903146 \\nThere is significant PPTg dysmetabolism associated with the risk allele of rs7903146 polymorphism as well as adipocyte expression of ', 'conclusions': 'There is significant PPTg dysmetabolism associated with the risk allele of rs7903146 polymorphism as well as adipocyte expression of ', 'methods': None, 'results': 'T allele of rs7903146 ', 'copyrights': 'Copyright © 2022 Madhu, Mishra, Mannar, Aslam, Banerjee and Agrawal.', 'doi': '10.3389/fendo.2022.973718\\n10.1186/1471-2350-10-15\\n10.2337/db09-1386\\n10.1007/s00125-006-0502-2\\n10.1038/ng1732\\n10.2337/db11-0550\\n10.1016/j.metabol.2007.04.012\\n10.1007/s00125-006-0507-x\\n10.1007/s11892-008-0026-x\\n10.1056/NEJMoa062418\\n10.1172/JCI30706\\n10.2337/db17-0318\\n10.1016/S0065-230X(08)60783-6\\n10.1016/0092-8674(92)90630-U\\n10.1155/2011/567305\\n10.1101/003202\\n10.2337/db12-0365\\n10.1074/jbc.M411487200\\n10.1016/j.cell.2012.10.053\\n10.1371/journal.pone.0043390\\n10.1038/srep43128\\n10.3945/jn.108.096461\\n10.1007/s13410-018-0678-2\\n10.2337/diabetes.48.6.1258\\n10.7326/0003-4819-131-1-199907060-00006\\n10.1016/j.nut.2005.01.009\\n10.1016/0168-8227(94)90269-0\\n10.1016/j.jacl.2017.08.001\\n10.1371/journal.pone.0145730\\n10.1111/joim.12540\\n10.1007/s13410-018-0678-2\\n10.1111/ddg.13528\\n10.2337/dc17-S005\\n10.1016/j.diabres.2008.01.016\\n10.1006/meth.2001.1262\\n10.2337/db06-0474\\n10.1007/s10038-007-0231-5\\n10.1016/j.metabol.2007.04.012\\n10.1016/j.atherosclerosissup.2010.03.002\\n10.1111/j.1469-1809.2008.00443.x\\n10.1002/hep.22659\\n10.3967/bes2016.108\\n10.2337/db06-0338\\n10.1186/s13098-018-0338-1\\n10.1016/j.bbrc.2008.03.006\\n10.1007/s00125-007-0850-6\\n10.1016/j.metabol.2008.04.016\\n10.1007/s00125-020-05292-4\\n10.1210/jc.2016-1594\\n10.1016/j.molmet.2019.03.003\\n10.1210/jcem.86.11.7992\\n10.1371/journal.pone.0000812\\n10.1371/journal.pone.0237974\\n10.1093/jn/134.1.205\\n10.3390/biom10091230\\n10.3390/biom10040601\\n10.1080/21623945.2020.1776082\\n10.1002/oby.21377\\n10.1371/journal.pone.0018284\\n10.1210/jc.2016-1594\\n10.1016/j.jacc.2011.02.038\\n10.1155/2012/983814', 'publication_date': datetime.date(2022, 10, 21), 'authors': [{'lastname': 'Madhu', 'firstname': 'Sri Venkata', 'initials': 'SV', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Mishra', 'firstname': 'Brijesh Kumar', 'initials': 'BK', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Mannar', 'firstname': 'Velmurugan', 'initials': 'V', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Aslam', 'firstname': 'Mohd', 'initials': 'M', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Banerjee', 'firstname': 'Basudev', 'initials': 'B', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Agrawal', 'firstname': 'Vivek', 'initials': 'V', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}]}\n",
      "\n",
      "{'pubmed_id': '36217307', 'title': 'Regulation of the THRA gene, encoding the thyroid hormone nuclear receptor TRα1, in intestinal lesions.', 'keywords': ['THRA', 'TRα1', 'colon cancer', 'intestinal organoids', 'thyroid hormone nuclear receptor'], 'journal': 'Molecular oncology', 'abstract': 'The THRA gene, encoding the thyroid hormone nuclear receptor TRα1, is expressed in an increasing gradient at the bottom of intestinal crypts, overlapping with high Wnt and Notch activities. Importantly, THRA is upregulated in colorectal cancers, particularly in the high-Wnt molecular subtype. The basis of this specific and/or altered expression pattern has remained unknown. To define the mechanisms controlling THRA transcription and TRα1 expression, we used multiple in\\u2009vitro and ex\\u2009vivo approaches. Promoter analysis demonstrated that transcription factors important for crypt homeostasis and altered in colorectal cancers, such as transcription factor 7-like 2 (TCF7L2; Wnt pathway), recombining binding protein suppressor of hairless (RBPJ; Notch pathway), and homeobox protein CDX2 (epithelial cell identity), modulate THRA activity. Specifically, although TCF7L2 and CDX2 stimulated THRA, RBPJ induced its repression. In-depth analysis of the Wnt-dependent increase showed direct regulation of the THRA promoter in cells and of TRα1 expression in murine enteroids. Given our previous results on the control of the Wnt pathway by TRα1, our new results unveil a complex regulatory loop and synergy between these endocrine and epithelial-cell-intrinsic signals. Our work describes, for the first time, the regulation of the THRA gene in specific cell and tumor contexts.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2022 The Authors. Molecular Oncology published by John Wiley & Sons Ltd on behalf of Federation of European Biochemical Societies.', 'doi': '10.1002/1878-0261.13298', 'publication_date': datetime.date(2022, 10, 12), 'authors': [{'lastname': 'Giolito', 'firstname': 'Maria Virginia', 'initials': 'MV', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.\\nINSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}, {'lastname': 'La Rosa', 'firstname': 'Théo', 'initials': 'T', 'affiliation': 'INSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}, {'lastname': 'Farhat', 'firstname': 'Diana', 'initials': 'D', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.\\nINSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}, {'lastname': 'Bodoirat', 'firstname': 'Serguei', 'initials': 'S', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.'}, {'lastname': 'Guardia', 'firstname': 'Gabriela D A', 'initials': 'GDA', 'affiliation': 'Centro de Oncologia Molecular, Hospital Sírio-Libanês, São Paulo, Brazil.'}, {'lastname': 'Domon-Dell', 'firstname': 'Claire', 'initials': 'C', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.'}, {'lastname': 'Galante', 'firstname': 'Pedro A F', 'initials': 'PAF', 'affiliation': 'Centro de Oncologia Molecular, Hospital Sírio-Libanês, São Paulo, Brazil.'}, {'lastname': 'Freund', 'firstname': 'Jean-Noel', 'initials': 'JN', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.'}, {'lastname': 'Plateroti', 'firstname': 'Michelina', 'initials': 'M', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.\\nINSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}]}\n",
      "\n",
      "{'pubmed_id': '36183778', 'title': 'Integrated analysis of microRNA expression profiles and function network in mice testes after low dose lead exposure from early puberty.', 'keywords': ['CeRNA', 'Low dose lead', 'Puberty', 'Reproductive toxicology', 'miRNA'], 'journal': 'Toxicology and applied pharmacology', 'abstract': 'There is evidence suggesting the participation of non-coding RNAs in male reproductive dysfunction induced by lead, and the significance of microRNAs has been highlighted recently because of their essential roles in gene regulatory networks. To comprehensively understand the functions of miRNA and the regulatory networks, RNA sequencing was carried out to obtain miRNA expression profiles in mice testes exposed to low dose Pb for 90\\xa0days at the onset of puberty. In total, 44 differentially expressed miRNAs with 26 up-regulated and 18 down-regulated were identified between 200\\xa0mg/L Pb group and control group (p\\xa0<\\xa00.05). Enrichment analysis confirmed that the target genes of DE miRNAs might participate in the metabolism of testicular cells. Furthermore, a miRNA-mRNA co-expression network consisting of 19 miRNAs and 106 mRNAs and a competing endogenous RNA network of lncRNA-miRNA-mRNA including 179 genes were established. Finally, the expressions of 4 miRNAs (mmu-miR-451a, mmu-miR-133a-3p, mmu-miR-1a-3p and mmu-miR-486a-3p) and 4 mRNAs (Gramd1b, Tcf7l2, Mov10 and Srcin1) involved in regulatory networks were verified by RT-qPCR. In conclusion, our research might provide targets for the mechanism studies of miRNAs in reproductive toxicity of Pb.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': 'Copyright © 2022. Published by Elsevier Inc.', 'doi': '10.1016/j.taap.2022.116260', 'publication_date': datetime.date(2022, 10, 3), 'authors': [{'lastname': 'Ma', 'firstname': 'Haitao', 'initials': 'H', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Zhang', 'firstname': 'Haoran', 'initials': 'H', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Yu', 'firstname': 'Jun', 'initials': 'J', 'affiliation': 'Department of Preventive Medicine, School of Basic Medical Sciences, Hubei University of Science and Technology, Xianning 437100, Hubei Province, China.'}, {'lastname': 'Wang', 'firstname': 'Ziqiong', 'initials': 'Z', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Zeng', 'firstname': 'Xiangchao', 'initials': 'X', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Ye', 'firstname': 'Jingping', 'initials': 'J', 'affiliation': 'Department of Pediatrics, Renmin Hospital of Wuhan University, Wuhan 430071, Hubei Province, China. Electronic address: jpy7012@163.com.'}, {'lastname': 'Wang', 'firstname': 'Chunhong', 'initials': 'C', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China. Electronic address: wchunhong027@whu.edu.cn.'}]}\n",
      "\n",
      "{'pubmed_id': '36182916', 'title': 'Leveraging family history in genetic association analyses of binary traits.', 'keywords': ['Family history', 'GWAS', 'Meta-analysis'], 'journal': 'BMC genomics', 'abstract': \"Considering relatives' health history in logistic regression for case-control genome-wide association studies (CC-GWAS) may provide new information that increases accuracy and power to detect disease associated genetic variants. We conducted simulations and analyzed type 2 diabetes (T2D) data from the Framingham Heart Study (FHS) to compare two methods, liability threshold model conditional on both case-control status and family history (LT-FH) and Fam-meta, which incorporate family history into CC-GWAS.\\nIn our simulation scenario of trait with modest T2D heritability (h\\nOverall, LT-FH and Fam-meta had higher power than CC-GWAS in simulations, especially using phenotypes that were more prevalent in older age groups, and both methods detected known genetic variants with lower P-values in real data application, highlighting the benefits of including family history in genetic association studies.\", 'conclusions': None, 'methods': None, 'results': 'In our simulation scenario of trait with modest T2D heritability (h', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1186/s12864-022-08897-8\\n10.1089/cmb.2012.0024\\n10.1371/journal.pone.0051589\\n10.1016/j.ajhg.2013.02.011\\n10.1086/521580\\n10.1038/s41588-020-0613-6\\n10.1002/gepi.21785\\n10.1093/bioinformatics/btr341\\n10.1038/ng1732\\n10.1093/nar/gkv1340\\n10.1038/ng.2653\\n10.1038/s41467-018-08000-4\\n10.1161/HYPERTENSIONAHA.116.08581\\n10.1002/em.21827\\n10.1101/2020.06.28.171561\\n10.1038/s41588-018-0205-x\\n10.1371/journal.pone.0230815\\n10.1016/j.ajhg.2022.01.009\\n10.1038/s41431-021-00980-0\\n10.1016/j.ajhg.2022.03.001', 'publication_date': datetime.date(2022, 10, 2), 'authors': [{'lastname': 'Zhang', 'firstname': 'Yixin', 'initials': 'Y', 'affiliation': 'Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA. zhangyix@bu.edu.'}, {'lastname': 'Meigs', 'firstname': 'James B', 'initials': 'JB', 'affiliation': 'Division of General Internal Medicine, Massachusetts General Hospital, Boston, MA, USA.\\nDepartment of Medicine, Harvard Medical School, Boston, MA, USA.'}, {'lastname': 'Liu', 'firstname': 'Ching-Ti', 'initials': 'CT', 'affiliation': 'Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA.'}, {'lastname': 'Dupuis', 'firstname': 'Josée', 'initials': 'J', 'affiliation': 'Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA.\\nDepartment of Epidemiology, Biostatistics and Occupational Health, McGill University, Montréal, Québec, Canada.'}, {'lastname': 'Sarnowski', 'firstname': 'Chloé', 'initials': 'C', 'affiliation': 'Department of Epidemiology, Human Genetics, and Environmental Sciences, The University of Texas Health Science Center, School of Public Health, Houston, TX, USA.'}]}\n",
      "\n",
      "{'pubmed_id': '36174902', 'title': 'Genetic risk score is associated with T2DM and diabetes complications risks.', 'keywords': ['Complications', 'Diabetes', 'Gene score', 'Polymorphism'], 'journal': 'Gene', 'abstract': \"Type 2 diabetes mellitus (T2DM) is a prototypical complex disease with polygenic architecture playing an important role in determining susceptibility to develop the disease (and its complications) in subjects exposed to modifiable lifestyle factors. A current challenge is to quantify the degree of the individual's genetic risk using genetic risk scores (GRS) capturing the results of genome-wide association studies while incorporating possible ethnicity- or population-specific differences.\\nThis study included three groups of T2DM (T2DM-I, N\\xa0=\\xa01,032; T2DM-II, N\\xa0=\\xa0353; and T2DM-III, N\\xa0=\\xa0399) patients and 2,481 diabetes-free subjects. The status of the microvascular and macrovascular diabetes complications were known for the T2DM-I patients. Overall, 21 single nucleotide polymorphisms (SNPs) were analyzed, and selected subsets were used to determine the GRS (both weighted - wGRS and unweighted - uGRS) for T2DM risk predictions (6 SNPs) and for predicting the risks of complications (7 SNPs).\\nThe strongest T2DM markers (P\\xa0<\\xa00.0001) were within the genes for TCF7L2 (transcription factor 7-like 2), FTO (fat mass and obesity associated protein) and ARAP1 (ankyrin repeat and PH domain 1). The T2DM-I subjects with uGRS values greater (Odds Ratio, 95\\xa0% Confidence Interval) than six had at least twice (2.00, 1.72-2.32) the risk of T2DM development (P\\xa0<\\xa00.0001), and these results were confirmed in the independent groups (T2DM-II 1.82, 1.45-2.27; T2DM-III 2.63, 2.11-3.27). The wGRS (>0.6) further improved (P\\xa0<\\xa00.000001) the risk estimations for all three T2DM groups. The uGRS was also a significant predictor of neuropathy (P\\xa0<\\xa00.0001), nephropathy (P\\xa0<\\xa00.005) and leg ischemia (P\\xa0<\\xa00.0005).\\nIf carefully selected and specified, GRS, both weighted and unweighted, could be significant predictors of T2DM development, as well as the diabetes complications development.\", 'conclusions': None, 'methods': None, 'results': 'The strongest T2DM markers (P\\xa0<\\xa00.0001) were within the genes for TCF7L2 (transcription factor 7-like 2), FTO (fat mass and obesity associated protein) and ARAP1 (ankyrin repeat and PH domain 1). The T2DM-I subjects with uGRS values greater (Odds Ratio, 95\\xa0% Confidence Interval) than six had at least twice (2.00, 1.72-2.32) the risk of T2DM development (P\\xa0<\\xa00.0001), and these results were confirmed in the independent groups (T2DM-II 1.82, 1.45-2.27; T2DM-III 2.63, 2.11-3.27). The wGRS (>0.6) further improved (P\\xa0<\\xa00.000001) the risk estimations for all three T2DM groups. The uGRS was also a significant predictor of neuropathy (P\\xa0<\\xa00.0001), nephropathy (P\\xa0<\\xa00.005) and leg ischemia (P\\xa0<\\xa00.0005).', 'copyrights': 'Copyright © 2022. Published by Elsevier B.V.', 'doi': '10.1016/j.gene.2022.146921', 'publication_date': datetime.date(2022, 9, 30), 'authors': [{'lastname': 'Hubacek', 'firstname': 'Jaroslav A', 'initials': 'JA', 'affiliation': 'Experimental Medicine Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic; 3rd Department of Internal Medicine, 1(st) Faculty of Medicine, Charles University, Prague, Czech Republic. Electronic address: jahb@ikem.cz.'}, {'lastname': 'Dlouha', 'firstname': 'Lucie', 'initials': 'L', 'affiliation': 'Experimental Medicine Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic; Department of Anthropology and Human Genetics, Faculty of Science, Charles University, Prague, Czech Republic.'}, {'lastname': 'Adamkova', 'firstname': 'Vera', 'initials': 'V', 'affiliation': 'Department of Preventive Cardiology, Institute for Clinical and Experimental Medicine, Prague, Czech Republic; Czech Technical University of Prague, Faculty of Biomedical Engineering, Prague, Czech Republic.'}, {'lastname': 'Dlouha', 'firstname': 'Dana', 'initials': 'D', 'affiliation': 'Experimental Medicine Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}, {'lastname': 'Pacal', 'firstname': 'Lukas', 'initials': 'L', 'affiliation': 'Department of Pathophysiology, Faculty of Medicine, Masaryk University, Brno, Czech Republic.'}, {'lastname': 'Kankova', 'firstname': 'Katerina', 'initials': 'K', 'affiliation': 'Department of Pathophysiology, Faculty of Medicine, Masaryk University, Brno, Czech Republic.'}, {'lastname': 'Galuska', 'firstname': 'David', 'initials': 'D', 'affiliation': 'Department of Pathophysiology, Faculty of Medicine, Masaryk University, Brno, Czech Republic.'}, {'lastname': 'Lanska', 'firstname': 'Vera', 'initials': 'V', 'affiliation': 'Statistical Unit, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}, {'lastname': 'Veleba', 'firstname': 'Jiri', 'initials': 'J', 'affiliation': 'Diabetes Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}, {'lastname': 'Pelikanova', 'firstname': 'Terezie', 'initials': 'T', 'affiliation': 'Diabetes Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}]}\n",
      "\n",
      "{'pubmed_id': '36163520', 'title': 'Development and validation of a model that predicts the risk of diabetic retinopathy in type 2 diabetes mellitus patients.', 'keywords': ['Diabetic retinopathy', 'Nomogram', 'Prediction model', 'Risk factors', 'Type 2 diabetes mellitus'], 'journal': 'Acta diabetologica', 'abstract': 'Diabetic retinopathy is the leading cause of blindness in people with type 2 diabetes. To enable primary care physicians to identify high-risk type 2 diabetic patients with diabetic retinopathy at an early stage, we developed a nomogram model to predict the risk of developing diabetic retinopathy in the Xinjiang type 2 diabetic population.\\nIn a retrospective study, we collected data on 834 patients with type 2 diabetes through an electronic medical record system. Stepwise regression was used to filter variables. Logistic regression was applied to build a nomogram prediction model and further validated in the training set. The c-index, forest plot, calibration plot, and clinical decision curve analysis were used to comprehensively validate the model and evaluate its accuracy and clinical validity.\\nFour predictors were selected to establish the final model: hypertension, blood urea nitrogen, duration of diabetes, and diabetic peripheral neuropathy. The model displayed medium predictive power with a C-index of 0.781(95%CI:0.741-0.822) in the training set and 0.865(95%CI:0.807-0.923)in the validation set. The calibration curve of the DR probability shows that the predicted results of the nomogram are in good agreement with the actual results. Decision curve analysis demonstrated that the novel nomogram was clinically valuable.\\nThe nomogram of the risk of developing diabetic nephropathy contains 4 characteristics. that can help primary care physicians quickly identify individuals at high risk of developing DR in patients with type 2 diabetes, to intervene as soon as possible.', 'conclusions': None, 'methods': None, 'results': 'Four predictors were selected to establish the final model: hypertension, blood urea nitrogen, duration of diabetes, and diabetic peripheral neuropathy. The model displayed medium predictive power with a C-index of 0.781(95%CI:0.741-0.822) in the training set and 0.865(95%CI:0.807-0.923)in the validation set. The calibration curve of the DR probability shows that the predicted results of the nomogram are in good agreement with the actual results. Decision curve analysis demonstrated that the novel nomogram was clinically valuable.', 'copyrights': '© 2022. Springer-Verlag Italia S.r.l., part of Springer Nature.', 'doi': '10.1007/s00592-022-01973-1\\n10.1016/j.dsx.2021.102239\\n10.3390/ijerph18147346\\n10.3346/jkms.2016.31.S2.S146\\n10.2337/dc11-1909\\n10.1001/archopht.122.4.552\\n10.1002/dmrr.1307\\n10.1097/IAE.0b013e31821c4252\\n10.1007/s00592-020-01540-6\\n10.1001/archopht.126.12.1740\\n10.1016/j.ophtha.2010.08.003\\n10.1136/bjophthalmol-2017-310316\\n10.1002/(SICI)1096-9136(199807)15:7<539::AID-DIA668>3.0.CO;2-S\\n10.1186/s12882-017-0776-2\\n10.1016/j.ebiom.2018.08.009\\n10.1097/MD.0000000000019236\\n10.1016/j.jdiacomp.2013.12.008\\n10.1016/j.ophtha.2006.10.023\\n10.1136/bmjopen-2018-023586\\n10.4103/ijo.IJO_1144_21\\n10.1371/journal.pone.0097928\\n10.1016/j.jdiacomp.2013.12.008\\n10.1016/j.atherosclerosis.2012.10.034\\n10.1002/mnfr.201900934\\n10.1016/j.gtc.2016.02.004\\n10.4103/ijo.IJO_1279_21\\n10.1507/endocrj.EJ20-0794\\n10.1172/JCI86181\\n10.2147/DMSO.S244061', 'publication_date': datetime.date(2022, 9, 28), 'authors': [{'lastname': 'Yang', 'firstname': 'Jing', 'initials': 'J', 'affiliation': 'State Key Laboratory of Pathogenesis, Prevention andTreatment of High Incidence Diseases in Central Asia, Department of Endocrinology, The First Affiliated Hospital of Xinjiang Medical University, Urumqi, 830017, China.'}, {'lastname': 'Jiang', 'firstname': 'Sheng', 'initials': 'S', 'affiliation': 'State Key Laboratory of Pathogenesis, Prevention andTreatment of High Incidence Diseases in Central Asia, Department of Endocrinology, The First Affiliated Hospital of Xinjiang Medical University, Urumqi, 830017, China. xjjiangsheng@126.com.'}]}\n",
      "\n",
      "{'pubmed_id': '36155628', 'title': 'Effect of TCF7L2 on the relationship between lifestyle factors and glycemic parameters: a systematic review.', 'keywords': ['Dietary factors', 'Glycemic parameters', 'Lifestyle factors', 'TCF7L2'], 'journal': 'Nutrition journal', 'abstract': 'Among candidate\\xa0genes related to type 2 diabetes (T2DM), one of the strongest genes is Transcription factor 7 like 2 (TCF7L2), regarding the Genome-Wide Association Studies. We aimed to conduct a systematic review of the literature on the modification effect of TCF7L2 on the relation between glycemic parameters and lifestyle factors.\\nA systematic literature search was done for relevant publications using electronic databases, including PubMed, EMBASE, Scopus, and Web of Science, from January 1, 2000, to November 2, 2021.\\nThirty-eight studies (16 observational studies, six meal test trials, and 16 randomized controlled trials (RCTs)) were included. Most observational studies had been conducted on participants with non-diabetes showing that TCF7L2 modified the association between diet (fatty acids and fiber) and insulin resistance. In addition, findings from meal test trials showed that, compared to non-risk-allele carriers, consumption of meals with different percentages of total dietary fat in healthy risk-allele carriers increased glucose concentrations and impaired insulin sensitivity. However, ten RCTs, with intervention periods of less than ten weeks and more than one year, showed that TCF7L2 did not modify glycemic parameters in response to a dietary intervention involving different macronutrients. However, two weight loss dietary RCTs with more than 1-year duration showed that serum glucose and insulin levels decreased and insulin resistance improved in non-risk allele subjects with overweight/obesity. Regarding artichoke extract supplementation (ALE), two RCTs observed that ALE supplementation significantly decreased insulin concentration and improved insulin resistance in the TT genotype of the rs7903146 variant of TCF7L2. In addition, four studies suggested that physical activity levels and smoking status modified the association between TCF7L2 and glycemic parameters. However, three studies observed no effect of TCF7L2 on glycemic parameters in participants with different levels of physical activity and smoking status.\\nThe modification effects of TCF7L2 on the relation between the lifestyle factors (diet, physical activity, and smoking status) and glycemic parameters were contradictory.\\nCRD42020196327.', 'conclusions': 'The modification effects of TCF7L2 on the relation between the lifestyle factors (diet, physical activity, and smoking status) and glycemic parameters were contradictory.', 'methods': None, 'results': 'Thirty-eight studies (16 observational studies, six meal test trials, and 16 randomized controlled trials (RCTs)) were included. Most observational studies had been conducted on participants with non-diabetes showing that TCF7L2 modified the association between diet (fatty acids and fiber) and insulin resistance. In addition, findings from meal test trials showed that, compared to non-risk-allele carriers, consumption of meals with different percentages of total dietary fat in healthy risk-allele carriers increased glucose concentrations and impaired insulin sensitivity. However, ten RCTs, with intervention periods of less than ten weeks and more than one year, showed that TCF7L2 did not modify glycemic parameters in response to a dietary intervention involving different macronutrients. However, two weight loss dietary RCTs with more than 1-year duration showed that serum glucose and insulin levels decreased and insulin resistance improved in non-risk allele subjects with overweight/obesity. Regarding artichoke extract supplementation (ALE), two RCTs observed that ALE supplementation significantly decreased insulin concentration and improved insulin resistance in the TT genotype of the rs7903146 variant of TCF7L2. In addition, four studies suggested that physical activity levels and smoking status modified the association between TCF7L2 and glycemic parameters. However, three studies observed no effect of TCF7L2 on glycemic parameters in participants with different levels of physical activity and smoking status.', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1186/s12937-022-00813-w\\n10.1016/j.diabres.2019.107843\\n10.1177/2047487319878371\\n10.1016/j.cmet.2008.08.006\\n10.1038/ng1732\\n10.1038/79216\\n10.1038/nature05616\\n10.1038/ng.120\\n10.2337/dc17-0290\\n10.1007/s11010-021-04086-5\\n10.1080/07435800.2019.1573827\\n10.2337/db09-0918\\n10.2337/db09-1169\\n10.1016/j.numecd.2006.04.001\\n10.1111/obr.12921\\n10.1016/j.gde.2018.02.001\\n10.1007/s11892-016-0758-y\\n10.1055/s-0031-1285832\\n10.1007/s00109-006-0134-5\\n10.1146/annurev-nutr-071811-150648\\n10.2337/dc12-2211\\n10.3390/ijms18061188\\n10.3945/ajcn.116.150094\\n10.3389/fgene.2020.590369\\n10.1097/CM9.0000000000000617\\n10.1038/ijo.2008.146\\n10.3945/ajcn.112.038125\\n10.3945/ajcn.2009.27947\\n10.1371/journal.pone.0043390\\n10.2337/db11-0973\\n10.1371/journal.pone.0230815\\n10.1038/oby.2007.77\\n10.1056/NEJMoa062418\\n10.1371/journal.pone.0021518\\n10.1136/bmj.n71\\n10.1136/bmj.l4898\\n10.1007/s00125-010-1940-4\\n10.1210/jc.2015-1172\\n10.1186/s13098-018-0338-1\\n10.1007/s00125-009-1307-x\\n10.1016/j.diabet.2017.05.001\\n10.1017/S0007114516002798\\n10.3945/jn.111.147447\\n10.1002/ptr.5951\\n10.3945/ajcn.2009.28379\\n10.2337/db09-1050\\n10.1007/s12263-012-0284-8\\n10.1016/j.joim.2018.05.006\\n10.1007/s00125-009-1630-2\\n10.2337/dc10-1150\\n10.1016/j.atherosclerosis.2010.10.027\\n10.1007/s00125-012-2634-x\\n10.2337/dc13-0955\\n10.1186/s12863-014-0134-3\\n10.1016/j.metabol.2016.11.018\\n10.1371/journal.pone.0188382\\n10.3390/nu13061936\\n10.1016/j.jnutbio.2010.11.020\\n10.2337/db08-1623\\n10.2337/db20-0156\\n10.1002/gepi.21991\\n10.1016/j.trsl.2021.07.001\\n10.1038/s41467-020-20500-w\\n10.1172/JCI89444\\n10.1038/nrendo.2014.175', 'publication_date': datetime.date(2022, 9, 27), 'authors': [{'lastname': 'Hosseinpour-Niazi', 'firstname': 'Somayeh', 'initials': 'S', 'affiliation': 'Nutrition and Endocrine Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Mirmiran', 'firstname': 'Parvin', 'initials': 'P', 'affiliation': 'Nutrition and Endocrine Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran. mirmiran@endocrine.ac.ir.'}, {'lastname': 'Hosseini', 'firstname': 'Shabnam', 'initials': 'S', 'affiliation': 'School of Human Nutrition, Faculty of Agricultural and Environmental Sciences, McGill University, Montreal, Quebec, Canada.'}, {'lastname': 'Hadaegh', 'firstname': 'Farzad', 'initials': 'F', 'affiliation': 'Prevention of Metabolic Disorders Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Ainy', 'firstname': 'Elaheh', 'initials': 'E', 'affiliation': 'Department of Vice Chancellor Research Affairs, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Daneshpour', 'firstname': 'Maryam S', 'initials': 'MS', 'affiliation': 'Cellular and Molecular Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Azizi', 'firstname': 'Fereidoun', 'initials': 'F', 'affiliation': 'Endocrine Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}]}\n",
      "\n",
      "{'pubmed_id': '36479247', 'title': 'Genomic profiling and network-level understanding uncover the potential genes and the pathways in hepatocellular carcinoma.', 'keywords': ['HCV and HCC', 'biomarkers', 'co-expression', 'gene expression/mutational profiling', 'network-level understanding'], 'journal': 'Frontiers in genetics', 'abstract': 'Data integration with phenotypes such as gene expression, pathways or function, and protein-protein interactions data has proven to be a highly promising technique for improving human complex diseases, particularly cancer patient outcome prediction. Hepatocellular carcinoma is one of the most prevalent cancers, and the most common cause is chronic HBV and HCV infection, which is linked to the majority of cases, and HBV and HCV play a role in multistep carcinogenesis progression. We examined the list of known hepatocellular carcinoma biomarkers with the publicly available expression profile dataset of hepatocellular carcinoma infected with HCV from day 1 to day 10 in this study. The study covers an overexpression pattern for the selected biomarkers in clinical hepatocellular carcinoma patients, a combined investigation of these biomarkers with the gathered temporal dataset, temporal expression profiling changes, and temporal pathway enrichment following HCV infection. Following a temporal analysis, it was discovered that the early stages of HCV infection tend to be more harmful in terms of expression shifting patterns, and that there is no significant change after that, followed by a set of genes that are consistently altered. PI3K, cAMP, TGF, TNF, Rap1, NF-kB, Apoptosis, Longevity regulating pathway, signaling pathways regulating pluripotency of stem cells, Cytokine-cytokine receptor interaction, p53 signaling, Wnt signaling, Toll-like receptor signaling, and Hippo signaling pathways are just a few of the most commonly enriched pathways. The majority of these pathways are well-known for their roles in the immune system, infection and inflammation, and human illnesses like cancer. We also find that ADCY8, MYC, PTK2, CTNNB1, TP53, RB1, PRKCA, TCF7L2, PAK1, ITPR2, CYP3A4, UGT1A6, GCK, and FGFR2/3 appear to be among the prominent genes based on the networks of genes and pathways based on the copy number alterations, mutations, and structural variants study.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': 'Copyright © 2022 El-Kafrawy, El-Daly, Bajrai, Alandijany, Faizo, Mobashir, Ahmed, Ahmed, Alam, Jeet, Kamal, Anwer, Khan, Tashkandi, Rizvi and Azhar.', 'doi': '10.3389/fgene.2022.880440\\n10.1016/j.ajpath.2012.08.005\\n10.1186/1479-5876-10-41\\n10.3389/fonc.2022.914032\\n10.1101/gr.087528.108\\n10.3390/ijms231911024\\n10.1016/j.dnarep.2019.06.005\\n10.1038/s41598-021-98031-7\\n10.3389/fcimb.2021.707905\\n10.1038/nrg1272\\n10.1093/nar/gkp942\\n10.1074/jbc.M010271200\\n10.1093/sysbio/syu081\\n10.1038/nrg3424\\n10.1245/s10434-013-2985-7\\n10.1038/nature07385\\n10.1098/rsif.2011.0390\\n10.1038/nbt.2203\\n10.1038/nature08987\\n10.1038/ng.3221\\n10.1038/msb4100200\\n10.1093/annonc/mdt419\\n10.1056/NEJMra1001683\\n10.3389/fendo.2021.641888\\n10.1038/nature06758\\n10.1002/wsbm.134\\n10.1093/nar/29.2.449\\n10.1038/nrc1299\\n10.1038/nrg3813\\n10.1038/nmeth.2562\\n10.1182/blood-2008-05-158865\\n10.21873/anticanres.11078\\n10.1038/nrg.2016.87\\n10.3390/biomedicines10051076\\n10.1016/j.dnarep.2007.11.007\\n10.1002/ijc.22605\\n10.1186/s13059-014-0436-9\\n10.1146/annurev-immunol-032712-100001\\n10.36013/jimsa.v1i1.3\\n10.1093/nar/gkm882\\n10.1093/nar/gkp896\\n10.1038/s41598-022-11143-6\\n10.1038/s41598-022-11143-6\\n10.1371/journal.pone.0062113\\n10.3389/fonc.2012.00202\\n10.1038/nature11412\\n10.1016/j.imu.2020.100422\\n10.1016/j.imu.2020.100422\\n10.1073/pnas.0304146101\\n10.1186/s13059-014-0473-4\\n10.1038/ng1956\\n10.1038/nrclinonc.2015.103\\n10.1053/j.gastro.2019.04.003\\n10.1053/j.gastro.2019.04.003\\n10.1093/nar/gkv1194\\n10.1038/nbt.3027\\n10.1016/j.ajpath.2013.02.026\\n10.1126/scitranslmed.aaa0763\\n10.1073/pnas.0609635104\\n10.1186/s13059-014-0439-6\\n10.3390/diseases3040294\\n10.1093/hmg/ddq079\\n10.1093/bioinformatics/bts448\\n10.1038/nrc3816\\n10.1146/annurev.med.45.1.297\\n10.1016/j.diabres.2018.03.002\\n10.1038/nrclinonc.2014.5\\n10.1101/gr.1239303\\n10.1186/1475-2840-11-15\\n10.1073/pnas.0506580102\\n10.1093/annonc/mdu005\\n10.1016/j.ccr.2010.05.026\\n10.1146/annurev.biophys.33.110502.132654\\n10.1074/mcp.M500279-MCP200\\n10.1126/science.aax9198\\n10.1126/science.aan2507\\n10.1146/annurev-pharmtox-010611-134628\\n10.1016/j.ccr.2005.10.001\\n10.2174/1381612826666201106093148\\n10.1038/nrclinonc.2014.6\\n10.1126/science.aaw0726\\n10.1038/nrg3871\\n10.1186/s40246-015-0056-9', 'publication_date': datetime.date(2022, 12, 9), 'authors': [{'lastname': 'El-Kafrawy', 'firstname': 'Sherif A', 'initials': 'SA', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'El-Daly', 'firstname': 'Mai M', 'initials': 'MM', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Bajrai', 'firstname': 'Leena H', 'initials': 'LH', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nBiochemistry Department, Faculty of Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Alandijany', 'firstname': 'Thamir A', 'initials': 'TA', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Faizo', 'firstname': 'Arwa A', 'initials': 'AA', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Mobashir', 'firstname': 'Mohammad', 'initials': 'M', 'affiliation': 'Department of Microbiology, Tumor and Cell Biology (MTC), Karolinska Institute, Stockholm, Sweden.\\nGenome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Ahmed', 'firstname': 'Sunbul S', 'initials': 'SS', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Ahmed', 'firstname': 'Sarfraz', 'initials': 'S', 'affiliation': 'Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Alam', 'firstname': 'Shoaib', 'initials': 'S', 'affiliation': 'Department of Biotechnology, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Jeet', 'firstname': 'Raja', 'initials': 'R', 'affiliation': 'Botany Department, Ganesh Dutt College, Begusarai, Bihar, India.'}, {'lastname': 'Kamal', 'firstname': 'Mohammad Amjad', 'initials': 'MA', 'affiliation': 'Institutes for Systems Genetics, Frontiers Science Center for Disease-related Molecular Network, West China Hospital, Sichuan University, Chengdu, China.\\nKing Fahd Medical Research Center, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Pharmacy, Faculty of Allied Health Sciences, Daffodil International University, Dhaka, Bangladesh.\\nEnzymoics, Hebersham, NSW, Australia.\\nNovel Global Community Educational Foundation, Hebersham, NSW, Australia.'}, {'lastname': 'Anwer', 'firstname': 'Syed Tauqeer', 'initials': 'ST', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Khan', 'firstname': 'Bushra', 'initials': 'B', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Tashkandi', 'firstname': 'Manal', 'initials': 'M', 'affiliation': 'Department of Biochemistry, College of Science, University of Jeddah, Jeddah, Saudi Arabia.'}, {'lastname': 'Rizvi', 'firstname': 'Moshahid A', 'initials': 'MA', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Azhar', 'firstname': 'Esam Ibraheem', 'initials': 'EI', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}]}\n",
      "\n",
      "{'pubmed_id': '36454469', 'title': 'Genomic Editing and Diabetes.', 'keywords': ['CRISPR/Cas9', 'Diabetes', 'Genetic editing', 'Long noncoding RNA', 'MicroRNA'], 'journal': 'Advances in experimental medicine and biology', 'abstract': 'The diabetes types and its complications have varying developmental and metabolic pathways. There is an interplay of nongenetic and genetic components in pathogenesis of diabetes and its complications. There are several established genes such as ABCC8, TCF7L2, SLC2A2, and CAPN10 which are known to influence blood insulin and glucose levels. Current management of diabetes types may include lifetime burdensome use of insulin, insulin sensitizers, insulin secretagogues, etc. There has been increasing interest in improving genetic editing tools such as CRISPR/Cas9 and using genetically edited stem cells to alter diabetes disease course or possibly cure it. Current research on microRNAs and long noncoding RNAs may provide insights into the pathways involved in development of diabetes and its complications. Consequently, developing further understanding of genetics and its messenger pathways in diabetes would enhance our ability to develop precise and accurate genetic editing tools which can translate into clinically useful therapeutics.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2023. The Author(s), under exclusive license to Springer Nature Singapore Pte Ltd.', 'doi': '10.1007/978-981-19-5642-3_14\\n10.1016/S0140-6736(16)00618-8\\n10.34172/hpp.2020.18\\n10.1016/j.ymgme.2003.10.005\\n10.2337/db17-0318\\n10.1016/j.ijbiomac.2020.04.240\\n10.1186/s12881-018-0553-5\\n10.1055/a-1008-9223\\n10.1007/s42000-018-0047-z\\n10.1016/j.molmet.2019.03.003\\n10.1172/jci.insight.136136\\n10.1002/humu.23995\\n10.1007/978-3-030-25905-1_18\\n10.1111/1753-0407.13144\\n10.1007/s00125-018-4759-z\\n10.1038/ng.3632\\n10.1530/JME-19-0253\\n10.1155/2017/3937893\\n10.1007/s00404-016-4140-8\\n10.1007/5584_2018_305\\n10.1080/21655979.2016.1189039\\n10.1371/journal.pbio.3000603\\n10.1016/j.bbrc.2019.09.001\\n10.1186/s12933-019-0824-2\\n10.1530/EJE-19-0206\\n10.1161/CIRCRESAHA.119.314898\\n10.3389/fimmu.2018.02711\\n10.1080/09674845.2017.1402404\\n10.1038/s41467-019-13587-3\\n10.1007/s11033-018-4429-8\\n10.1073/pnas.1914353117\\n10.3390/ijms21114020\\n10.1016/j.atherosclerosis.2018.08.031\\n10.7150/ijms.34200', 'publication_date': datetime.date(2022, 12, 2), 'authors': [{'lastname': 'Shah', 'firstname': 'Parth', 'initials': 'P', 'affiliation': 'ObvioHealth, New York, NY, USA. prshah06@gmail.com.'}]}\n",
      "\n",
      "{'pubmed_id': '36399471', 'title': 'Bioinformatic analysis and clinical diagnostic value of hsa_circ_0004099 in acute ischemic stroke.', 'keywords': [], 'journal': 'PloS one', 'abstract': 'This study investigates the expression and effect of hsa_circ_0004099 in acute ischemic stroke (AIS). We conducted a case-controlled study that included 40 patients with AIS within 24 hours and 40 healthy subjects during the same period as a control group. Differentially expressed circular RNAs (circRNAs) were obtained using GEO2R, and the expression of hsa_circ_0004099 was verified using RT-PCR. Correlation analysis of the National Institutes of Health Stroke Scale (NIHSS) disease severity score and ischemic time with hsa_circ_0004099 expression levels was also performed. The receiver operating characteristic (ROC) curve of hsa_circ_0004099 was constructed, and bioinformatic analysis of hsa_circ_0004099 was performed. NIHSS scores negatively correlated with hsa_circ_0004099 levels (P<0.001, r = -0.7053), whereas infarct time was negatively correlated with hsa_circ_0004099 levels (P<0.001, r = -0.5130); hsa_circ_0004099 could benefit clinical diagnosis (area under the curve [AUC]: 0.923 [95% confidence interval [CI]: 0.8680-0.9904]). Kyoto encyclopedia of genes and genomes (KEGG) analysis showed that hsa_circ_0004099 was enriched in several cancer pathways, which were collectively enriched in four genes namely TCF7L2, NRAS, CTNNB1, and KRAS. Eight core proteins were screened using a protein-protein interaction (PPI) network namely SMAD4, HIF1A, CTNNB1, CDKN1B, CDK6, FOXO3, KRAS, and NRAS. hsa_circ_0004099 is a potential clinical diagnostic marker. In addition, the possible role of hsa_circ_0004099 in the pathogenesis of AIS was analyzed using bioinformatics, which provided a new potential molecular target for AIS treatment.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': 'Copyright: © 2022 Zheng et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.', 'doi': '10.1371/journal.pone.0277832\\n10.1016/S0140-6736(18)31874-9\\n10.1001/jama.2015.3058\\n10.29271/jcpsp.2019.09.898\\n10.6061/clinics/2021/e2958\\n10.1002/wrna.1294\\n10.2174/1567205017666201111114928\\n10.1007/s00018-017-2688-5\\n10.1016/j.gene.2019.03.053\\n10.18632/aging.103424\\n10.1002/jcp.29165\\n10.1038/s41598-020-66990-y\\n10.3389/fnins.2020.00081\\n10.1016/j.ajhg.2016.10.006\\n10.1016/j.atherosclerosis.2018.01.028\\n10.1161/JAHA.114.001119\\n10.1371/journal.pone.0141214\\n10.4037/aacnacc2018483\\n10.1002/jcla.22954\\n10.7754/Clin.Lab.2020.191266\\n10.1161/STROKEAHA.119.027348\\n10.1016/j.brainres.2020.146835\\n10.3892/mmr.2019.10317\\n10.1007/s12975-020-00851-8\\n10.1007/s10528-022-10225-4\\n10.1155/2021/5585206\\n10.1093/jnen/nlab106\\n10.1016/j.gene.2020.145022\\n10.1159/000514543\\n10.1590/1806-9282.20210509\\n10.1016/j.diabres.2013.12.051\\n10.1007/s00432-021-03842-w\\n10.1002/ana.26059\\n10.1155/2021/6745282\\n10.1002/ana.23883\\n10.1186/s12885-022-09186-z\\n10.1002/acn3.51154\\n10.1007/s11010-022-04428-x\\n10.3389/fnagi.2022.830494\\n10.1155/2020/2507074\\n10.17116/jnevro201611612210-13\\n10.3389/fonc.2021.757650\\n10.1161/STROKEAHA.111.000504\\n10.1016/j.jns.2015.05.036\\n10.3390/metabo12030250\\n10.1093/gerona/glac016\\n10.3390/ijms222011131\\n10.1080/00207454.2018.1564290\\n10.1016/j.gep.2021.119218', 'publication_date': datetime.date(2022, 11, 19), 'authors': [{'lastname': 'Zheng', 'firstname': 'Jiqing', 'initials': 'J', 'affiliation': 'Deparment of Rehabilitation, The Second Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi, China.'}, {'lastname': 'Luo', 'firstname': 'Shuiming', 'initials': 'S', 'affiliation': 'Deparment of Rehabilitation, The Second Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi, China.'}, {'lastname': 'Long', 'firstname': 'Yaobin', 'initials': 'Y', 'affiliation': 'Deparment of Rehabilitation, The Second Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi, China.'}]}\n",
      "\n",
      "{'pubmed_id': '36382598', 'title': 'Identification of odontogenic ameloblast associated as a novel target gene of the Wnt/β-catenin signaling pathway.', 'keywords': ['ODAM', 'Wnt signaling', 'cell cycle', 'colorectal cancer', 'hepatoblastoma'], 'journal': 'Cancer science', 'abstract': 'The Wnt/β-catenin signaling pathway plays a key role in development and carcinogenesis. Although some target genes of this signaling have been identified in various tissues and neoplasms, the comprehensive understanding of the target genes and their roles in the development of human cancer, including hepatoma and colorectal cancer remain to be fully elucidated. In this study, we searched for genes regulated by the Wnt signaling in liver cancer using HuH-7 hepatoma cells. A comparison of the expression profiles between cells expressing an active form of mutant β-catenin and cells expressing enhanced green fluorescent protein (EGFP) identified seven genes upregulated by the mutant β-catenin gene (CTNNB1). Among the seven genes, we focused in this study on ODAM, odontogenic,\\xa0ameloblast associated, as a novel target gene. Interestingly, its expression was frequently upregulated in hepatocellular carcinoma, colorectal adenocarcinoma, and hepatoblastoma. We additionally identified a distant enhancer region that was associated with the β-catenin/TCF7L2 complex. Further analyses revealed that ODAM plays an important role in the regulation of the cell cycle, DNA synthesis, and cell proliferation. These data may be useful for clarification of the main molecular mechanism(s) underlying these cancers.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2022 The Authors. Cancer Science published by John Wiley & Sons Australia, Ltd on behalf of Japanese Cancer Association.', 'doi': '10.1111/cas.15657', 'publication_date': datetime.date(2022, 11, 17), 'authors': [{'lastname': 'Yamaguchi', 'firstname': 'Kiyoshi', 'initials': 'K', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Horie', 'firstname': 'Chiaki', 'initials': 'C', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Takane', 'firstname': 'Kiyoko', 'initials': 'K', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Ikenoue', 'firstname': 'Tsuneo', 'initials': 'T', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Nakagawa', 'firstname': 'Saya', 'initials': 'S', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Isobe', 'firstname': 'Yumiko', 'initials': 'Y', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Ota', 'firstname': 'Yasunori', 'initials': 'Y', 'affiliation': 'Department of Pathology, Research Hospital, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Ushiku', 'firstname': 'Tetsuo', 'initials': 'T', 'affiliation': 'Department of Pathology, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Tanaka', 'firstname': 'Mariko', 'initials': 'M', 'affiliation': 'Department of Pathology, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Fujishiro', 'firstname': 'Jun', 'initials': 'J', 'affiliation': 'Department of Pediatric Surgery, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Hoshino', 'firstname': 'Noriko', 'initials': 'N', 'affiliation': 'Department of Pediatric Surgery, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Arisue', 'firstname': 'Atsuhiro', 'initials': 'A', 'affiliation': 'Department of Surgery, Iwate Medical University School of Medicine, Yahaba, Japan.'}, {'lastname': 'Nishizuka', 'firstname': 'Satoshi', 'initials': 'S', 'affiliation': 'Division of Biomedical Research and Development, Iwate Medical University Institute for Biomedical Sciences, Yahaba, Japan.'}, {'lastname': 'Aikou', 'firstname': 'Susumu', 'initials': 'S', 'affiliation': 'Division of Frontier Surgery, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Shida', 'firstname': 'Dai', 'initials': 'D', 'affiliation': 'Division of Frontier Surgery, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Furukawa', 'firstname': 'Yoichi', 'initials': 'Y', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}]}\n",
      "\n",
      "{'pubmed_id': '36382190', 'title': 'Advances in multi-omics study of biomarkers of glycolipid metabolism disorder.', 'keywords': ['1,5-AG, 1,5-anhydroglucitol', '2-AAA, 2-aminoadipic acid', '2-DGE, two-dimensional gel electrophoresis', 'ABCC8, ATP binding cassette subfamily C member 8', 'ADA, American Diabetes Association', 'AhR, aromatic hydrocarbon receptor', 'BA, bile acid', 'BCAA, branched-chain amino acid', 'BCKAs, branched chain keto acids', 'Biomarkers', 'CE, cholesterol ester', 'CFL1, cofilin-1', 'CRP, C reactive protein', 'CYP450, cytochrome P450', 'DAG, diacylglycerol', 'DPP-4, dipeptyl peptidase 4', 'EASD, European Association for the Study of Diabetes', 'FA, fatty acid', 'FFA, free fatty acid', 'FMT, fecal microbiota transplantation', 'FTO, fat mass and obesity-associated', 'GAS5, growth arrest-specific transcript 5', 'GC–MS/MS, gas chromatography-tandem mass spectrometry', 'GLP-1, glucagon-like peptide 1', 'GLP-1R, glucagon-like peptide 1 receptor', 'GSIS, glucose-stimulated insulin secretion', 'GWAS, genome-wide association study', 'Glycolipid metabolism disorder', 'HGF, hepatocyte growth factor', 'HMG-CoA, hydroxymethylglutaryl-coenzyme A', 'HMGCR, 3-hydroxy-methylglutaryl coenzyme A reductase', 'HPLC, high performance liquid chromatography', 'HbA1C, glycylated hemoglobin', 'IL-1ra, interleukin-1 receptor antagonist', 'IMP, imidazole propionate', 'IR, insulin resistance', 'JNK, c-Jun-N-terminal-kinase', 'KCNJ11, potassium inwardly rectifying channel subfamily J member 11', 'L-GPC, linoleoyl-glycerophosphocholine', 'LC-MS/MS, liquid chromatography-tandem mass spectrometry', 'LPC, lysophosphatidylcholine', 'LPE, lysophosphatidylethanolamines', 'LPS, lipopolysaccharide', 'MAPK, mitogen-activated protein kinase', 'MASP, mannose-binding lectin-associated serine protease 1', 'MATE1, multidrug and toxin extrusion protein 1', 'MDR1, multidrug resistance mutation 1', 'MGT, magnesium transporter', 'MS, Mass Spectrometry', 'Mechanism study', 'Multi-omics', 'NK, natural killer', 'NMR, nuclear magnetic resonance', 'OCT, organic cationic transporter', 'OGTT, oral glucose tolerance test', 'PC-PL, phosphatidylcholine-plasmalogen', 'PE, phosphatidylethanolamines', 'PNPLA3, patatin-like phospholipase domain-containing protein 3', 'PPAR, peroxisome proliferator-activated receptor', 'PTBP1, polypyrimidine tract-binding protein 1', 'PTP, protein tyrosine phosphatase', 'RNS, reactive nitrogen species', 'ROS, reactive oxygen species', 'SCFA, short-chain fatty acid', 'SLC30A8, solute carrier family 30 member 8', 'SLC47A1, solute carrier family 47 member 1', 'SLC5A2, solute carrier family 5 member 2', 'SM, sphingomyelin', 'SNP, single-nucleotide polymorphism', 'SSPG, steady-state plasma glucose', 'SUR1, sulfonylurea receptor 1', 'T2DM, type 2 diabetes mellitus', 'TAG, triacylglycerol', 'TCA, tricarboxylic acid', 'TCF7L2, transcription factor 7-like 2', 'TF, transcription factor', 'TMA, trimethylamine', 'TMAO, trimethylamine oxide', 'TNF-α, tumor necrosis factor alpha', 'TOF-MS, time-of-flight mass spectrum', 'TUG1, taurine upregulated gene 1', 'USP20, ubiquitin-specificpeptidase 20', 'WBC, white blood cell', 'circRNA, circular RNA', 'hADCS, human adipose-derived stem cells', 'hsCRP, high-sensitivity C-reactive protein', 'lnc-BATE1, brown adipose tissue enriched long non-coding RNA 1', 'lncRNA SHGL, lncRNA suppressor of hepatic gluconeogenesis and lipogenesis', 'lncRNA, long non-coding RNA', 'mRNA, messenger RNAs', 'mTORC1, mechanistic target of rapamycin complex 1', 'miRNA, micro RNA', 'ncRNA, non-coding RNA', 't-PA, tissue plasminogen activator', 'α-HB, α-hydroxybutyrate'], 'journal': 'Computational and structural biotechnology journal', 'abstract': 'Glycolipid metabolism disorder are major threats to human health and life. Genetic, environmental, psychological, cellular, and molecular factors contribute to their pathogenesis. Several studies demonstrated that neuroendocrine axis dysfunction, insulin resistance, oxidative stress, chronic inflammatory response, and gut microbiota dysbiosis are core pathological links associated with it. However, the underlying molecular mechanisms and therapeutic targets of glycolipid metabolism disorder remain to be elucidated. Progress in high-throughput technologies has helped clarify the pathophysiology of glycolipid metabolism disorder. In the present review, we explored the ways and means by which genomics, transcriptomics, proteomics, metabolomics, and gut microbiomics could help identify novel candidate biomarkers for the clinical management of glycolipid metabolism disorder. We also discuss the limitations and recommended future research directions of multi-omics studies on these diseases.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2022 The Authors. Published by Elsevier B.V. on behalf of Research Network of Computational and Structural Biotechnology.', 'doi': '10.1016/j.csbj.2022.10.030\\n10.1038/nrendo.2017.151\\n10.3961/jpmph.19.100\\n10.1038/s41586-020-2338-1\\n10.1155/2019/2958920\\n10.1016/j.jff.2021.104556\\n10.3390/nu12123652\\n10.1007/s00125-019-4938-6\\n10.1016/j.appet.2018.01.015\\n10.1016/j.molmet.2018.12.006\\n10.2337/db11-0785\\n10.1016/j.cmet.2013.12.015\\n10.1016/j.cmet.2015.04.008\\n10.1016/j.metabol.2021.154892\\n10.1038/s41580-021-00390-6\\n10.1038/s41586-019-1797-8\\n10.1016/j.redox.2021.101887\\n10.1038/nrendo.2015.189\\n10.1016/j.celrep.2021.109161\\n10.1007/s13238-020-00814-7\\n10.3390/microorganisms8040527\\n10.1016/j.arr.2021.101346\\n10.1038/s41576-020-0216-1\\n10.1038/nature19806\\n10.1074/jbc.AAC118.006541\\n10.1038/ng1732\\n10.1186/s12881-018-0553-5\\n10.1016/j.numecd.2017.10.012\\n10.1016/j.diabet.2016.05.003\\n10.1186/s12881-018-0614-9\\n10.2337/diabetes.47.3.490\\n10.3390/ijms19072124\\n10.1038/nature05616\\n10.1126/science.1142382\\n10.2337/db08-0047\\n10.2337/db08-0877\\n10.1126/science.1151710\\n10.2147/DMSO.S154978\\n10.1016/s0014-5793(97)00488-2\\n10.1126/science.270.5239.1166\\n10.1007/s00125-013-3031-9\\n10.2337/diab.45.6.825\\n10.1038/s41467-018-04951-w\\n10.1038/nature18642\\n10.1038/s41586-020-2263-3\\n10.1038/nature14177\\n10.1210/er.2013-1084\\n10.1016/j.cmet.2019.03.009\\n10.1016/j.diabres.2015.05.003\\n10.1007/s40291-014-0126-z\\n10.1016/j.diabres.2006.10.021\\n10.2337/dc07-2248\\n10.1016/j.ejim.2011.10.018\\n10.1155/2015/176949\\n10.1210/en.2011-0286\\n10.1007/s40262-015-0307-x\\n10.1097/FPC.0000000000000268\\n10.1007/s00125-018-4675-2\\n10.1016/j.amjcard.2003.08.078\\n10.1001/jama.291.23.2821\\n10.1016/s0021-9150(03)00094-7\\n10.1016/s0021-9150(01)00410-5\\n10.1038/nsmb.2921\\n10.1186/s40246-018-0173-3\\n10.1016/j.cmet.2012.08.010\\n10.1159/000373999\\n10.1016/j.ijcard.2016.11.182\\n10.1159/000484191\\n10.2337/db17-0799\\n10.1073/pnas.0804549105\\n10.2337/dbi19-0016\\n10.1038/nature03076\\n10.1007/s00109-015-1296-9\\n10.1038/nm.3862\\n10.2337/db17-0816\\n10.1038/s41574-019-0260-0\\n10.1038/s41467-020-15529-w\\n10.3390/cells9030659\\n10.1038/nature11928\\n10.1038/srep12453\\n10.1007/s00592-016-0943-0\\n10.3892/ijmm.2018.3625\\n10.3390/cells9061473\\n10.1038/s42255-019-0078-z\\n10.1016/j.yexcr.2019.111753\\n10.2152/jmi.54.289\\n10.1016/j.jpba.2009.11.016\\n10.1016/j.atherosclerosis.2013.02.035\\n10.2337/db15-0881\\n10.1007/s10654-018-0475-8\\n10.1016/j.mce.2014.11.021\\n10.5483/bmbrep.2012.45.12.235\\n10.1016/j.jprot.2013.10.005\\n10.3389/fphar.2021.735410\\n10.3390/cells10112832\\n10.2337/dc15-2251\\n10.1111/1753-0407.12645\\n10.3803/EnM.2019.34.3.234\\n10.1007/s00125-017-4222-6\\n10.1074/jbc.M106703200\\n10.1210/jc.2014-2357\\n10.2337/dc17-1642\\n10.1172/JCI64801\\n10.1056/NEJMc1412427\\n10.2337/dc13-0800\\n10.1007/s00125-016-4069-2\\n10.2337/dc18-0840\\n10.1373/clinchem.2014.228965\\n10.1021/pr300523x\\n10.1016/j.dsx.2018.09.009\\n10.1016/j.arcmed.2017.11.003\\n10.1136/gutjnl-2020-323071\\n10.1038/s41574-019-0156-z\\n10.1038/sj.emboj.7601049\\n10.3389/fphar.2018.01382\\n10.3945/ajcn.117.157107\\n10.1111/obr.12843\\n10.1038/s41575-019-0258-z\\n10.1016/j.cmet.2018.07.001\\n10.1016/j.cell.2018.09.055\\n10.1007/s12020-012-9760-6\\n10.1007/s00125-016-4041-1\\n10.2337/db12-0707\\n10.1038/1881110a0\\n10.3390/ijms22136862\\n10.1007/s11306-019-1484-7\\n10.3389/fnut.2019.00021\\n10.3748/wjg.v21.i29.8787\\n10.1017/S0007114516001045\\n10.5582/ddt.2018.01036\\n10.1038/tpj.2012.43\\n10.1038/nature12198\\n10.1016/j.diabet.2016.04.004\\n10.1186/s12876-015-0330-2\\n10.1136/gutjnl-2014-308778\\n10.1038/nature05414\\n10.1038/nm.4358\\n10.1016/j.chom.2019.07.004\\n10.1038/s41467-019-10189-x\\n10.1007/s00018-019-03434-5\\n10.1038/s41586-020-2928-y\\n10.1016/j.jprot.2020.103823\\n10.1038/nature18646\\n10.1038/s41586-019-1236-x\\n10.1038/s41591-019-0414-6\\n10.1038/nrd.2016.32\\n10.3389/fphys.2011.00094\\n10.1111/jdi.12673\\n10.1016/j.nut.2011.08.013', 'publication_date': datetime.date(2022, 11, 17), 'authors': [{'lastname': 'Fang', 'firstname': 'Xinyi', 'initials': 'X', 'affiliation': \"Institute of Metabolic Diseases, Guang'anmen Hospital, China Academy of Chinese Medical Sciences, Beijing 100053, China.\\nGraduate College, Beijing University of Chinese Medicine, Beijing 100029, China.\"}, {'lastname': 'Miao', 'firstname': 'Runyu', 'initials': 'R', 'affiliation': \"Institute of Metabolic Diseases, Guang'anmen Hospital, China Academy of Chinese Medical Sciences, Beijing 100053, China.\\nGraduate College, Beijing University of Chinese Medicine, Beijing 100029, China.\"}, {'lastname': 'Wei', 'firstname': 'Jiahua', 'initials': 'J', 'affiliation': 'Changchun University of Chinese Medicine, Jilin 130117, China.'}, {'lastname': 'Wu', 'firstname': 'Haoran', 'initials': 'H', 'affiliation': 'Graduate College, Beijing University of Chinese Medicine, Beijing 100029, China.'}, {'lastname': 'Tian', 'firstname': 'Jiaxing', 'initials': 'J', 'affiliation': \"Institute of Metabolic Diseases, Guang'anmen Hospital, China Academy of Chinese Medical Sciences, Beijing 100053, China.\"}]}\n",
      "\n",
      "{'pubmed_id': '36369331', 'title': 'Association between TCF7L2 polymorphism and type 2 diabetes mellitus susceptibility: a case-control study among the Bangladeshi population.', 'keywords': ['Bangladesh', 'Chittagong', 'Comorbidities', 'Diabetes', 'Polymorphism', 'TCF7L2'], 'journal': 'Molecular biology reports', 'abstract': 'Diabetes is a severe health burden for Bangladesh. Genetic polymorphism has been reported to be one of the major risk factors for diabetes in various studies. TCF7L2 (transcription factor 7 like 2) transcripts in the human β-cell have effects on β-cell survival, function, and Wnt signaling activation. This study aimed to evaluate the frequency and association of various polymorphisms namely TCF7L2 rs12255372 and rs7903146 among Bangladeshi patients with T2DM (Type 2 Diabetes Mellitus).\\nThis case-control study included 300 patients with T2DM and 234 healthy individuals from two health facilities in the Chattogram Division of Bangladesh. Anthropometric measurements were assessed using a self-reported, structured, eight-item questionnaire. The polymorphisms were identified by PCR-RFLP and sequencing method.\\nA strong association of T2DM\\xa0with polymorphisms was observed, including rs12255372 (p\\u2009=\\u20090.0004) and rs7903146 (p\\u2009=\\u20090.005). It was observed that the risk genotype at rs12255372 was associated with age (p\\u2009=\\u20090.009), a family history of diabetes (p\\u2009<\\u20090.0001), and HbA1C (p\\u2009<\\u20090.0001). Furthermore, it was found that rs12255372 was substantially associated with hypertension (p\\u2009=\\u20090.03), eye problems (p\\u2009=\\u20090.01), and neurological abnormalities (p\\u2009=\\u20090.02).\\nThis study postulates that TCF7L2 genetic polymorphism is associated with the risk of T2DM among the\\xa0studied Bangladeshi population. The findings should be replicated through more studies with a large number of samples and in different populations.', 'conclusions': 'This study postulates that TCF7L2 genetic polymorphism is associated with the risk of T2DM among the\\xa0studied Bangladeshi population. The findings should be replicated through more studies with a large number of samples and in different populations.', 'methods': None, 'results': 'A strong association of T2DM\\xa0with polymorphisms was observed, including rs12255372 (p\\u2009=\\u20090.0004) and rs7903146 (p\\u2009=\\u20090.005). It was observed that the risk genotype at rs12255372 was associated with age (p\\u2009=\\u20090.009), a family history of diabetes (p\\u2009<\\u20090.0001), and HbA1C (p\\u2009<\\u20090.0001). Furthermore, it was found that rs12255372 was substantially associated with hypertension (p\\u2009=\\u20090.03), eye problems (p\\u2009=\\u20090.01), and neurological abnormalities (p\\u2009=\\u20090.02).', 'copyrights': '© 2022. The Author(s), under exclusive licence to Springer Nature B.V.', 'doi': '10.1007/s11033-022-08081-x\\n10.1016/j.metabol.2019.153952\\n10.1038/nrdp.2015.19\\n10.1159/000356747\\n10.1007/s00125-006-0506-y\\n10.3390/ijms21176275\\n10.2471/BLT.13.128371\\n10.1038/nature05616\\n10.4161/isl.23345\\n10.1172/JCI30706\\n10.1007/s00109-007-0203-4\\n10.1007/s00125-006-0502-2\\n10.2337/db09-1386\\n10.2337/db06-0341\\n10.2337/db06-0643\\n10.1186/1471-2350-9-106\\n10.1007/s00125-007-0618-z\\n10.1038/sj.ejhg.5201773\\n10.1016/j.banm.2021.09.001\\n10.1590/S1415-47572010005000056\\n10.1590/S1415-47572012005000029\\n10.1046/j.1365-2125.2001.00346.x\\n10.4103/2321-0656.193997\\n10.4093/dmj.2015.39.5.395\\n10.7754/clin.lab.2013.130809\\n10.1016/j.metabol.2007.04.012\\n10.1016/j.jgeb.2016.10.004\\n10.1038/ng1960\\n10.2337/db07-0012\\n10.1097/MD.0000000000014288\\n10.1186/1471-2350-9-72\\n10.1186/1471-2350-10-15\\n10.2337/db08-0906\\n10.1086/520599\\n10.1038/ng2043\\n10.1126/science.1142364\\n10.1186/s40200-015-0148-z\\n10.4093/dmj.2015.39.6.512\\n10.2337/db07-0421\\n10.1016/j.gene.2012.09.034\\n10.1186/s13104-021-05598-5\\n10.1186/1472-6823-10-9\\n10.1331/JAPhA.2009.09077\\n10.1016/j.diabres.2010.01.024\\n10.1155/2013/409083\\n10.1016/j.phrs.2015.05.013\\n10.3305/nh.2013.28.sup2.6714\\n10.1111/j.1469-1809.2008.00443.x\\n10.1371/journal.pone.0188382\\n10.1590/0004-2730000003510', 'publication_date': datetime.date(2022, 11, 13), 'authors': [{'lastname': 'Salauddin', 'firstname': 'Asma', 'initials': 'A', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Chakma', 'firstname': 'Kallyan', 'initials': 'K', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Hasan', 'firstname': 'Md Mahbub', 'initials': 'MM', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Akter', 'firstname': 'Farhana', 'initials': 'F', 'affiliation': 'Department of Endocrinology, Chittagong Medical College, Chattogram, 4203, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Chowdhury', 'firstname': 'Nowshad Asgar', 'initials': 'NA', 'affiliation': 'Chittagong Diabetic General Hospital, Chattogram, Bangladesh.'}, {'lastname': 'Chowdhury', 'firstname': 'Sumon Rahman', 'initials': 'SR', 'affiliation': 'Chittagong Diabetic General Hospital, Chattogram, Bangladesh.'}, {'lastname': 'Mannan', 'firstname': 'Adnan', 'initials': 'A', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh. adnan.mannan@cu.ac.bd.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh. adnan.mannan@cu.ac.bd.'}]}\n",
      "\n",
      "{'pubmed_id': '36364957', 'title': 'Indolepropionic Acid, a Gut Bacteria-Produced Tryptophan Metabolite and the Risk of Type 2 Diabetes and Non-Alcoholic Fatty Liver Disease.', 'keywords': ['NAFLD', 'diet', 'gut microbiota', 'indolepropionic acid', 'insulin', 'type 2 diabetes'], 'journal': 'Nutrients', 'abstract': 'An intricate relationship between gut microbiota, diet, and the human body has recently been extensively investigated. Gut microbiota and gut-derived metabolites, especially, tryptophan derivatives, modulate metabolic and immune functions in health and disease. One of the tryptophan derivatives, indolepropionic acid (IPA), is increasingly being studied as a marker for the onset and development of metabolic disorders, including type 2 diabetes (T2D) and non-alcoholic fatty liver disease (NAFLD). The IPA levels heavily depend on the diet, particularly dietary fiber, and show huge variations among individuals. We suggest that these variations could partially be explained using genetic variants known to be associated with specific diseases such as T2D. In this narrative review, we elaborate on the beneficial effects of IPA in the mitigation of T2D and NAFLD, and further study the putative interactions between IPA and well-known genetic variants (', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': None, 'doi': '10.3390/nu14214695\\n10.1042/bj0170907\\n10.1007/s12031-002-0036-0\\n10.3390/ijms23031222\\n10.1080/19490976.2019.1586038\\n10.1007/s13668-020-00307-3\\n10.1210/endocr/bqaa235\\n10.1038/srep46337\\n10.1038/s41387-018-0046-9\\n10.3389/fendo.2022.841703\\n10.3389/fphar.2019.00877\\n10.1016/S0140-6736(69)90757-0\\n10.1017/S0954422417000117\\n10.1016/S2468-1253(19)30257-2\\n10.1007/s00125-006-0198-3\\n10.1080/13813455.2017.1398262\\n10.1038/nature24661\\n10.3389/fmicb.2020.575586\\n10.1038/s41586-022-04884-x\\n10.3390/nu13103509\\n10.1017/S0007114500001070\\n10.1155/2016/8952520\\n10.1007/s00535-018-1521-8\\n10.1038/s41467-018-05470-4\\n10.3390/ijms222212435\\n10.1128/JB.187.6.2084-2092.2005\\n10.1016/j.chom.2018.05.003\\n10.3390/ijms22189904\\n10.1186/s40168-020-00837-6\\n10.1038/s41467-019-13498-3\\n10.1016/j.immuni.2013.08.003\\n10.1073/pnas.0812874106\\n10.1016/j.immuni.2014.06.014\\n10.1111/nmo.13178\\n10.1038/s41467-021-23460-x\\n10.1016/j.ajpath.2018.01.011\\n10.1080/19490976.2020.1788899\\n10.1186/s40779-022-00397-w\\n10.3390/metabo10050181\\n10.1038/s41467-020-14676-4\\n10.1016/j.xcrm.2021.100246\\n10.3389/fimmu.2021.653208\\n10.1038/s42003-020-01383-5\\n10.3389/fmed.2021.648259\\n10.1136/gutjnl-2021-324053\\n10.1016/j.ebiom.2019.11.051\\n10.1007/s00253-019-10156-y\\n10.1016/j.nutres.2020.03.005\\n10.1128/mSystems.00313-20\\n10.1002/mnfr.201900580\\n10.1172/jci.insight.145207\\n10.2337/dc21-0099\\n10.3390/microorganisms8010111\\n10.1093/jn/nxy053\\n10.1007/s11306-011-0320-5\\n10.1093/cdn/nzz040.P20-003-19\\n10.1002/mnfr.202100349\\n10.1111/febs.16470\\n10.1002/oby.22870\\n10.1093/cdn/nzaa062_016\\n10.1186/s13073-020-00806-6\\n10.2337/dc21-1705\\n10.1093/gerona/glt152\\n10.1371/journal.pone.0198156\\n10.2337/dc19-2533\\n10.3390/metabo12010069\\n10.3390/metabo11040228\\n10.1080/13813455.2021.2024577\\n10.1152/ajpgi.00105.2020\\n10.1016/j.bmcl.2006.05.007\\n10.3390/nu11030591\\n10.1038/ng.2982\\n10.1007/s00125-007-0656-6\\n10.1038/oby.2008.618\\n10.2337/diabetes.51.8.2581\\n10.1007/s00125-012-2752-5\\n10.2337/db08-1607\\n10.1038/ng.2383\\n10.1038/s41598-020-65051-8\\n10.1016/j.molmet.2020.101092\\n10.1111/joim.12892\\n10.3390/nu14112187\\n10.1002/hep.26342\\n10.1111/ijpo.270\\n10.1161/CIRCRESAHA.122.321253\\n10.1016/j.jvs.2017.09.029\\n10.1016/S0041-1345(02)03616-3\\n10.1038/s12276-019-0304-5\\n10.1002/1097-4644(20010601)81:3<507::AID-JCB1064>3.0.CO;2-M\\n10.3390/nu12103100\\n10.3390/cancers12092411\\n10.1039/D1FO02181H\\n10.1136/gutjnl-2019-319664\\n10.1096/fj.14-259515\\n10.1007/s11356-022-21075-3\\n10.1016/j.tox.2021.152996\\n10.1002/smll.201905233\\n10.18999/nagjms.79.4.477', 'publication_date': datetime.date(2022, 11, 12), 'authors': [{'lastname': 'Sehgal', 'firstname': 'Ratika', 'initials': 'R', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.'}, {'lastname': 'de Mello', 'firstname': 'Vanessa D', 'initials': 'VD', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.'}, {'lastname': 'Männistö', 'firstname': 'Ville', 'initials': 'V', 'affiliation': 'Department of Medicine, University of Eastern Finland, Kuopio University Hospital, 70211 Kuopio, Finland.'}, {'lastname': 'Lindström', 'firstname': 'Jaana', 'initials': 'J', 'affiliation': 'Public Health Prevention Unit, Finnish Institute for Health and Welfare, 00271 Helsinki, Finland.'}, {'lastname': 'Tuomilehto', 'firstname': 'Jaakko', 'initials': 'J', 'affiliation': 'Public Health Prevention Unit, Finnish Institute for Health and Welfare, 00271 Helsinki, Finland.\\nDepartment of Public Health, University of Helsinki, 00014 Helsinki, Finland.\\nDiabetes Research Group, King Abdulaziz University, Jeddah 21589, Saudi Arabia.'}, {'lastname': 'Pihlajamäki', 'firstname': 'Jussi', 'initials': 'J', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.\\nDepartment of Medicine, Endocrinology and Clinical Nutrition, Kuopio University Hospital, 70210 Kuopio, Finland.'}, {'lastname': 'Uusitupa', 'firstname': 'Matti', 'initials': 'M', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.'}]}\n",
      "\n",
      "{'pubmed_id': '36360783', 'title': 'Decoding Diabetes Biomarkers and Related Molecular Mechanisms by Using Machine Learning, Text Mining, and Gene Expression Analysis.', 'keywords': ['bioinformatics', 'diabetes', 'gene expression', 'protein–protein interaction network', 'text mining'], 'journal': 'International journal of environmental research and public health', 'abstract': 'The molecular basis of diabetes mellitus is yet to be fully elucidated. We aimed to identify the most frequently reported and differential expressed genes (DEGs) in diabetes by using bioinformatics approaches. Text mining was used to screen 40,225 article abstracts from diabetes literature. These studies highlighted 5939 diabetes-related genes spread across 22 human chromosomes, with 112 genes mentioned in more than 50 studies. Among these genes, ', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': None, 'doi': '10.3390/ijerph192113890\\n10.1038/414782a\\n10.2522/ptj.20080008\\n10.1371/journal.pone.0243270\\n10.3389/fgene.2020.00734\\n10.3390/cancers13215268\\n10.1097/MD.0000000000020338\\n10.1007/s11011-019-00465-6\\n10.1007/s12551-018-0490-8\\n10.1097/MD.0000000000017504\\n10.1111/brv.12457\\n10.1186/s12874-019-0681-4\\n10.1016/j.cmpb.2019.04.008\\n10.3390/e23010018\\n10.1016/j.ijmedinf.2021.104453\\n10.1093/nar/gks1193\\n10.1093/bioinformatics/bty721\\n10.1093/bioinformatics/btz931\\n10.1093/bioinformatics/btr261\\n10.1093/nar/gkq973\\n10.36462/H.BioSci.20195\\n10.1093/bioinformatics/btp616\\n10.1172/JCI200216857\\n10.1172/JCI200522365\\n10.1186/s13098-020-00557-9\\n10.1093/eurheartj/ehab724.3196\\n10.1007/s12013-007-0025-6\\n10.1074/jbc.R109.040543\\n10.1016/j.jgg.2017.11.007\\n10.1016/j.molmet.2014.12.007\\n10.1016/j.biopha.2017.03.056\\n10.1016/j.biopha.2017.03.102\\n10.2337/db09-1187\\n10.1530/JOE-17-0002\\n10.1038/ng2068\\n10.1038/s41598-019-54545-9\\n10.1194/jlr.M080382\\n10.1038/s41419-020-03312-0\\n10.1016/j.reprotox.2021.10.008\\n10.3390/ijms22136924\\n10.1016/j.str.2008.10.018\\n10.1038/ncomms3863\\n10.1016/j.cellsig.2015.09.016\\n10.1002/jcb.20343\\n10.1186/s12916-020-1503-6\\n10.2147/DMSO.S345081\\n10.1016/j.ejphar.2022.174956\\n10.1210/jc.2009-0384\\n10.1007/s001250051573\\n10.1159/000295775\\n10.3389/fimmu.2015.00493\\n10.1039/b914182k\\n10.1681/ASN.2005080859\\n10.3390/cells8070751\\n10.2337/db20-0038\\n10.1210/jc.2018-02094\\n10.1371/journal.pone.0193684\\n10.1002/humu.23639\\n10.1093/hmg/ddx262\\n10.1371/journal.pone.0234132\\n10.1159/000524849\\n10.3390/cells9030730\\n10.1016/j.jid.2017.11.038\\n10.3390/cells8020178\\n10.1038/nm1185', 'publication_date': datetime.date(2022, 11, 12), 'authors': [{'lastname': 'Elsherbini', 'firstname': 'Amira M', 'initials': 'AM', 'affiliation': 'Department of Oral Biology, Faculty of Dentistry, Mansoura University, Mansoura 35116, Egypt.'}, {'lastname': 'Alsamman', 'firstname': 'Alsamman M', 'initials': 'AM', 'affiliation': 'Agricultural Genetic Engineering Research Institute, Agricultural Research Center, Giza 12619, Egypt.'}, {'lastname': 'Elsherbiny', 'firstname': 'Nehal M', 'initials': 'NM', 'affiliation': 'Department of Pharmaceutical Chemistry, Faculty of Pharmacy, University of Tabuk, Tabuk 71491, Saudi Arabia.\\nDepartment of Biochemistry, Faculty of Pharmacy, Mansoura University, Mansoura 35116, Egypt.'}, {'lastname': 'El-Sherbiny', 'firstname': 'Mohamed', 'initials': 'M', 'affiliation': 'Department of Basic Medical Sciences, College of Medicine, AlMaarefa University, Riyadh 71666, Saudi Arabia.\\nDepartment of Anatomy, Mansoura Faculty of Medicine, Mansoura University, Mansoura 35116, Egypt.'}, {'lastname': 'Ahmed', 'firstname': 'Rehab', 'initials': 'R', 'affiliation': 'Department of Natural Products and Alternative Medicine, Faculty of Pharmacy, University of Tabuk, Tabuk 71491, Saudi Arabia.\\nDepartment of Pharmaceutics, Faculty of Pharmacy, University of Khartoum, Khartoum 11111, Sudan.'}, {'lastname': 'Ebrahim', 'firstname': 'Hasnaa Ali', 'initials': 'HA', 'affiliation': 'Department of Basic Medical Sciences, College of Medicine, Princess Nourah bint Abdulrahman University, Riyadh 11671, Saudi Arabia.'}, {'lastname': 'Bakkach', 'firstname': 'Joaira', 'initials': 'J', 'affiliation': 'Biomedical Genomics and Oncogenetics Research Laboratory, Faculty of Sciences and Techniques of Tangier, Abdelmalek Essaâdi University Morocco, Tétouan 93000, Morocco.'}]}\n",
      "\n",
      "{'pubmed_id': '36358989', 'title': 'FGF9 Promotes Expression of HAS2 in Palatal Elevation via the Wnt/β-Catenin/TCF7L2 Pathway.', 'keywords': ['FGF9', 'HAS2', 'TCF7L2', 'Wnt/β-catenin pathway', 'hyaluronic acid'], 'journal': 'Biomolecules', 'abstract': 'Using Ddx4-Cre mice, we generated the \\nFGF9 promotes HAS2 expression via Wnt/β-catenin/TCF7L2 pathway with TCF7L2 activating transcription of ', 'conclusions': None, 'methods': None, 'results': '', 'copyrights': None, 'doi': '10.3390/biom12111639\\n10.3389/fphys.2016.00067\\n10.1002/ajmg.a.31965\\n10.3109/08977199509036882\\n10.1038/ng.316\\n10.1002/(SICI)1097-0177(199909)216:1<72::AID-DVDY9>3.0.CO;2-9\\n10.1177/0022034518785336\\n10.1242/dev.128.11.2095\\n10.3389/fphys.2021.653040\\n10.1177/0022034517703580\\n10.1038/s41368-021-00122-8\\n10.1016/j.ydbio.2019.09.004\\n10.1182/blood-2014-07-590513\\n10.1080/15476278.2020.1735239\\n10.3389/fphys.2013.00010\\n10.1002/ar.24263\\n10.3389/fcell.2019.00340\\n10.1177/0022034519872478\\n10.3390/biom10111525\\n10.1530/REP-16-0240\\n10.3389/fcvm.2021.701279\\n10.1371/journal.pone.0241281\\n10.1016/j.biopha.2018.04.026\\n10.1002/bdrc.20180\\n10.1242/dev.134577\\n10.1002/lio2.214\\n10.3390/cells9030608\\n10.1016/j.mce.2016.05.023\\n10.1016/j.ydbio.2016.04.005\\n10.1007/BF00428098\\n10.1016/j.ydbio.2016.01.008\\n10.1042/bj3480029\\n10.1074/jbc.274.35.25085\\n10.3389/fcell.2020.00264\\n10.1242/dmm.037051\\n10.1242/dev.158048\\n10.1126/science.aat1178\\n10.1016/j.cell.2012.05.012\\n10.1038/s41598-018-20641-5\\n10.1126/scisignal.aat9519\\n10.1016/j.devcel.2008.04.015\\n10.1016/j.cellsig.2017.08.008\\n10.15252/embj.201798873\\n10.1186/s13098-019-0449-3\\n10.1210/me.2003-0225\\n10.1016/j.semcdb.2009.09.002\\n10.3892/etm.2019.8308\\n10.1016/j.chemosphere.2020.126889\\n10.1002/anie.200462552', 'publication_date': datetime.date(2022, 11, 12), 'authors': [{'lastname': 'Sun', 'firstname': 'Yidan', 'initials': 'Y', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Ying', 'firstname': 'Xiyu', 'initials': 'X', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Li', 'firstname': 'Ruomei', 'initials': 'R', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Weng', 'firstname': 'Mengjia', 'initials': 'M', 'affiliation': 'Department of Stomatology, Huadong Hospital, Fudan University, Shanghai 200040, China.'}, {'lastname': 'Shi', 'firstname': 'Jiajun', 'initials': 'J', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Chen', 'firstname': 'Zhenqi', 'initials': 'Z', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}]}\n",
      "\n",
      "{'pubmed_id': '36344976', 'title': 'CHRNB2 represses pancreatic cancer migration and invasion via inhibiting β-catenin pathway.', 'keywords': ['CHRNB2', 'Lymph node ratio', 'Pancreatic cancer', 'Weighted gene co-expression network analysis', 'β-catenin pathway'], 'journal': 'Cancer cell international', 'abstract': 'Pancreatic cancer is one of the most lethal disease with highly fatal and aggressive properties. Lymph node ratio (LNR), the ratio of the number of metastatic lymph nodes to the total number of examined lymph nodes, is an important index to assess lymphatic metastasis and predict prognosis, but the molecular mechanism underlying high LNR was unclear.\\nGene expression and clinical information data of pancreatic cancer were obtained from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO). Patients in TCGA were averagely divided into low and high LNR groups. Then, Weighted Gene Co-expression Network Analysis (WGCNA) was utilized to build co-expression network to explore LNR-related modules and hub genes. GO and KEGG analysis was performed to find key pathways related to lymph node metastasis. Next, GSE101448 and the overall survival data in TCGA was employed to further select significant genes from hub genes. Considering the key role of CHRNB2 in LNR and survival, gene set enrichment analysis (GSEA) was applied to find pathways related to CHRNB2 expression in pancreatic cancer. The contribution of CHRNB2 to migrative and invasive ability of pancreatic cancer cells was confirmed by Transwell assays. We finally explored the role of CHRNB2 in EMT and β-catenin pathway via Western Blot.\\nHigh LNR was significantly related to high T stages and poor prognosis. In WGCNA, 14 hub genes (COL5A1, FN1, THBS2, etc.) were positively related to high LNR, 104 hub genes (FFAR1, SCG5, TMEM63C, etc.) were negatively related to high LNR. After taking the intersection with GSE101448, 13 genes (CDK5R2, SYT7, CACNA2D2, etc.) which might prevent lymph node metastasis were further selected. Among them, CHRNB2 showed the strongest relationship with long survival. Moreover, CHRNB2 also negatively related to the T stages and LNR. Next, knockdown of CHRNB2 expression could acetylcholine (ACh)-independently increase the migration and invasion of pancreatic cancer cells, while CHRNB2 overexpression ACh-independently decrease the migration and invasion of pancreatic cancer cells. For exploring the underlying mechanism, CHRNB2 downregulated β-catenin pathway might through controlling its upstream regulators such as SOX6, SRY, SOX17, and TCF7L2.\\nCHRNB2 negatively relates to lymph node metastasis in pancreatic cancer patients. CHRNB2 could inhibit β-catenin pathway, EMT, migration and invasion of pancreatic cancer cells via ACh-independent mechanism.', 'conclusions': None, 'methods': None, 'results': 'High LNR was significantly related to high T stages and poor prognosis. In WGCNA, 14 hub genes (COL5A1, FN1, THBS2, etc.) were positively related to high LNR, 104 hub genes (FFAR1, SCG5, TMEM63C, etc.) were negatively related to high LNR. After taking the intersection with GSE101448, 13 genes (CDK5R2, SYT7, CACNA2D2, etc.) which might prevent lymph node metastasis were further selected. Among them, CHRNB2 showed the strongest relationship with long survival. Moreover, CHRNB2 also negatively related to the T stages and LNR. Next, knockdown of CHRNB2 expression could acetylcholine (ACh)-independently increase the migration and invasion of pancreatic cancer cells, while CHRNB2 overexpression ACh-independently decrease the migration and invasion of pancreatic cancer cells. For exploring the underlying mechanism, CHRNB2 downregulated β-catenin pathway might through controlling its upstream regulators such as SOX6, SRY, SOX17, and TCF7L2.', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1186/s12935-022-02768-8\\n10.3390/ijms23031231\\n10.1172/JCI146431\\n10.3390/ijms20174242\\n10.1038/s41573-022-00410-w', 'publication_date': datetime.date(2022, 11, 8), 'authors': [{'lastname': 'Qin', 'firstname': 'Cheng', 'initials': 'C', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Li', 'firstname': 'Tianhao', 'initials': 'T', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.\\nDivision of Plastic Surgery, Department of Surgery, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Wang', 'firstname': 'Yuanyang', 'initials': 'Y', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Zhao', 'firstname': 'Bangbo', 'initials': 'B', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Li', 'firstname': 'Zeru', 'initials': 'Z', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Li', 'firstname': 'Tianyu', 'initials': 'T', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Yang', 'firstname': 'Xiaoying', 'initials': 'X', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Zhao', 'firstname': 'Yutong', 'initials': 'Y', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Wang', 'firstname': 'Weibin', 'initials': 'W', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China. wwb_xh@163.com.'}]}\n",
      "\n",
      "{'pubmed_id': '36300877', 'title': 'Clinical signs of type 1 diabetes are associated with type 2 diabetes marker transcription factor 7-like 2 polymorphism.', 'keywords': ['Anti-islet autoantibodies', 'Transcription factor 7-like 2 gene', 'Type 1 diabetes'], 'journal': 'Journal of diabetes investigation', 'abstract': 'We aimed to assess the distribution of transcription factor 7-like 2 gene TCF7L2 (rs7903146) polymorphism and to find possible associations between TCF7L2 and the characteristics of type 1 diabetes.\\nWe studied 190 newly diagnosed type 1 diabetes patients (median age 12.7\\u2009years, range 2.0-72.5) and 246 controls (median age 23.8\\u2009years, range 1.4-81.5) for TCF7L2 single nucleotide polymorphism. We determined anti-islet autoantibodies, random C-peptide levels, diabetes associated HLA DR/DQ haplotypes and genotypes in all patients.\\nThere were no differences in the distribution of TCF7L2 single nucleotide polymorphism between patients and controls. However, patients with in type 1 diabetes, after adjusting for age and sex, subjects carrying C allele were at risk for a C-peptide level lower than 0.5\\u2009nmol/L (OR 5.65 [95% CI: 1.14-27.92]) and for zinc transporter 8 autoantibody positivity (5.22 [1.34-20.24]). Participants without T allele were associated with a higher level of islet antigen-2 autoantibodies (3.51 [1.49-8.27]) and zinc transporter 8 autoantibodies (2.39 [1.14-4.99]).\\nThe connection of TCF7L2 polymorphism with zinc transporter 8 and islet antigen-2 autoantibodies and C-peptide levels in patients supports the viewpoint that TCF7L2 is associated with the clinical signs and autoimmune characteristics of type 1 diabetes. The mechanisms of the interaction between the TCF7L2 risk genotype and anti-islet autoantibodies need to be studied further.', 'conclusions': None, 'methods': None, 'results': 'There were no differences in the distribution of TCF7L2 single nucleotide polymorphism between patients and controls. However, patients with in type 1 diabetes, after adjusting for age and sex, subjects carrying C allele were at risk for a C-peptide level lower than 0.5\\u2009nmol/L (OR 5.65 [95% CI: 1.14-27.92]) and for zinc transporter 8 autoantibody positivity (5.22 [1.34-20.24]). Participants without T allele were associated with a higher level of islet antigen-2 autoantibodies (3.51 [1.49-8.27]) and zinc transporter 8 autoantibodies (2.39 [1.14-4.99]).', 'copyrights': '© 2022 The Authors. Journal of Diabetes Investigation published by Asian Association for the Study of Diabetes (AASD) and John Wiley & Sons Australia, Ltd.', 'doi': '10.1111/jdi.13933', 'publication_date': datetime.date(2022, 10, 28), 'authors': [{'lastname': 'Ergür', 'firstname': 'Efe', 'initials': 'E', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Ergür', 'firstname': 'Ege', 'initials': 'E', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Alnek', 'firstname': 'Kristi', 'initials': 'K', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Metsküla', 'firstname': 'Kaja', 'initials': 'K', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Peet', 'firstname': 'Aleksandr', 'initials': 'A', 'affiliation': \"Department of Pediatrics, Institute of Clinical Medicine, University of Tartu, Tartu, Estonia.\\nChildren's Clinic of Tartu University Hospital, Tartu, Estonia.\"}, {'lastname': 'Lubi', 'firstname': 'Maire', 'initials': 'M', 'affiliation': 'Department of Internal Medicine, Institute of Clinical Medicine, University of Tartu, Tartu, Estonia.\\nInternal Medicine Clinic of Tartu University Hospital, Tartu, Estonia.'}, {'lastname': 'Heilman', 'firstname': 'Kaire', 'initials': 'K', 'affiliation': \"Tallinn Children's Hospital, Tallinn, Estonia.\"}, {'lastname': 'Uibo', 'firstname': 'Raivo', 'initials': 'R', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}]}\n",
      "\n",
      "{'pubmed_id': '36295578', 'title': 'ITGA2 Gene Polymorphism Is Associated with Type 2 Diabetes Mellitus in the Kazakhstan Population.', 'keywords': ['BglII polymorphism in ITGA2 gene', 'diabetic retinopathy', 'type 2 diabetes mellitus', 'α2β1 integrin receptor'], 'journal': 'Medicina (Kaunas, Lithuania)', 'abstract': '', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': None, 'doi': '10.3390/medicina58101416\\n10.1038/nrendo.2017.151\\n10.1172/jci.insight.93751\\n10.1016/j.visres.2017.03.006\\n10.1016/j.preteyeres.2013.02.001\\n10.1038/nrdp.2016.12\\n10.1016/j.pop.2015.05.005\\n10.2337/diab.46.11.1829\\n10.1046/j.1464-5491.2002.00820.x\\n10.1038/s41581-020-0278-5\\n10.21037/aes.2018.08.04\\n10.1093/ndt/gfl641\\n10.1097/IIO.0b013e31819fd5d7\\n10.1155/2010/861291\\n10.12809/hkmj164869\\n10.1371/journal.pone.0094160\\n10.3109/02713683.2014.932386\\n10.1007/s00125-015-3569-9\\n10.3109/02713683.2015.1084642\\n10.1155/2016/5305282\\n10.2337/diabetes.50.6.1505\\n10.15171/jnp.2018.54\\n10.1182/blood.V92.7.2382\\n10.3132/dvdr.2005.001\\n10.1186/s12933-018-0763-3\\n10.1055/s-0035-1556731\\n10.5858/2002-126-0305-IOPCRP\\n10.1182/blood.V95.5.1560.005k43_1560_1564\\n10.1007/s10038-003-0060-0\\n10.1080/08820538.2019.1632354\\n10.2147/TACG.S31979\\n10.1182/blood.V89.6.1939\\n10.1182/blood.V97.6.1721\\n10.17513/spno.29283\\n10.1177/1099800420977685\\n10.4172/2155-6156.1000223', 'publication_date': datetime.date(2022, 10, 28), 'authors': [{'lastname': 'Magazova', 'firstname': 'Aizhan', 'initials': 'A', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.\\nDepartment of Biochemistry, Asfendiyarov Kazakh National Medical University, Almaty 050012, Kazakhstan.'}, {'lastname': 'Ashirbekov', 'firstname': 'Yeldar', 'initials': 'Y', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.'}, {'lastname': 'Abaildayev', 'firstname': 'Arman', 'initials': 'A', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.'}, {'lastname': 'Satken', 'firstname': 'Kantemir', 'initials': 'K', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.'}, {'lastname': 'Balmukhanova', 'firstname': 'Altynay', 'initials': 'A', 'affiliation': 'Department of Health Policy and Organization, Al-Farabi Kazakh National University, Almaty 050040, Kazakhstan.'}, {'lastname': 'Akanov', 'firstname': 'Zhanay', 'initials': 'Z', 'affiliation': 'City Center of Diabetes AAA Medical Clinic, Almaty 050040, Kazakhstan.'}, {'lastname': 'Jainakbayev', 'firstname': 'Nurlan', 'initials': 'N', 'affiliation': 'Department of Public Health, Kazakh-Russian Medical University, Almaty 050004, Kazakhstan.'}, {'lastname': 'Balmukhanova', 'firstname': 'Aigul', 'initials': 'A', 'affiliation': 'International Medical School, Caspian University, Almaty 050000, Kazakhstan.'}, {'lastname': 'Sharipov', 'firstname': 'Kamalidin', 'initials': 'K', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.\\nDepartment of Biochemistry, Asfendiyarov Kazakh National Medical University, Almaty 050012, Kazakhstan.'}]}\n",
      "\n",
      "{'pubmed_id': '36282337', 'title': 'Milder loss of insulin-containing islets in individuals with type 1 diabetes and type 2 diabetes-associated TCF7L2 genetic variants.', 'keywords': ['African American', 'Genetics', 'Heterogeneity', 'Insulin-containing cells', 'Pancreas', 'Phenotype', 'Precision medicine', 'Race', 'TCF7L2', 'Type 1 diabetes'], 'journal': 'Diabetologia', 'abstract': 'TCF7L2 variants are the strongest genetic risk factor for type 2 diabetes. In individuals with type 1 diabetes, these variants are associated with a higher C-peptide AUC, a lower glucose AUC during an OGTT, single autoantibody positivity near diagnosis, particularly in individuals older than 12 years of age, and a lower frequency of type 1 diabetes-associated HLA genotypes. Based on initial observations from clinical cohorts, we tested the hypothesis that type 2 diabetes-predisposing TCF7L2 genetic variants are associated with a higher percentage of residual insulin-containing cells (ICI%) in pancreases of donors with type 1 diabetes, by examining genomic data and pancreatic tissue samples from the Network for Pancreatic Organ donors with Diabetes (nPOD) programme.\\nWe analysed nPOD donors with type 1 diabetes (n=110; mean±SD age at type 1 diabetes onset 12.2±7.9 years, mean±SD diabetes duration 15.3±13.7 years, 53% male, 80% non-Hispanic White, 12.7% African American, 7.3% Hispanic) using data pertaining to residual beta cell number; quantified islets containing insulin-positive beta cells in pancreatic tissue sections; and expressed these values as a percentage of the total number of islets from each donor (mean ± SD ICI% 9.8±21.5, range 0-92.2).\\nDonors with a high ICI% (≥5) (n=30; 27%) vs a low ICI% (<5) (n=80; 73%) were older at onset (15.3±6.9 vs 11.1±8 years, p=0.013), had a shorter diabetes duration at donor tissue procurement (7.0±7.4 vs 18.5±14.3 years, p<0.001), a higher African ancestry score (0.2±0.3 vs 0.1±0.2, p=0.043) and a lower European ancestry score (0.7±0.3 vs 0.9±0.3, p=0.023). After adjustment for age of onset (p=0.105), diabetes duration (p<0.001), BMI z score (p=0.145), sex (p=0.351) and African American race (p=0.053), donors with the TCF7L2 rs7903146 T allele (TC or TT, 45.5%) were 2.93 times (95% CI 1.02, 8.47) more likely to have a high ICI% than those without it (CC) (p=0.047).\\nOverall, these data support the presence of a type 1 diabetes endotype associated with a genetic factor that predisposes to type 2 diabetes, with donors in this category exhibiting less severe beta cell loss. It is possible that in these individuals the disease pathogenesis may include mechanisms associated with type 2 diabetes and thus this may provide an explanation for the poor response to immunotherapies to prevent type 1 diabetes or its progression in a subset of individuals. If so, strategies that target both type 1 diabetes and type 2 diabetes-associated factors when they are present may increase the success of prevention and treatment in these individuals.', 'conclusions': None, 'methods': None, 'results': 'Donors with a high ICI% (≥5) (n=30; 27%) vs a low ICI% (<5) (n=80; 73%) were older at onset (15.3±6.9 vs 11.1±8 years, p=0.013), had a shorter diabetes duration at donor tissue procurement (7.0±7.4 vs 18.5±14.3 years, p<0.001), a higher African ancestry score (0.2±0.3 vs 0.1±0.2, p=0.043) and a lower European ancestry score (0.7±0.3 vs 0.9±0.3, p=0.023). After adjustment for age of onset (p=0.105), diabetes duration (p<0.001), BMI z score (p=0.145), sex (p=0.351) and African American race (p=0.053), donors with the TCF7L2 rs7903146 T allele (TC or TT, 45.5%) were 2.93 times (95% CI 1.02, 8.47) more likely to have a high ICI% than those without it (CC) (p=0.047).', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1007/s00125-022-05818-y', 'publication_date': datetime.date(2022, 10, 26), 'authors': [{'lastname': 'Redondo', 'firstname': 'Maria J', 'initials': 'MJ', 'affiliation': \"Department of Pediatrics, Baylor College of Medicine, Texas Children's Hospital, Houston, TX, USA. redondo@bcm.edu.\"}, {'lastname': 'Richardson', 'firstname': 'Sarah J', 'initials': 'SJ', 'affiliation': 'Islet Biology Group (IBEx), Exeter Centre of Excellence for Diabetes Research (EXCEED), University of Exeter College of Medicine and Health, Exeter, UK. s.richardson@exeter.ac.uk.'}, {'lastname': 'Perry', 'firstname': 'Daniel', 'initials': 'D', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}, {'lastname': 'Minard', 'firstname': 'Charles G', 'initials': 'CG', 'affiliation': 'Institute for Clinical and Translational Research, Baylor College of Medicine, Houston, TX, USA.'}, {'lastname': 'Carr', 'firstname': 'Alice L J', 'initials': 'ALJ', 'affiliation': 'Islet Biology Group (IBEx), Exeter Centre of Excellence for Diabetes Research (EXCEED), University of Exeter College of Medicine and Health, Exeter, UK.'}, {'lastname': 'Brusko', 'firstname': 'Todd', 'initials': 'T', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}, {'lastname': 'Kusmartseva', 'firstname': 'Irina', 'initials': 'I', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}, {'lastname': 'Pugliese', 'firstname': 'Alberto', 'initials': 'A', 'affiliation': 'Diabetes Research Institute, Department of Medicine, Division of Endocrinology, Department of Microbiology and Immunology, Miller School of Medicine, University of Miami, Miami, FL, USA.'}, {'lastname': 'Atkinson', 'firstname': 'Mark A', 'initials': 'MA', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}]}\n",
      "\n",
      "{'pubmed_id': '36263318', 'title': '', 'keywords': ['TCF7L2 gene', 'adipose tissue', 'postprandial hypertriglyceridemia', 'prediabetes', 'type 2 diabetes mellitus'], 'journal': 'Frontiers in endocrinology', 'abstract': 'Postprandial lipid responses to a standardized fat challenge test were performed in 620 Asian Indian subjects (310 with NGT and 310 with T2DM/prediabetes) and compared between the risk and wild genotypes of the rs7903146 \\nT allele of rs7903146 \\nThere is significant PPTg dysmetabolism associated with the risk allele of rs7903146 polymorphism as well as adipocyte expression of ', 'conclusions': 'There is significant PPTg dysmetabolism associated with the risk allele of rs7903146 polymorphism as well as adipocyte expression of ', 'methods': None, 'results': 'T allele of rs7903146 ', 'copyrights': 'Copyright © 2022 Madhu, Mishra, Mannar, Aslam, Banerjee and Agrawal.', 'doi': '10.3389/fendo.2022.973718\\n10.1186/1471-2350-10-15\\n10.2337/db09-1386\\n10.1007/s00125-006-0502-2\\n10.1038/ng1732\\n10.2337/db11-0550\\n10.1016/j.metabol.2007.04.012\\n10.1007/s00125-006-0507-x\\n10.1007/s11892-008-0026-x\\n10.1056/NEJMoa062418\\n10.1172/JCI30706\\n10.2337/db17-0318\\n10.1016/S0065-230X(08)60783-6\\n10.1016/0092-8674(92)90630-U\\n10.1155/2011/567305\\n10.1101/003202\\n10.2337/db12-0365\\n10.1074/jbc.M411487200\\n10.1016/j.cell.2012.10.053\\n10.1371/journal.pone.0043390\\n10.1038/srep43128\\n10.3945/jn.108.096461\\n10.1007/s13410-018-0678-2\\n10.2337/diabetes.48.6.1258\\n10.7326/0003-4819-131-1-199907060-00006\\n10.1016/j.nut.2005.01.009\\n10.1016/0168-8227(94)90269-0\\n10.1016/j.jacl.2017.08.001\\n10.1371/journal.pone.0145730\\n10.1111/joim.12540\\n10.1007/s13410-018-0678-2\\n10.1111/ddg.13528\\n10.2337/dc17-S005\\n10.1016/j.diabres.2008.01.016\\n10.1006/meth.2001.1262\\n10.2337/db06-0474\\n10.1007/s10038-007-0231-5\\n10.1016/j.metabol.2007.04.012\\n10.1016/j.atherosclerosissup.2010.03.002\\n10.1111/j.1469-1809.2008.00443.x\\n10.1002/hep.22659\\n10.3967/bes2016.108\\n10.2337/db06-0338\\n10.1186/s13098-018-0338-1\\n10.1016/j.bbrc.2008.03.006\\n10.1007/s00125-007-0850-6\\n10.1016/j.metabol.2008.04.016\\n10.1007/s00125-020-05292-4\\n10.1210/jc.2016-1594\\n10.1016/j.molmet.2019.03.003\\n10.1210/jcem.86.11.7992\\n10.1371/journal.pone.0000812\\n10.1371/journal.pone.0237974\\n10.1093/jn/134.1.205\\n10.3390/biom10091230\\n10.3390/biom10040601\\n10.1080/21623945.2020.1776082\\n10.1002/oby.21377\\n10.1371/journal.pone.0018284\\n10.1210/jc.2016-1594\\n10.1016/j.jacc.2011.02.038\\n10.1155/2012/983814', 'publication_date': datetime.date(2022, 10, 21), 'authors': [{'lastname': 'Madhu', 'firstname': 'Sri Venkata', 'initials': 'SV', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Mishra', 'firstname': 'Brijesh Kumar', 'initials': 'BK', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Mannar', 'firstname': 'Velmurugan', 'initials': 'V', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Aslam', 'firstname': 'Mohd', 'initials': 'M', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Banerjee', 'firstname': 'Basudev', 'initials': 'B', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Agrawal', 'firstname': 'Vivek', 'initials': 'V', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}]}\n",
      "\n",
      "{'pubmed_id': '36217307', 'title': 'Regulation of the THRA gene, encoding the thyroid hormone nuclear receptor TRα1, in intestinal lesions.', 'keywords': ['THRA', 'TRα1', 'colon cancer', 'intestinal organoids', 'thyroid hormone nuclear receptor'], 'journal': 'Molecular oncology', 'abstract': 'The THRA gene, encoding the thyroid hormone nuclear receptor TRα1, is expressed in an increasing gradient at the bottom of intestinal crypts, overlapping with high Wnt and Notch activities. Importantly, THRA is upregulated in colorectal cancers, particularly in the high-Wnt molecular subtype. The basis of this specific and/or altered expression pattern has remained unknown. To define the mechanisms controlling THRA transcription and TRα1 expression, we used multiple in\\u2009vitro and ex\\u2009vivo approaches. Promoter analysis demonstrated that transcription factors important for crypt homeostasis and altered in colorectal cancers, such as transcription factor 7-like 2 (TCF7L2; Wnt pathway), recombining binding protein suppressor of hairless (RBPJ; Notch pathway), and homeobox protein CDX2 (epithelial cell identity), modulate THRA activity. Specifically, although TCF7L2 and CDX2 stimulated THRA, RBPJ induced its repression. In-depth analysis of the Wnt-dependent increase showed direct regulation of the THRA promoter in cells and of TRα1 expression in murine enteroids. Given our previous results on the control of the Wnt pathway by TRα1, our new results unveil a complex regulatory loop and synergy between these endocrine and epithelial-cell-intrinsic signals. Our work describes, for the first time, the regulation of the THRA gene in specific cell and tumor contexts.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2022 The Authors. Molecular Oncology published by John Wiley & Sons Ltd on behalf of Federation of European Biochemical Societies.', 'doi': '10.1002/1878-0261.13298', 'publication_date': datetime.date(2022, 10, 12), 'authors': [{'lastname': 'Giolito', 'firstname': 'Maria Virginia', 'initials': 'MV', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.\\nINSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}, {'lastname': 'La Rosa', 'firstname': 'Théo', 'initials': 'T', 'affiliation': 'INSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}, {'lastname': 'Farhat', 'firstname': 'Diana', 'initials': 'D', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.\\nINSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}, {'lastname': 'Bodoirat', 'firstname': 'Serguei', 'initials': 'S', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.'}, {'lastname': 'Guardia', 'firstname': 'Gabriela D A', 'initials': 'GDA', 'affiliation': 'Centro de Oncologia Molecular, Hospital Sírio-Libanês, São Paulo, Brazil.'}, {'lastname': 'Domon-Dell', 'firstname': 'Claire', 'initials': 'C', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.'}, {'lastname': 'Galante', 'firstname': 'Pedro A F', 'initials': 'PAF', 'affiliation': 'Centro de Oncologia Molecular, Hospital Sírio-Libanês, São Paulo, Brazil.'}, {'lastname': 'Freund', 'firstname': 'Jean-Noel', 'initials': 'JN', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.'}, {'lastname': 'Plateroti', 'firstname': 'Michelina', 'initials': 'M', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.\\nINSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}]}\n",
      "\n",
      "{'pubmed_id': '36183778', 'title': 'Integrated analysis of microRNA expression profiles and function network in mice testes after low dose lead exposure from early puberty.', 'keywords': ['CeRNA', 'Low dose lead', 'Puberty', 'Reproductive toxicology', 'miRNA'], 'journal': 'Toxicology and applied pharmacology', 'abstract': 'There is evidence suggesting the participation of non-coding RNAs in male reproductive dysfunction induced by lead, and the significance of microRNAs has been highlighted recently because of their essential roles in gene regulatory networks. To comprehensively understand the functions of miRNA and the regulatory networks, RNA sequencing was carried out to obtain miRNA expression profiles in mice testes exposed to low dose Pb for 90\\xa0days at the onset of puberty. In total, 44 differentially expressed miRNAs with 26 up-regulated and 18 down-regulated were identified between 200\\xa0mg/L Pb group and control group (p\\xa0<\\xa00.05). Enrichment analysis confirmed that the target genes of DE miRNAs might participate in the metabolism of testicular cells. Furthermore, a miRNA-mRNA co-expression network consisting of 19 miRNAs and 106 mRNAs and a competing endogenous RNA network of lncRNA-miRNA-mRNA including 179 genes were established. Finally, the expressions of 4 miRNAs (mmu-miR-451a, mmu-miR-133a-3p, mmu-miR-1a-3p and mmu-miR-486a-3p) and 4 mRNAs (Gramd1b, Tcf7l2, Mov10 and Srcin1) involved in regulatory networks were verified by RT-qPCR. In conclusion, our research might provide targets for the mechanism studies of miRNAs in reproductive toxicity of Pb.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': 'Copyright © 2022. Published by Elsevier Inc.', 'doi': '10.1016/j.taap.2022.116260', 'publication_date': datetime.date(2022, 10, 3), 'authors': [{'lastname': 'Ma', 'firstname': 'Haitao', 'initials': 'H', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Zhang', 'firstname': 'Haoran', 'initials': 'H', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Yu', 'firstname': 'Jun', 'initials': 'J', 'affiliation': 'Department of Preventive Medicine, School of Basic Medical Sciences, Hubei University of Science and Technology, Xianning 437100, Hubei Province, China.'}, {'lastname': 'Wang', 'firstname': 'Ziqiong', 'initials': 'Z', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Zeng', 'firstname': 'Xiangchao', 'initials': 'X', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Ye', 'firstname': 'Jingping', 'initials': 'J', 'affiliation': 'Department of Pediatrics, Renmin Hospital of Wuhan University, Wuhan 430071, Hubei Province, China. Electronic address: jpy7012@163.com.'}, {'lastname': 'Wang', 'firstname': 'Chunhong', 'initials': 'C', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China. Electronic address: wchunhong027@whu.edu.cn.'}]}\n",
      "\n",
      "{'pubmed_id': '36182916', 'title': 'Leveraging family history in genetic association analyses of binary traits.', 'keywords': ['Family history', 'GWAS', 'Meta-analysis'], 'journal': 'BMC genomics', 'abstract': \"Considering relatives' health history in logistic regression for case-control genome-wide association studies (CC-GWAS) may provide new information that increases accuracy and power to detect disease associated genetic variants. We conducted simulations and analyzed type 2 diabetes (T2D) data from the Framingham Heart Study (FHS) to compare two methods, liability threshold model conditional on both case-control status and family history (LT-FH) and Fam-meta, which incorporate family history into CC-GWAS.\\nIn our simulation scenario of trait with modest T2D heritability (h\\nOverall, LT-FH and Fam-meta had higher power than CC-GWAS in simulations, especially using phenotypes that were more prevalent in older age groups, and both methods detected known genetic variants with lower P-values in real data application, highlighting the benefits of including family history in genetic association studies.\", 'conclusions': None, 'methods': None, 'results': 'In our simulation scenario of trait with modest T2D heritability (h', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1186/s12864-022-08897-8\\n10.1089/cmb.2012.0024\\n10.1371/journal.pone.0051589\\n10.1016/j.ajhg.2013.02.011\\n10.1086/521580\\n10.1038/s41588-020-0613-6\\n10.1002/gepi.21785\\n10.1093/bioinformatics/btr341\\n10.1038/ng1732\\n10.1093/nar/gkv1340\\n10.1038/ng.2653\\n10.1038/s41467-018-08000-4\\n10.1161/HYPERTENSIONAHA.116.08581\\n10.1002/em.21827\\n10.1101/2020.06.28.171561\\n10.1038/s41588-018-0205-x\\n10.1371/journal.pone.0230815\\n10.1016/j.ajhg.2022.01.009\\n10.1038/s41431-021-00980-0\\n10.1016/j.ajhg.2022.03.001', 'publication_date': datetime.date(2022, 10, 2), 'authors': [{'lastname': 'Zhang', 'firstname': 'Yixin', 'initials': 'Y', 'affiliation': 'Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA. zhangyix@bu.edu.'}, {'lastname': 'Meigs', 'firstname': 'James B', 'initials': 'JB', 'affiliation': 'Division of General Internal Medicine, Massachusetts General Hospital, Boston, MA, USA.\\nDepartment of Medicine, Harvard Medical School, Boston, MA, USA.'}, {'lastname': 'Liu', 'firstname': 'Ching-Ti', 'initials': 'CT', 'affiliation': 'Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA.'}, {'lastname': 'Dupuis', 'firstname': 'Josée', 'initials': 'J', 'affiliation': 'Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA.\\nDepartment of Epidemiology, Biostatistics and Occupational Health, McGill University, Montréal, Québec, Canada.'}, {'lastname': 'Sarnowski', 'firstname': 'Chloé', 'initials': 'C', 'affiliation': 'Department of Epidemiology, Human Genetics, and Environmental Sciences, The University of Texas Health Science Center, School of Public Health, Houston, TX, USA.'}]}\n",
      "\n",
      "{'pubmed_id': '36174902', 'title': 'Genetic risk score is associated with T2DM and diabetes complications risks.', 'keywords': ['Complications', 'Diabetes', 'Gene score', 'Polymorphism'], 'journal': 'Gene', 'abstract': \"Type 2 diabetes mellitus (T2DM) is a prototypical complex disease with polygenic architecture playing an important role in determining susceptibility to develop the disease (and its complications) in subjects exposed to modifiable lifestyle factors. A current challenge is to quantify the degree of the individual's genetic risk using genetic risk scores (GRS) capturing the results of genome-wide association studies while incorporating possible ethnicity- or population-specific differences.\\nThis study included three groups of T2DM (T2DM-I, N\\xa0=\\xa01,032; T2DM-II, N\\xa0=\\xa0353; and T2DM-III, N\\xa0=\\xa0399) patients and 2,481 diabetes-free subjects. The status of the microvascular and macrovascular diabetes complications were known for the T2DM-I patients. Overall, 21 single nucleotide polymorphisms (SNPs) were analyzed, and selected subsets were used to determine the GRS (both weighted - wGRS and unweighted - uGRS) for T2DM risk predictions (6 SNPs) and for predicting the risks of complications (7 SNPs).\\nThe strongest T2DM markers (P\\xa0<\\xa00.0001) were within the genes for TCF7L2 (transcription factor 7-like 2), FTO (fat mass and obesity associated protein) and ARAP1 (ankyrin repeat and PH domain 1). The T2DM-I subjects with uGRS values greater (Odds Ratio, 95\\xa0% Confidence Interval) than six had at least twice (2.00, 1.72-2.32) the risk of T2DM development (P\\xa0<\\xa00.0001), and these results were confirmed in the independent groups (T2DM-II 1.82, 1.45-2.27; T2DM-III 2.63, 2.11-3.27). The wGRS (>0.6) further improved (P\\xa0<\\xa00.000001) the risk estimations for all three T2DM groups. The uGRS was also a significant predictor of neuropathy (P\\xa0<\\xa00.0001), nephropathy (P\\xa0<\\xa00.005) and leg ischemia (P\\xa0<\\xa00.0005).\\nIf carefully selected and specified, GRS, both weighted and unweighted, could be significant predictors of T2DM development, as well as the diabetes complications development.\", 'conclusions': None, 'methods': None, 'results': 'The strongest T2DM markers (P\\xa0<\\xa00.0001) were within the genes for TCF7L2 (transcription factor 7-like 2), FTO (fat mass and obesity associated protein) and ARAP1 (ankyrin repeat and PH domain 1). The T2DM-I subjects with uGRS values greater (Odds Ratio, 95\\xa0% Confidence Interval) than six had at least twice (2.00, 1.72-2.32) the risk of T2DM development (P\\xa0<\\xa00.0001), and these results were confirmed in the independent groups (T2DM-II 1.82, 1.45-2.27; T2DM-III 2.63, 2.11-3.27). The wGRS (>0.6) further improved (P\\xa0<\\xa00.000001) the risk estimations for all three T2DM groups. The uGRS was also a significant predictor of neuropathy (P\\xa0<\\xa00.0001), nephropathy (P\\xa0<\\xa00.005) and leg ischemia (P\\xa0<\\xa00.0005).', 'copyrights': 'Copyright © 2022. Published by Elsevier B.V.', 'doi': '10.1016/j.gene.2022.146921', 'publication_date': datetime.date(2022, 9, 30), 'authors': [{'lastname': 'Hubacek', 'firstname': 'Jaroslav A', 'initials': 'JA', 'affiliation': 'Experimental Medicine Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic; 3rd Department of Internal Medicine, 1(st) Faculty of Medicine, Charles University, Prague, Czech Republic. Electronic address: jahb@ikem.cz.'}, {'lastname': 'Dlouha', 'firstname': 'Lucie', 'initials': 'L', 'affiliation': 'Experimental Medicine Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic; Department of Anthropology and Human Genetics, Faculty of Science, Charles University, Prague, Czech Republic.'}, {'lastname': 'Adamkova', 'firstname': 'Vera', 'initials': 'V', 'affiliation': 'Department of Preventive Cardiology, Institute for Clinical and Experimental Medicine, Prague, Czech Republic; Czech Technical University of Prague, Faculty of Biomedical Engineering, Prague, Czech Republic.'}, {'lastname': 'Dlouha', 'firstname': 'Dana', 'initials': 'D', 'affiliation': 'Experimental Medicine Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}, {'lastname': 'Pacal', 'firstname': 'Lukas', 'initials': 'L', 'affiliation': 'Department of Pathophysiology, Faculty of Medicine, Masaryk University, Brno, Czech Republic.'}, {'lastname': 'Kankova', 'firstname': 'Katerina', 'initials': 'K', 'affiliation': 'Department of Pathophysiology, Faculty of Medicine, Masaryk University, Brno, Czech Republic.'}, {'lastname': 'Galuska', 'firstname': 'David', 'initials': 'D', 'affiliation': 'Department of Pathophysiology, Faculty of Medicine, Masaryk University, Brno, Czech Republic.'}, {'lastname': 'Lanska', 'firstname': 'Vera', 'initials': 'V', 'affiliation': 'Statistical Unit, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}, {'lastname': 'Veleba', 'firstname': 'Jiri', 'initials': 'J', 'affiliation': 'Diabetes Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}, {'lastname': 'Pelikanova', 'firstname': 'Terezie', 'initials': 'T', 'affiliation': 'Diabetes Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}]}\n",
      "\n",
      "{'pubmed_id': '36163520', 'title': 'Development and validation of a model that predicts the risk of diabetic retinopathy in type 2 diabetes mellitus patients.', 'keywords': ['Diabetic retinopathy', 'Nomogram', 'Prediction model', 'Risk factors', 'Type 2 diabetes mellitus'], 'journal': 'Acta diabetologica', 'abstract': 'Diabetic retinopathy is the leading cause of blindness in people with type 2 diabetes. To enable primary care physicians to identify high-risk type 2 diabetic patients with diabetic retinopathy at an early stage, we developed a nomogram model to predict the risk of developing diabetic retinopathy in the Xinjiang type 2 diabetic population.\\nIn a retrospective study, we collected data on 834 patients with type 2 diabetes through an electronic medical record system. Stepwise regression was used to filter variables. Logistic regression was applied to build a nomogram prediction model and further validated in the training set. The c-index, forest plot, calibration plot, and clinical decision curve analysis were used to comprehensively validate the model and evaluate its accuracy and clinical validity.\\nFour predictors were selected to establish the final model: hypertension, blood urea nitrogen, duration of diabetes, and diabetic peripheral neuropathy. The model displayed medium predictive power with a C-index of 0.781(95%CI:0.741-0.822) in the training set and 0.865(95%CI:0.807-0.923)in the validation set. The calibration curve of the DR probability shows that the predicted results of the nomogram are in good agreement with the actual results. Decision curve analysis demonstrated that the novel nomogram was clinically valuable.\\nThe nomogram of the risk of developing diabetic nephropathy contains 4 characteristics. that can help primary care physicians quickly identify individuals at high risk of developing DR in patients with type 2 diabetes, to intervene as soon as possible.', 'conclusions': None, 'methods': None, 'results': 'Four predictors were selected to establish the final model: hypertension, blood urea nitrogen, duration of diabetes, and diabetic peripheral neuropathy. The model displayed medium predictive power with a C-index of 0.781(95%CI:0.741-0.822) in the training set and 0.865(95%CI:0.807-0.923)in the validation set. The calibration curve of the DR probability shows that the predicted results of the nomogram are in good agreement with the actual results. Decision curve analysis demonstrated that the novel nomogram was clinically valuable.', 'copyrights': '© 2022. Springer-Verlag Italia S.r.l., part of Springer Nature.', 'doi': '10.1007/s00592-022-01973-1\\n10.1016/j.dsx.2021.102239\\n10.3390/ijerph18147346\\n10.3346/jkms.2016.31.S2.S146\\n10.2337/dc11-1909\\n10.1001/archopht.122.4.552\\n10.1002/dmrr.1307\\n10.1097/IAE.0b013e31821c4252\\n10.1007/s00592-020-01540-6\\n10.1001/archopht.126.12.1740\\n10.1016/j.ophtha.2010.08.003\\n10.1136/bjophthalmol-2017-310316\\n10.1002/(SICI)1096-9136(199807)15:7<539::AID-DIA668>3.0.CO;2-S\\n10.1186/s12882-017-0776-2\\n10.1016/j.ebiom.2018.08.009\\n10.1097/MD.0000000000019236\\n10.1016/j.jdiacomp.2013.12.008\\n10.1016/j.ophtha.2006.10.023\\n10.1136/bmjopen-2018-023586\\n10.4103/ijo.IJO_1144_21\\n10.1371/journal.pone.0097928\\n10.1016/j.jdiacomp.2013.12.008\\n10.1016/j.atherosclerosis.2012.10.034\\n10.1002/mnfr.201900934\\n10.1016/j.gtc.2016.02.004\\n10.4103/ijo.IJO_1279_21\\n10.1507/endocrj.EJ20-0794\\n10.1172/JCI86181\\n10.2147/DMSO.S244061', 'publication_date': datetime.date(2022, 9, 28), 'authors': [{'lastname': 'Yang', 'firstname': 'Jing', 'initials': 'J', 'affiliation': 'State Key Laboratory of Pathogenesis, Prevention andTreatment of High Incidence Diseases in Central Asia, Department of Endocrinology, The First Affiliated Hospital of Xinjiang Medical University, Urumqi, 830017, China.'}, {'lastname': 'Jiang', 'firstname': 'Sheng', 'initials': 'S', 'affiliation': 'State Key Laboratory of Pathogenesis, Prevention andTreatment of High Incidence Diseases in Central Asia, Department of Endocrinology, The First Affiliated Hospital of Xinjiang Medical University, Urumqi, 830017, China. xjjiangsheng@126.com.'}]}\n",
      "\n",
      "{'pubmed_id': '36155628', 'title': 'Effect of TCF7L2 on the relationship between lifestyle factors and glycemic parameters: a systematic review.', 'keywords': ['Dietary factors', 'Glycemic parameters', 'Lifestyle factors', 'TCF7L2'], 'journal': 'Nutrition journal', 'abstract': 'Among candidate\\xa0genes related to type 2 diabetes (T2DM), one of the strongest genes is Transcription factor 7 like 2 (TCF7L2), regarding the Genome-Wide Association Studies. We aimed to conduct a systematic review of the literature on the modification effect of TCF7L2 on the relation between glycemic parameters and lifestyle factors.\\nA systematic literature search was done for relevant publications using electronic databases, including PubMed, EMBASE, Scopus, and Web of Science, from January 1, 2000, to November 2, 2021.\\nThirty-eight studies (16 observational studies, six meal test trials, and 16 randomized controlled trials (RCTs)) were included. Most observational studies had been conducted on participants with non-diabetes showing that TCF7L2 modified the association between diet (fatty acids and fiber) and insulin resistance. In addition, findings from meal test trials showed that, compared to non-risk-allele carriers, consumption of meals with different percentages of total dietary fat in healthy risk-allele carriers increased glucose concentrations and impaired insulin sensitivity. However, ten RCTs, with intervention periods of less than ten weeks and more than one year, showed that TCF7L2 did not modify glycemic parameters in response to a dietary intervention involving different macronutrients. However, two weight loss dietary RCTs with more than 1-year duration showed that serum glucose and insulin levels decreased and insulin resistance improved in non-risk allele subjects with overweight/obesity. Regarding artichoke extract supplementation (ALE), two RCTs observed that ALE supplementation significantly decreased insulin concentration and improved insulin resistance in the TT genotype of the rs7903146 variant of TCF7L2. In addition, four studies suggested that physical activity levels and smoking status modified the association between TCF7L2 and glycemic parameters. However, three studies observed no effect of TCF7L2 on glycemic parameters in participants with different levels of physical activity and smoking status.\\nThe modification effects of TCF7L2 on the relation between the lifestyle factors (diet, physical activity, and smoking status) and glycemic parameters were contradictory.\\nCRD42020196327.', 'conclusions': 'The modification effects of TCF7L2 on the relation between the lifestyle factors (diet, physical activity, and smoking status) and glycemic parameters were contradictory.', 'methods': None, 'results': 'Thirty-eight studies (16 observational studies, six meal test trials, and 16 randomized controlled trials (RCTs)) were included. Most observational studies had been conducted on participants with non-diabetes showing that TCF7L2 modified the association between diet (fatty acids and fiber) and insulin resistance. In addition, findings from meal test trials showed that, compared to non-risk-allele carriers, consumption of meals with different percentages of total dietary fat in healthy risk-allele carriers increased glucose concentrations and impaired insulin sensitivity. However, ten RCTs, with intervention periods of less than ten weeks and more than one year, showed that TCF7L2 did not modify glycemic parameters in response to a dietary intervention involving different macronutrients. However, two weight loss dietary RCTs with more than 1-year duration showed that serum glucose and insulin levels decreased and insulin resistance improved in non-risk allele subjects with overweight/obesity. Regarding artichoke extract supplementation (ALE), two RCTs observed that ALE supplementation significantly decreased insulin concentration and improved insulin resistance in the TT genotype of the rs7903146 variant of TCF7L2. In addition, four studies suggested that physical activity levels and smoking status modified the association between TCF7L2 and glycemic parameters. However, three studies observed no effect of TCF7L2 on glycemic parameters in participants with different levels of physical activity and smoking status.', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1186/s12937-022-00813-w\\n10.1016/j.diabres.2019.107843\\n10.1177/2047487319878371\\n10.1016/j.cmet.2008.08.006\\n10.1038/ng1732\\n10.1038/79216\\n10.1038/nature05616\\n10.1038/ng.120\\n10.2337/dc17-0290\\n10.1007/s11010-021-04086-5\\n10.1080/07435800.2019.1573827\\n10.2337/db09-0918\\n10.2337/db09-1169\\n10.1016/j.numecd.2006.04.001\\n10.1111/obr.12921\\n10.1016/j.gde.2018.02.001\\n10.1007/s11892-016-0758-y\\n10.1055/s-0031-1285832\\n10.1007/s00109-006-0134-5\\n10.1146/annurev-nutr-071811-150648\\n10.2337/dc12-2211\\n10.3390/ijms18061188\\n10.3945/ajcn.116.150094\\n10.3389/fgene.2020.590369\\n10.1097/CM9.0000000000000617\\n10.1038/ijo.2008.146\\n10.3945/ajcn.112.038125\\n10.3945/ajcn.2009.27947\\n10.1371/journal.pone.0043390\\n10.2337/db11-0973\\n10.1371/journal.pone.0230815\\n10.1038/oby.2007.77\\n10.1056/NEJMoa062418\\n10.1371/journal.pone.0021518\\n10.1136/bmj.n71\\n10.1136/bmj.l4898\\n10.1007/s00125-010-1940-4\\n10.1210/jc.2015-1172\\n10.1186/s13098-018-0338-1\\n10.1007/s00125-009-1307-x\\n10.1016/j.diabet.2017.05.001\\n10.1017/S0007114516002798\\n10.3945/jn.111.147447\\n10.1002/ptr.5951\\n10.3945/ajcn.2009.28379\\n10.2337/db09-1050\\n10.1007/s12263-012-0284-8\\n10.1016/j.joim.2018.05.006\\n10.1007/s00125-009-1630-2\\n10.2337/dc10-1150\\n10.1016/j.atherosclerosis.2010.10.027\\n10.1007/s00125-012-2634-x\\n10.2337/dc13-0955\\n10.1186/s12863-014-0134-3\\n10.1016/j.metabol.2016.11.018\\n10.1371/journal.pone.0188382\\n10.3390/nu13061936\\n10.1016/j.jnutbio.2010.11.020\\n10.2337/db08-1623\\n10.2337/db20-0156\\n10.1002/gepi.21991\\n10.1016/j.trsl.2021.07.001\\n10.1038/s41467-020-20500-w\\n10.1172/JCI89444\\n10.1038/nrendo.2014.175', 'publication_date': datetime.date(2022, 9, 27), 'authors': [{'lastname': 'Hosseinpour-Niazi', 'firstname': 'Somayeh', 'initials': 'S', 'affiliation': 'Nutrition and Endocrine Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Mirmiran', 'firstname': 'Parvin', 'initials': 'P', 'affiliation': 'Nutrition and Endocrine Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran. mirmiran@endocrine.ac.ir.'}, {'lastname': 'Hosseini', 'firstname': 'Shabnam', 'initials': 'S', 'affiliation': 'School of Human Nutrition, Faculty of Agricultural and Environmental Sciences, McGill University, Montreal, Quebec, Canada.'}, {'lastname': 'Hadaegh', 'firstname': 'Farzad', 'initials': 'F', 'affiliation': 'Prevention of Metabolic Disorders Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Ainy', 'firstname': 'Elaheh', 'initials': 'E', 'affiliation': 'Department of Vice Chancellor Research Affairs, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Daneshpour', 'firstname': 'Maryam S', 'initials': 'MS', 'affiliation': 'Cellular and Molecular Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Azizi', 'firstname': 'Fereidoun', 'initials': 'F', 'affiliation': 'Endocrine Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}]}\n",
      "\n",
      "{'pubmed_id': '36479247', 'title': 'Genomic profiling and network-level understanding uncover the potential genes and the pathways in hepatocellular carcinoma.', 'keywords': ['HCV and HCC', 'biomarkers', 'co-expression', 'gene expression/mutational profiling', 'network-level understanding'], 'journal': 'Frontiers in genetics', 'abstract': 'Data integration with phenotypes such as gene expression, pathways or function, and protein-protein interactions data has proven to be a highly promising technique for improving human complex diseases, particularly cancer patient outcome prediction. Hepatocellular carcinoma is one of the most prevalent cancers, and the most common cause is chronic HBV and HCV infection, which is linked to the majority of cases, and HBV and HCV play a role in multistep carcinogenesis progression. We examined the list of known hepatocellular carcinoma biomarkers with the publicly available expression profile dataset of hepatocellular carcinoma infected with HCV from day 1 to day 10 in this study. The study covers an overexpression pattern for the selected biomarkers in clinical hepatocellular carcinoma patients, a combined investigation of these biomarkers with the gathered temporal dataset, temporal expression profiling changes, and temporal pathway enrichment following HCV infection. Following a temporal analysis, it was discovered that the early stages of HCV infection tend to be more harmful in terms of expression shifting patterns, and that there is no significant change after that, followed by a set of genes that are consistently altered. PI3K, cAMP, TGF, TNF, Rap1, NF-kB, Apoptosis, Longevity regulating pathway, signaling pathways regulating pluripotency of stem cells, Cytokine-cytokine receptor interaction, p53 signaling, Wnt signaling, Toll-like receptor signaling, and Hippo signaling pathways are just a few of the most commonly enriched pathways. The majority of these pathways are well-known for their roles in the immune system, infection and inflammation, and human illnesses like cancer. We also find that ADCY8, MYC, PTK2, CTNNB1, TP53, RB1, PRKCA, TCF7L2, PAK1, ITPR2, CYP3A4, UGT1A6, GCK, and FGFR2/3 appear to be among the prominent genes based on the networks of genes and pathways based on the copy number alterations, mutations, and structural variants study.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': 'Copyright © 2022 El-Kafrawy, El-Daly, Bajrai, Alandijany, Faizo, Mobashir, Ahmed, Ahmed, Alam, Jeet, Kamal, Anwer, Khan, Tashkandi, Rizvi and Azhar.', 'doi': '10.3389/fgene.2022.880440\\n10.1016/j.ajpath.2012.08.005\\n10.1186/1479-5876-10-41\\n10.3389/fonc.2022.914032\\n10.1101/gr.087528.108\\n10.3390/ijms231911024\\n10.1016/j.dnarep.2019.06.005\\n10.1038/s41598-021-98031-7\\n10.3389/fcimb.2021.707905\\n10.1038/nrg1272\\n10.1093/nar/gkp942\\n10.1074/jbc.M010271200\\n10.1093/sysbio/syu081\\n10.1038/nrg3424\\n10.1245/s10434-013-2985-7\\n10.1038/nature07385\\n10.1098/rsif.2011.0390\\n10.1038/nbt.2203\\n10.1038/nature08987\\n10.1038/ng.3221\\n10.1038/msb4100200\\n10.1093/annonc/mdt419\\n10.1056/NEJMra1001683\\n10.3389/fendo.2021.641888\\n10.1038/nature06758\\n10.1002/wsbm.134\\n10.1093/nar/29.2.449\\n10.1038/nrc1299\\n10.1038/nrg3813\\n10.1038/nmeth.2562\\n10.1182/blood-2008-05-158865\\n10.21873/anticanres.11078\\n10.1038/nrg.2016.87\\n10.3390/biomedicines10051076\\n10.1016/j.dnarep.2007.11.007\\n10.1002/ijc.22605\\n10.1186/s13059-014-0436-9\\n10.1146/annurev-immunol-032712-100001\\n10.36013/jimsa.v1i1.3\\n10.1093/nar/gkm882\\n10.1093/nar/gkp896\\n10.1038/s41598-022-11143-6\\n10.1038/s41598-022-11143-6\\n10.1371/journal.pone.0062113\\n10.3389/fonc.2012.00202\\n10.1038/nature11412\\n10.1016/j.imu.2020.100422\\n10.1016/j.imu.2020.100422\\n10.1073/pnas.0304146101\\n10.1186/s13059-014-0473-4\\n10.1038/ng1956\\n10.1038/nrclinonc.2015.103\\n10.1053/j.gastro.2019.04.003\\n10.1053/j.gastro.2019.04.003\\n10.1093/nar/gkv1194\\n10.1038/nbt.3027\\n10.1016/j.ajpath.2013.02.026\\n10.1126/scitranslmed.aaa0763\\n10.1073/pnas.0609635104\\n10.1186/s13059-014-0439-6\\n10.3390/diseases3040294\\n10.1093/hmg/ddq079\\n10.1093/bioinformatics/bts448\\n10.1038/nrc3816\\n10.1146/annurev.med.45.1.297\\n10.1016/j.diabres.2018.03.002\\n10.1038/nrclinonc.2014.5\\n10.1101/gr.1239303\\n10.1186/1475-2840-11-15\\n10.1073/pnas.0506580102\\n10.1093/annonc/mdu005\\n10.1016/j.ccr.2010.05.026\\n10.1146/annurev.biophys.33.110502.132654\\n10.1074/mcp.M500279-MCP200\\n10.1126/science.aax9198\\n10.1126/science.aan2507\\n10.1146/annurev-pharmtox-010611-134628\\n10.1016/j.ccr.2005.10.001\\n10.2174/1381612826666201106093148\\n10.1038/nrclinonc.2014.6\\n10.1126/science.aaw0726\\n10.1038/nrg3871\\n10.1186/s40246-015-0056-9', 'publication_date': datetime.date(2022, 12, 9), 'authors': [{'lastname': 'El-Kafrawy', 'firstname': 'Sherif A', 'initials': 'SA', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'El-Daly', 'firstname': 'Mai M', 'initials': 'MM', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Bajrai', 'firstname': 'Leena H', 'initials': 'LH', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nBiochemistry Department, Faculty of Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Alandijany', 'firstname': 'Thamir A', 'initials': 'TA', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Faizo', 'firstname': 'Arwa A', 'initials': 'AA', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}, {'lastname': 'Mobashir', 'firstname': 'Mohammad', 'initials': 'M', 'affiliation': 'Department of Microbiology, Tumor and Cell Biology (MTC), Karolinska Institute, Stockholm, Sweden.\\nGenome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Ahmed', 'firstname': 'Sunbul S', 'initials': 'SS', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Ahmed', 'firstname': 'Sarfraz', 'initials': 'S', 'affiliation': 'Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Alam', 'firstname': 'Shoaib', 'initials': 'S', 'affiliation': 'Department of Biotechnology, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Jeet', 'firstname': 'Raja', 'initials': 'R', 'affiliation': 'Botany Department, Ganesh Dutt College, Begusarai, Bihar, India.'}, {'lastname': 'Kamal', 'firstname': 'Mohammad Amjad', 'initials': 'MA', 'affiliation': 'Institutes for Systems Genetics, Frontiers Science Center for Disease-related Molecular Network, West China Hospital, Sichuan University, Chengdu, China.\\nKing Fahd Medical Research Center, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Pharmacy, Faculty of Allied Health Sciences, Daffodil International University, Dhaka, Bangladesh.\\nEnzymoics, Hebersham, NSW, Australia.\\nNovel Global Community Educational Foundation, Hebersham, NSW, Australia.'}, {'lastname': 'Anwer', 'firstname': 'Syed Tauqeer', 'initials': 'ST', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Khan', 'firstname': 'Bushra', 'initials': 'B', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Tashkandi', 'firstname': 'Manal', 'initials': 'M', 'affiliation': 'Department of Biochemistry, College of Science, University of Jeddah, Jeddah, Saudi Arabia.'}, {'lastname': 'Rizvi', 'firstname': 'Moshahid A', 'initials': 'MA', 'affiliation': 'Genome Biology Lab, Department of Biosciences, Jamia Millia Islamia, New Delhi, India.'}, {'lastname': 'Azhar', 'firstname': 'Esam Ibraheem', 'initials': 'EI', 'affiliation': 'Special Infectious Agents Unit-BSL3, King Fahd Medical Research Centre, King Abdulaziz University, Jeddah, Saudi Arabia.\\nDepartment of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, King Abdulaziz University, Jeddah, Saudi Arabia.'}]}\n",
      "\n",
      "{'pubmed_id': '36454469', 'title': 'Genomic Editing and Diabetes.', 'keywords': ['CRISPR/Cas9', 'Diabetes', 'Genetic editing', 'Long noncoding RNA', 'MicroRNA'], 'journal': 'Advances in experimental medicine and biology', 'abstract': 'The diabetes types and its complications have varying developmental and metabolic pathways. There is an interplay of nongenetic and genetic components in pathogenesis of diabetes and its complications. There are several established genes such as ABCC8, TCF7L2, SLC2A2, and CAPN10 which are known to influence blood insulin and glucose levels. Current management of diabetes types may include lifetime burdensome use of insulin, insulin sensitizers, insulin secretagogues, etc. There has been increasing interest in improving genetic editing tools such as CRISPR/Cas9 and using genetically edited stem cells to alter diabetes disease course or possibly cure it. Current research on microRNAs and long noncoding RNAs may provide insights into the pathways involved in development of diabetes and its complications. Consequently, developing further understanding of genetics and its messenger pathways in diabetes would enhance our ability to develop precise and accurate genetic editing tools which can translate into clinically useful therapeutics.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2023. The Author(s), under exclusive license to Springer Nature Singapore Pte Ltd.', 'doi': '10.1007/978-981-19-5642-3_14\\n10.1016/S0140-6736(16)00618-8\\n10.34172/hpp.2020.18\\n10.1016/j.ymgme.2003.10.005\\n10.2337/db17-0318\\n10.1016/j.ijbiomac.2020.04.240\\n10.1186/s12881-018-0553-5\\n10.1055/a-1008-9223\\n10.1007/s42000-018-0047-z\\n10.1016/j.molmet.2019.03.003\\n10.1172/jci.insight.136136\\n10.1002/humu.23995\\n10.1007/978-3-030-25905-1_18\\n10.1111/1753-0407.13144\\n10.1007/s00125-018-4759-z\\n10.1038/ng.3632\\n10.1530/JME-19-0253\\n10.1155/2017/3937893\\n10.1007/s00404-016-4140-8\\n10.1007/5584_2018_305\\n10.1080/21655979.2016.1189039\\n10.1371/journal.pbio.3000603\\n10.1016/j.bbrc.2019.09.001\\n10.1186/s12933-019-0824-2\\n10.1530/EJE-19-0206\\n10.1161/CIRCRESAHA.119.314898\\n10.3389/fimmu.2018.02711\\n10.1080/09674845.2017.1402404\\n10.1038/s41467-019-13587-3\\n10.1007/s11033-018-4429-8\\n10.1073/pnas.1914353117\\n10.3390/ijms21114020\\n10.1016/j.atherosclerosis.2018.08.031\\n10.7150/ijms.34200', 'publication_date': datetime.date(2022, 12, 2), 'authors': [{'lastname': 'Shah', 'firstname': 'Parth', 'initials': 'P', 'affiliation': 'ObvioHealth, New York, NY, USA. prshah06@gmail.com.'}]}\n",
      "\n",
      "{'pubmed_id': '36399471', 'title': 'Bioinformatic analysis and clinical diagnostic value of hsa_circ_0004099 in acute ischemic stroke.', 'keywords': [], 'journal': 'PloS one', 'abstract': 'This study investigates the expression and effect of hsa_circ_0004099 in acute ischemic stroke (AIS). We conducted a case-controlled study that included 40 patients with AIS within 24 hours and 40 healthy subjects during the same period as a control group. Differentially expressed circular RNAs (circRNAs) were obtained using GEO2R, and the expression of hsa_circ_0004099 was verified using RT-PCR. Correlation analysis of the National Institutes of Health Stroke Scale (NIHSS) disease severity score and ischemic time with hsa_circ_0004099 expression levels was also performed. The receiver operating characteristic (ROC) curve of hsa_circ_0004099 was constructed, and bioinformatic analysis of hsa_circ_0004099 was performed. NIHSS scores negatively correlated with hsa_circ_0004099 levels (P<0.001, r = -0.7053), whereas infarct time was negatively correlated with hsa_circ_0004099 levels (P<0.001, r = -0.5130); hsa_circ_0004099 could benefit clinical diagnosis (area under the curve [AUC]: 0.923 [95% confidence interval [CI]: 0.8680-0.9904]). Kyoto encyclopedia of genes and genomes (KEGG) analysis showed that hsa_circ_0004099 was enriched in several cancer pathways, which were collectively enriched in four genes namely TCF7L2, NRAS, CTNNB1, and KRAS. Eight core proteins were screened using a protein-protein interaction (PPI) network namely SMAD4, HIF1A, CTNNB1, CDKN1B, CDK6, FOXO3, KRAS, and NRAS. hsa_circ_0004099 is a potential clinical diagnostic marker. In addition, the possible role of hsa_circ_0004099 in the pathogenesis of AIS was analyzed using bioinformatics, which provided a new potential molecular target for AIS treatment.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': 'Copyright: © 2022 Zheng et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.', 'doi': '10.1371/journal.pone.0277832\\n10.1016/S0140-6736(18)31874-9\\n10.1001/jama.2015.3058\\n10.29271/jcpsp.2019.09.898\\n10.6061/clinics/2021/e2958\\n10.1002/wrna.1294\\n10.2174/1567205017666201111114928\\n10.1007/s00018-017-2688-5\\n10.1016/j.gene.2019.03.053\\n10.18632/aging.103424\\n10.1002/jcp.29165\\n10.1038/s41598-020-66990-y\\n10.3389/fnins.2020.00081\\n10.1016/j.ajhg.2016.10.006\\n10.1016/j.atherosclerosis.2018.01.028\\n10.1161/JAHA.114.001119\\n10.1371/journal.pone.0141214\\n10.4037/aacnacc2018483\\n10.1002/jcla.22954\\n10.7754/Clin.Lab.2020.191266\\n10.1161/STROKEAHA.119.027348\\n10.1016/j.brainres.2020.146835\\n10.3892/mmr.2019.10317\\n10.1007/s12975-020-00851-8\\n10.1007/s10528-022-10225-4\\n10.1155/2021/5585206\\n10.1093/jnen/nlab106\\n10.1016/j.gene.2020.145022\\n10.1159/000514543\\n10.1590/1806-9282.20210509\\n10.1016/j.diabres.2013.12.051\\n10.1007/s00432-021-03842-w\\n10.1002/ana.26059\\n10.1155/2021/6745282\\n10.1002/ana.23883\\n10.1186/s12885-022-09186-z\\n10.1002/acn3.51154\\n10.1007/s11010-022-04428-x\\n10.3389/fnagi.2022.830494\\n10.1155/2020/2507074\\n10.17116/jnevro201611612210-13\\n10.3389/fonc.2021.757650\\n10.1161/STROKEAHA.111.000504\\n10.1016/j.jns.2015.05.036\\n10.3390/metabo12030250\\n10.1093/gerona/glac016\\n10.3390/ijms222011131\\n10.1080/00207454.2018.1564290\\n10.1016/j.gep.2021.119218', 'publication_date': datetime.date(2022, 11, 19), 'authors': [{'lastname': 'Zheng', 'firstname': 'Jiqing', 'initials': 'J', 'affiliation': 'Deparment of Rehabilitation, The Second Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi, China.'}, {'lastname': 'Luo', 'firstname': 'Shuiming', 'initials': 'S', 'affiliation': 'Deparment of Rehabilitation, The Second Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi, China.'}, {'lastname': 'Long', 'firstname': 'Yaobin', 'initials': 'Y', 'affiliation': 'Deparment of Rehabilitation, The Second Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi, China.'}]}\n",
      "\n",
      "{'pubmed_id': '36382598', 'title': 'Identification of odontogenic ameloblast associated as a novel target gene of the Wnt/β-catenin signaling pathway.', 'keywords': ['ODAM', 'Wnt signaling', 'cell cycle', 'colorectal cancer', 'hepatoblastoma'], 'journal': 'Cancer science', 'abstract': 'The Wnt/β-catenin signaling pathway plays a key role in development and carcinogenesis. Although some target genes of this signaling have been identified in various tissues and neoplasms, the comprehensive understanding of the target genes and their roles in the development of human cancer, including hepatoma and colorectal cancer remain to be fully elucidated. In this study, we searched for genes regulated by the Wnt signaling in liver cancer using HuH-7 hepatoma cells. A comparison of the expression profiles between cells expressing an active form of mutant β-catenin and cells expressing enhanced green fluorescent protein (EGFP) identified seven genes upregulated by the mutant β-catenin gene (CTNNB1). Among the seven genes, we focused in this study on ODAM, odontogenic,\\xa0ameloblast associated, as a novel target gene. Interestingly, its expression was frequently upregulated in hepatocellular carcinoma, colorectal adenocarcinoma, and hepatoblastoma. We additionally identified a distant enhancer region that was associated with the β-catenin/TCF7L2 complex. Further analyses revealed that ODAM plays an important role in the regulation of the cell cycle, DNA synthesis, and cell proliferation. These data may be useful for clarification of the main molecular mechanism(s) underlying these cancers.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2022 The Authors. Cancer Science published by John Wiley & Sons Australia, Ltd on behalf of Japanese Cancer Association.', 'doi': '10.1111/cas.15657', 'publication_date': datetime.date(2022, 11, 17), 'authors': [{'lastname': 'Yamaguchi', 'firstname': 'Kiyoshi', 'initials': 'K', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Horie', 'firstname': 'Chiaki', 'initials': 'C', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Takane', 'firstname': 'Kiyoko', 'initials': 'K', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Ikenoue', 'firstname': 'Tsuneo', 'initials': 'T', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Nakagawa', 'firstname': 'Saya', 'initials': 'S', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Isobe', 'firstname': 'Yumiko', 'initials': 'Y', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Ota', 'firstname': 'Yasunori', 'initials': 'Y', 'affiliation': 'Department of Pathology, Research Hospital, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Ushiku', 'firstname': 'Tetsuo', 'initials': 'T', 'affiliation': 'Department of Pathology, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Tanaka', 'firstname': 'Mariko', 'initials': 'M', 'affiliation': 'Department of Pathology, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Fujishiro', 'firstname': 'Jun', 'initials': 'J', 'affiliation': 'Department of Pediatric Surgery, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Hoshino', 'firstname': 'Noriko', 'initials': 'N', 'affiliation': 'Department of Pediatric Surgery, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Arisue', 'firstname': 'Atsuhiro', 'initials': 'A', 'affiliation': 'Department of Surgery, Iwate Medical University School of Medicine, Yahaba, Japan.'}, {'lastname': 'Nishizuka', 'firstname': 'Satoshi', 'initials': 'S', 'affiliation': 'Division of Biomedical Research and Development, Iwate Medical University Institute for Biomedical Sciences, Yahaba, Japan.'}, {'lastname': 'Aikou', 'firstname': 'Susumu', 'initials': 'S', 'affiliation': 'Division of Frontier Surgery, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Shida', 'firstname': 'Dai', 'initials': 'D', 'affiliation': 'Division of Frontier Surgery, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}, {'lastname': 'Furukawa', 'firstname': 'Yoichi', 'initials': 'Y', 'affiliation': 'Division of Clinical Genome Research, The Institute of Medical Science, The University of Tokyo, Tokyo, Japan.'}]}\n",
      "\n",
      "{'pubmed_id': '36382190', 'title': 'Advances in multi-omics study of biomarkers of glycolipid metabolism disorder.', 'keywords': ['1,5-AG, 1,5-anhydroglucitol', '2-AAA, 2-aminoadipic acid', '2-DGE, two-dimensional gel electrophoresis', 'ABCC8, ATP binding cassette subfamily C member 8', 'ADA, American Diabetes Association', 'AhR, aromatic hydrocarbon receptor', 'BA, bile acid', 'BCAA, branched-chain amino acid', 'BCKAs, branched chain keto acids', 'Biomarkers', 'CE, cholesterol ester', 'CFL1, cofilin-1', 'CRP, C reactive protein', 'CYP450, cytochrome P450', 'DAG, diacylglycerol', 'DPP-4, dipeptyl peptidase 4', 'EASD, European Association for the Study of Diabetes', 'FA, fatty acid', 'FFA, free fatty acid', 'FMT, fecal microbiota transplantation', 'FTO, fat mass and obesity-associated', 'GAS5, growth arrest-specific transcript 5', 'GC–MS/MS, gas chromatography-tandem mass spectrometry', 'GLP-1, glucagon-like peptide 1', 'GLP-1R, glucagon-like peptide 1 receptor', 'GSIS, glucose-stimulated insulin secretion', 'GWAS, genome-wide association study', 'Glycolipid metabolism disorder', 'HGF, hepatocyte growth factor', 'HMG-CoA, hydroxymethylglutaryl-coenzyme A', 'HMGCR, 3-hydroxy-methylglutaryl coenzyme A reductase', 'HPLC, high performance liquid chromatography', 'HbA1C, glycylated hemoglobin', 'IL-1ra, interleukin-1 receptor antagonist', 'IMP, imidazole propionate', 'IR, insulin resistance', 'JNK, c-Jun-N-terminal-kinase', 'KCNJ11, potassium inwardly rectifying channel subfamily J member 11', 'L-GPC, linoleoyl-glycerophosphocholine', 'LC-MS/MS, liquid chromatography-tandem mass spectrometry', 'LPC, lysophosphatidylcholine', 'LPE, lysophosphatidylethanolamines', 'LPS, lipopolysaccharide', 'MAPK, mitogen-activated protein kinase', 'MASP, mannose-binding lectin-associated serine protease 1', 'MATE1, multidrug and toxin extrusion protein 1', 'MDR1, multidrug resistance mutation 1', 'MGT, magnesium transporter', 'MS, Mass Spectrometry', 'Mechanism study', 'Multi-omics', 'NK, natural killer', 'NMR, nuclear magnetic resonance', 'OCT, organic cationic transporter', 'OGTT, oral glucose tolerance test', 'PC-PL, phosphatidylcholine-plasmalogen', 'PE, phosphatidylethanolamines', 'PNPLA3, patatin-like phospholipase domain-containing protein 3', 'PPAR, peroxisome proliferator-activated receptor', 'PTBP1, polypyrimidine tract-binding protein 1', 'PTP, protein tyrosine phosphatase', 'RNS, reactive nitrogen species', 'ROS, reactive oxygen species', 'SCFA, short-chain fatty acid', 'SLC30A8, solute carrier family 30 member 8', 'SLC47A1, solute carrier family 47 member 1', 'SLC5A2, solute carrier family 5 member 2', 'SM, sphingomyelin', 'SNP, single-nucleotide polymorphism', 'SSPG, steady-state plasma glucose', 'SUR1, sulfonylurea receptor 1', 'T2DM, type 2 diabetes mellitus', 'TAG, triacylglycerol', 'TCA, tricarboxylic acid', 'TCF7L2, transcription factor 7-like 2', 'TF, transcription factor', 'TMA, trimethylamine', 'TMAO, trimethylamine oxide', 'TNF-α, tumor necrosis factor alpha', 'TOF-MS, time-of-flight mass spectrum', 'TUG1, taurine upregulated gene 1', 'USP20, ubiquitin-specificpeptidase 20', 'WBC, white blood cell', 'circRNA, circular RNA', 'hADCS, human adipose-derived stem cells', 'hsCRP, high-sensitivity C-reactive protein', 'lnc-BATE1, brown adipose tissue enriched long non-coding RNA 1', 'lncRNA SHGL, lncRNA suppressor of hepatic gluconeogenesis and lipogenesis', 'lncRNA, long non-coding RNA', 'mRNA, messenger RNAs', 'mTORC1, mechanistic target of rapamycin complex 1', 'miRNA, micro RNA', 'ncRNA, non-coding RNA', 't-PA, tissue plasminogen activator', 'α-HB, α-hydroxybutyrate'], 'journal': 'Computational and structural biotechnology journal', 'abstract': 'Glycolipid metabolism disorder are major threats to human health and life. Genetic, environmental, psychological, cellular, and molecular factors contribute to their pathogenesis. Several studies demonstrated that neuroendocrine axis dysfunction, insulin resistance, oxidative stress, chronic inflammatory response, and gut microbiota dysbiosis are core pathological links associated with it. However, the underlying molecular mechanisms and therapeutic targets of glycolipid metabolism disorder remain to be elucidated. Progress in high-throughput technologies has helped clarify the pathophysiology of glycolipid metabolism disorder. In the present review, we explored the ways and means by which genomics, transcriptomics, proteomics, metabolomics, and gut microbiomics could help identify novel candidate biomarkers for the clinical management of glycolipid metabolism disorder. We also discuss the limitations and recommended future research directions of multi-omics studies on these diseases.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2022 The Authors. Published by Elsevier B.V. on behalf of Research Network of Computational and Structural Biotechnology.', 'doi': '10.1016/j.csbj.2022.10.030\\n10.1038/nrendo.2017.151\\n10.3961/jpmph.19.100\\n10.1038/s41586-020-2338-1\\n10.1155/2019/2958920\\n10.1016/j.jff.2021.104556\\n10.3390/nu12123652\\n10.1007/s00125-019-4938-6\\n10.1016/j.appet.2018.01.015\\n10.1016/j.molmet.2018.12.006\\n10.2337/db11-0785\\n10.1016/j.cmet.2013.12.015\\n10.1016/j.cmet.2015.04.008\\n10.1016/j.metabol.2021.154892\\n10.1038/s41580-021-00390-6\\n10.1038/s41586-019-1797-8\\n10.1016/j.redox.2021.101887\\n10.1038/nrendo.2015.189\\n10.1016/j.celrep.2021.109161\\n10.1007/s13238-020-00814-7\\n10.3390/microorganisms8040527\\n10.1016/j.arr.2021.101346\\n10.1038/s41576-020-0216-1\\n10.1038/nature19806\\n10.1074/jbc.AAC118.006541\\n10.1038/ng1732\\n10.1186/s12881-018-0553-5\\n10.1016/j.numecd.2017.10.012\\n10.1016/j.diabet.2016.05.003\\n10.1186/s12881-018-0614-9\\n10.2337/diabetes.47.3.490\\n10.3390/ijms19072124\\n10.1038/nature05616\\n10.1126/science.1142382\\n10.2337/db08-0047\\n10.2337/db08-0877\\n10.1126/science.1151710\\n10.2147/DMSO.S154978\\n10.1016/s0014-5793(97)00488-2\\n10.1126/science.270.5239.1166\\n10.1007/s00125-013-3031-9\\n10.2337/diab.45.6.825\\n10.1038/s41467-018-04951-w\\n10.1038/nature18642\\n10.1038/s41586-020-2263-3\\n10.1038/nature14177\\n10.1210/er.2013-1084\\n10.1016/j.cmet.2019.03.009\\n10.1016/j.diabres.2015.05.003\\n10.1007/s40291-014-0126-z\\n10.1016/j.diabres.2006.10.021\\n10.2337/dc07-2248\\n10.1016/j.ejim.2011.10.018\\n10.1155/2015/176949\\n10.1210/en.2011-0286\\n10.1007/s40262-015-0307-x\\n10.1097/FPC.0000000000000268\\n10.1007/s00125-018-4675-2\\n10.1016/j.amjcard.2003.08.078\\n10.1001/jama.291.23.2821\\n10.1016/s0021-9150(03)00094-7\\n10.1016/s0021-9150(01)00410-5\\n10.1038/nsmb.2921\\n10.1186/s40246-018-0173-3\\n10.1016/j.cmet.2012.08.010\\n10.1159/000373999\\n10.1016/j.ijcard.2016.11.182\\n10.1159/000484191\\n10.2337/db17-0799\\n10.1073/pnas.0804549105\\n10.2337/dbi19-0016\\n10.1038/nature03076\\n10.1007/s00109-015-1296-9\\n10.1038/nm.3862\\n10.2337/db17-0816\\n10.1038/s41574-019-0260-0\\n10.1038/s41467-020-15529-w\\n10.3390/cells9030659\\n10.1038/nature11928\\n10.1038/srep12453\\n10.1007/s00592-016-0943-0\\n10.3892/ijmm.2018.3625\\n10.3390/cells9061473\\n10.1038/s42255-019-0078-z\\n10.1016/j.yexcr.2019.111753\\n10.2152/jmi.54.289\\n10.1016/j.jpba.2009.11.016\\n10.1016/j.atherosclerosis.2013.02.035\\n10.2337/db15-0881\\n10.1007/s10654-018-0475-8\\n10.1016/j.mce.2014.11.021\\n10.5483/bmbrep.2012.45.12.235\\n10.1016/j.jprot.2013.10.005\\n10.3389/fphar.2021.735410\\n10.3390/cells10112832\\n10.2337/dc15-2251\\n10.1111/1753-0407.12645\\n10.3803/EnM.2019.34.3.234\\n10.1007/s00125-017-4222-6\\n10.1074/jbc.M106703200\\n10.1210/jc.2014-2357\\n10.2337/dc17-1642\\n10.1172/JCI64801\\n10.1056/NEJMc1412427\\n10.2337/dc13-0800\\n10.1007/s00125-016-4069-2\\n10.2337/dc18-0840\\n10.1373/clinchem.2014.228965\\n10.1021/pr300523x\\n10.1016/j.dsx.2018.09.009\\n10.1016/j.arcmed.2017.11.003\\n10.1136/gutjnl-2020-323071\\n10.1038/s41574-019-0156-z\\n10.1038/sj.emboj.7601049\\n10.3389/fphar.2018.01382\\n10.3945/ajcn.117.157107\\n10.1111/obr.12843\\n10.1038/s41575-019-0258-z\\n10.1016/j.cmet.2018.07.001\\n10.1016/j.cell.2018.09.055\\n10.1007/s12020-012-9760-6\\n10.1007/s00125-016-4041-1\\n10.2337/db12-0707\\n10.1038/1881110a0\\n10.3390/ijms22136862\\n10.1007/s11306-019-1484-7\\n10.3389/fnut.2019.00021\\n10.3748/wjg.v21.i29.8787\\n10.1017/S0007114516001045\\n10.5582/ddt.2018.01036\\n10.1038/tpj.2012.43\\n10.1038/nature12198\\n10.1016/j.diabet.2016.04.004\\n10.1186/s12876-015-0330-2\\n10.1136/gutjnl-2014-308778\\n10.1038/nature05414\\n10.1038/nm.4358\\n10.1016/j.chom.2019.07.004\\n10.1038/s41467-019-10189-x\\n10.1007/s00018-019-03434-5\\n10.1038/s41586-020-2928-y\\n10.1016/j.jprot.2020.103823\\n10.1038/nature18646\\n10.1038/s41586-019-1236-x\\n10.1038/s41591-019-0414-6\\n10.1038/nrd.2016.32\\n10.3389/fphys.2011.00094\\n10.1111/jdi.12673\\n10.1016/j.nut.2011.08.013', 'publication_date': datetime.date(2022, 11, 17), 'authors': [{'lastname': 'Fang', 'firstname': 'Xinyi', 'initials': 'X', 'affiliation': \"Institute of Metabolic Diseases, Guang'anmen Hospital, China Academy of Chinese Medical Sciences, Beijing 100053, China.\\nGraduate College, Beijing University of Chinese Medicine, Beijing 100029, China.\"}, {'lastname': 'Miao', 'firstname': 'Runyu', 'initials': 'R', 'affiliation': \"Institute of Metabolic Diseases, Guang'anmen Hospital, China Academy of Chinese Medical Sciences, Beijing 100053, China.\\nGraduate College, Beijing University of Chinese Medicine, Beijing 100029, China.\"}, {'lastname': 'Wei', 'firstname': 'Jiahua', 'initials': 'J', 'affiliation': 'Changchun University of Chinese Medicine, Jilin 130117, China.'}, {'lastname': 'Wu', 'firstname': 'Haoran', 'initials': 'H', 'affiliation': 'Graduate College, Beijing University of Chinese Medicine, Beijing 100029, China.'}, {'lastname': 'Tian', 'firstname': 'Jiaxing', 'initials': 'J', 'affiliation': \"Institute of Metabolic Diseases, Guang'anmen Hospital, China Academy of Chinese Medical Sciences, Beijing 100053, China.\"}]}\n",
      "\n",
      "{'pubmed_id': '36369331', 'title': 'Association between TCF7L2 polymorphism and type 2 diabetes mellitus susceptibility: a case-control study among the Bangladeshi population.', 'keywords': ['Bangladesh', 'Chittagong', 'Comorbidities', 'Diabetes', 'Polymorphism', 'TCF7L2'], 'journal': 'Molecular biology reports', 'abstract': 'Diabetes is a severe health burden for Bangladesh. Genetic polymorphism has been reported to be one of the major risk factors for diabetes in various studies. TCF7L2 (transcription factor 7 like 2) transcripts in the human β-cell have effects on β-cell survival, function, and Wnt signaling activation. This study aimed to evaluate the frequency and association of various polymorphisms namely TCF7L2 rs12255372 and rs7903146 among Bangladeshi patients with T2DM (Type 2 Diabetes Mellitus).\\nThis case-control study included 300 patients with T2DM and 234 healthy individuals from two health facilities in the Chattogram Division of Bangladesh. Anthropometric measurements were assessed using a self-reported, structured, eight-item questionnaire. The polymorphisms were identified by PCR-RFLP and sequencing method.\\nA strong association of T2DM\\xa0with polymorphisms was observed, including rs12255372 (p\\u2009=\\u20090.0004) and rs7903146 (p\\u2009=\\u20090.005). It was observed that the risk genotype at rs12255372 was associated with age (p\\u2009=\\u20090.009), a family history of diabetes (p\\u2009<\\u20090.0001), and HbA1C (p\\u2009<\\u20090.0001). Furthermore, it was found that rs12255372 was substantially associated with hypertension (p\\u2009=\\u20090.03), eye problems (p\\u2009=\\u20090.01), and neurological abnormalities (p\\u2009=\\u20090.02).\\nThis study postulates that TCF7L2 genetic polymorphism is associated with the risk of T2DM among the\\xa0studied Bangladeshi population. The findings should be replicated through more studies with a large number of samples and in different populations.', 'conclusions': 'This study postulates that TCF7L2 genetic polymorphism is associated with the risk of T2DM among the\\xa0studied Bangladeshi population. The findings should be replicated through more studies with a large number of samples and in different populations.', 'methods': None, 'results': 'A strong association of T2DM\\xa0with polymorphisms was observed, including rs12255372 (p\\u2009=\\u20090.0004) and rs7903146 (p\\u2009=\\u20090.005). It was observed that the risk genotype at rs12255372 was associated with age (p\\u2009=\\u20090.009), a family history of diabetes (p\\u2009<\\u20090.0001), and HbA1C (p\\u2009<\\u20090.0001). Furthermore, it was found that rs12255372 was substantially associated with hypertension (p\\u2009=\\u20090.03), eye problems (p\\u2009=\\u20090.01), and neurological abnormalities (p\\u2009=\\u20090.02).', 'copyrights': '© 2022. The Author(s), under exclusive licence to Springer Nature B.V.', 'doi': '10.1007/s11033-022-08081-x\\n10.1016/j.metabol.2019.153952\\n10.1038/nrdp.2015.19\\n10.1159/000356747\\n10.1007/s00125-006-0506-y\\n10.3390/ijms21176275\\n10.2471/BLT.13.128371\\n10.1038/nature05616\\n10.4161/isl.23345\\n10.1172/JCI30706\\n10.1007/s00109-007-0203-4\\n10.1007/s00125-006-0502-2\\n10.2337/db09-1386\\n10.2337/db06-0341\\n10.2337/db06-0643\\n10.1186/1471-2350-9-106\\n10.1007/s00125-007-0618-z\\n10.1038/sj.ejhg.5201773\\n10.1016/j.banm.2021.09.001\\n10.1590/S1415-47572010005000056\\n10.1590/S1415-47572012005000029\\n10.1046/j.1365-2125.2001.00346.x\\n10.4103/2321-0656.193997\\n10.4093/dmj.2015.39.5.395\\n10.7754/clin.lab.2013.130809\\n10.1016/j.metabol.2007.04.012\\n10.1016/j.jgeb.2016.10.004\\n10.1038/ng1960\\n10.2337/db07-0012\\n10.1097/MD.0000000000014288\\n10.1186/1471-2350-9-72\\n10.1186/1471-2350-10-15\\n10.2337/db08-0906\\n10.1086/520599\\n10.1038/ng2043\\n10.1126/science.1142364\\n10.1186/s40200-015-0148-z\\n10.4093/dmj.2015.39.6.512\\n10.2337/db07-0421\\n10.1016/j.gene.2012.09.034\\n10.1186/s13104-021-05598-5\\n10.1186/1472-6823-10-9\\n10.1331/JAPhA.2009.09077\\n10.1016/j.diabres.2010.01.024\\n10.1155/2013/409083\\n10.1016/j.phrs.2015.05.013\\n10.3305/nh.2013.28.sup2.6714\\n10.1111/j.1469-1809.2008.00443.x\\n10.1371/journal.pone.0188382\\n10.1590/0004-2730000003510', 'publication_date': datetime.date(2022, 11, 13), 'authors': [{'lastname': 'Salauddin', 'firstname': 'Asma', 'initials': 'A', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Chakma', 'firstname': 'Kallyan', 'initials': 'K', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Hasan', 'firstname': 'Md Mahbub', 'initials': 'MM', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Akter', 'firstname': 'Farhana', 'initials': 'F', 'affiliation': 'Department of Endocrinology, Chittagong Medical College, Chattogram, 4203, Bangladesh.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh.'}, {'lastname': 'Chowdhury', 'firstname': 'Nowshad Asgar', 'initials': 'NA', 'affiliation': 'Chittagong Diabetic General Hospital, Chattogram, Bangladesh.'}, {'lastname': 'Chowdhury', 'firstname': 'Sumon Rahman', 'initials': 'SR', 'affiliation': 'Chittagong Diabetic General Hospital, Chattogram, Bangladesh.'}, {'lastname': 'Mannan', 'firstname': 'Adnan', 'initials': 'A', 'affiliation': 'Department of Genetic Engineering & Biotechnology, Faculty of Biological Sciences, University of Chittagong, Chattogram, 4331, Bangladesh. adnan.mannan@cu.ac.bd.\\nDisease Biology and Molecular Epidemiology Research Group, Chattogram, Bangladesh. adnan.mannan@cu.ac.bd.'}]}\n",
      "\n",
      "{'pubmed_id': '36364957', 'title': 'Indolepropionic Acid, a Gut Bacteria-Produced Tryptophan Metabolite and the Risk of Type 2 Diabetes and Non-Alcoholic Fatty Liver Disease.', 'keywords': ['NAFLD', 'diet', 'gut microbiota', 'indolepropionic acid', 'insulin', 'type 2 diabetes'], 'journal': 'Nutrients', 'abstract': 'An intricate relationship between gut microbiota, diet, and the human body has recently been extensively investigated. Gut microbiota and gut-derived metabolites, especially, tryptophan derivatives, modulate metabolic and immune functions in health and disease. One of the tryptophan derivatives, indolepropionic acid (IPA), is increasingly being studied as a marker for the onset and development of metabolic disorders, including type 2 diabetes (T2D) and non-alcoholic fatty liver disease (NAFLD). The IPA levels heavily depend on the diet, particularly dietary fiber, and show huge variations among individuals. We suggest that these variations could partially be explained using genetic variants known to be associated with specific diseases such as T2D. In this narrative review, we elaborate on the beneficial effects of IPA in the mitigation of T2D and NAFLD, and further study the putative interactions between IPA and well-known genetic variants (', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': None, 'doi': '10.3390/nu14214695\\n10.1042/bj0170907\\n10.1007/s12031-002-0036-0\\n10.3390/ijms23031222\\n10.1080/19490976.2019.1586038\\n10.1007/s13668-020-00307-3\\n10.1210/endocr/bqaa235\\n10.1038/srep46337\\n10.1038/s41387-018-0046-9\\n10.3389/fendo.2022.841703\\n10.3389/fphar.2019.00877\\n10.1016/S0140-6736(69)90757-0\\n10.1017/S0954422417000117\\n10.1016/S2468-1253(19)30257-2\\n10.1007/s00125-006-0198-3\\n10.1080/13813455.2017.1398262\\n10.1038/nature24661\\n10.3389/fmicb.2020.575586\\n10.1038/s41586-022-04884-x\\n10.3390/nu13103509\\n10.1017/S0007114500001070\\n10.1155/2016/8952520\\n10.1007/s00535-018-1521-8\\n10.1038/s41467-018-05470-4\\n10.3390/ijms222212435\\n10.1128/JB.187.6.2084-2092.2005\\n10.1016/j.chom.2018.05.003\\n10.3390/ijms22189904\\n10.1186/s40168-020-00837-6\\n10.1038/s41467-019-13498-3\\n10.1016/j.immuni.2013.08.003\\n10.1073/pnas.0812874106\\n10.1016/j.immuni.2014.06.014\\n10.1111/nmo.13178\\n10.1038/s41467-021-23460-x\\n10.1016/j.ajpath.2018.01.011\\n10.1080/19490976.2020.1788899\\n10.1186/s40779-022-00397-w\\n10.3390/metabo10050181\\n10.1038/s41467-020-14676-4\\n10.1016/j.xcrm.2021.100246\\n10.3389/fimmu.2021.653208\\n10.1038/s42003-020-01383-5\\n10.3389/fmed.2021.648259\\n10.1136/gutjnl-2021-324053\\n10.1016/j.ebiom.2019.11.051\\n10.1007/s00253-019-10156-y\\n10.1016/j.nutres.2020.03.005\\n10.1128/mSystems.00313-20\\n10.1002/mnfr.201900580\\n10.1172/jci.insight.145207\\n10.2337/dc21-0099\\n10.3390/microorganisms8010111\\n10.1093/jn/nxy053\\n10.1007/s11306-011-0320-5\\n10.1093/cdn/nzz040.P20-003-19\\n10.1002/mnfr.202100349\\n10.1111/febs.16470\\n10.1002/oby.22870\\n10.1093/cdn/nzaa062_016\\n10.1186/s13073-020-00806-6\\n10.2337/dc21-1705\\n10.1093/gerona/glt152\\n10.1371/journal.pone.0198156\\n10.2337/dc19-2533\\n10.3390/metabo12010069\\n10.3390/metabo11040228\\n10.1080/13813455.2021.2024577\\n10.1152/ajpgi.00105.2020\\n10.1016/j.bmcl.2006.05.007\\n10.3390/nu11030591\\n10.1038/ng.2982\\n10.1007/s00125-007-0656-6\\n10.1038/oby.2008.618\\n10.2337/diabetes.51.8.2581\\n10.1007/s00125-012-2752-5\\n10.2337/db08-1607\\n10.1038/ng.2383\\n10.1038/s41598-020-65051-8\\n10.1016/j.molmet.2020.101092\\n10.1111/joim.12892\\n10.3390/nu14112187\\n10.1002/hep.26342\\n10.1111/ijpo.270\\n10.1161/CIRCRESAHA.122.321253\\n10.1016/j.jvs.2017.09.029\\n10.1016/S0041-1345(02)03616-3\\n10.1038/s12276-019-0304-5\\n10.1002/1097-4644(20010601)81:3<507::AID-JCB1064>3.0.CO;2-M\\n10.3390/nu12103100\\n10.3390/cancers12092411\\n10.1039/D1FO02181H\\n10.1136/gutjnl-2019-319664\\n10.1096/fj.14-259515\\n10.1007/s11356-022-21075-3\\n10.1016/j.tox.2021.152996\\n10.1002/smll.201905233\\n10.18999/nagjms.79.4.477', 'publication_date': datetime.date(2022, 11, 12), 'authors': [{'lastname': 'Sehgal', 'firstname': 'Ratika', 'initials': 'R', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.'}, {'lastname': 'de Mello', 'firstname': 'Vanessa D', 'initials': 'VD', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.'}, {'lastname': 'Männistö', 'firstname': 'Ville', 'initials': 'V', 'affiliation': 'Department of Medicine, University of Eastern Finland, Kuopio University Hospital, 70211 Kuopio, Finland.'}, {'lastname': 'Lindström', 'firstname': 'Jaana', 'initials': 'J', 'affiliation': 'Public Health Prevention Unit, Finnish Institute for Health and Welfare, 00271 Helsinki, Finland.'}, {'lastname': 'Tuomilehto', 'firstname': 'Jaakko', 'initials': 'J', 'affiliation': 'Public Health Prevention Unit, Finnish Institute for Health and Welfare, 00271 Helsinki, Finland.\\nDepartment of Public Health, University of Helsinki, 00014 Helsinki, Finland.\\nDiabetes Research Group, King Abdulaziz University, Jeddah 21589, Saudi Arabia.'}, {'lastname': 'Pihlajamäki', 'firstname': 'Jussi', 'initials': 'J', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.\\nDepartment of Medicine, Endocrinology and Clinical Nutrition, Kuopio University Hospital, 70210 Kuopio, Finland.'}, {'lastname': 'Uusitupa', 'firstname': 'Matti', 'initials': 'M', 'affiliation': 'Institute of Public Health and Clinical Nutrition, Department of Clinical Nutrition, University of Eastern Finland, 70211 Kuopio, Finland.'}]}\n",
      "\n",
      "{'pubmed_id': '36360783', 'title': 'Decoding Diabetes Biomarkers and Related Molecular Mechanisms by Using Machine Learning, Text Mining, and Gene Expression Analysis.', 'keywords': ['bioinformatics', 'diabetes', 'gene expression', 'protein–protein interaction network', 'text mining'], 'journal': 'International journal of environmental research and public health', 'abstract': 'The molecular basis of diabetes mellitus is yet to be fully elucidated. We aimed to identify the most frequently reported and differential expressed genes (DEGs) in diabetes by using bioinformatics approaches. Text mining was used to screen 40,225 article abstracts from diabetes literature. These studies highlighted 5939 diabetes-related genes spread across 22 human chromosomes, with 112 genes mentioned in more than 50 studies. Among these genes, ', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': None, 'doi': '10.3390/ijerph192113890\\n10.1038/414782a\\n10.2522/ptj.20080008\\n10.1371/journal.pone.0243270\\n10.3389/fgene.2020.00734\\n10.3390/cancers13215268\\n10.1097/MD.0000000000020338\\n10.1007/s11011-019-00465-6\\n10.1007/s12551-018-0490-8\\n10.1097/MD.0000000000017504\\n10.1111/brv.12457\\n10.1186/s12874-019-0681-4\\n10.1016/j.cmpb.2019.04.008\\n10.3390/e23010018\\n10.1016/j.ijmedinf.2021.104453\\n10.1093/nar/gks1193\\n10.1093/bioinformatics/bty721\\n10.1093/bioinformatics/btz931\\n10.1093/bioinformatics/btr261\\n10.1093/nar/gkq973\\n10.36462/H.BioSci.20195\\n10.1093/bioinformatics/btp616\\n10.1172/JCI200216857\\n10.1172/JCI200522365\\n10.1186/s13098-020-00557-9\\n10.1093/eurheartj/ehab724.3196\\n10.1007/s12013-007-0025-6\\n10.1074/jbc.R109.040543\\n10.1016/j.jgg.2017.11.007\\n10.1016/j.molmet.2014.12.007\\n10.1016/j.biopha.2017.03.056\\n10.1016/j.biopha.2017.03.102\\n10.2337/db09-1187\\n10.1530/JOE-17-0002\\n10.1038/ng2068\\n10.1038/s41598-019-54545-9\\n10.1194/jlr.M080382\\n10.1038/s41419-020-03312-0\\n10.1016/j.reprotox.2021.10.008\\n10.3390/ijms22136924\\n10.1016/j.str.2008.10.018\\n10.1038/ncomms3863\\n10.1016/j.cellsig.2015.09.016\\n10.1002/jcb.20343\\n10.1186/s12916-020-1503-6\\n10.2147/DMSO.S345081\\n10.1016/j.ejphar.2022.174956\\n10.1210/jc.2009-0384\\n10.1007/s001250051573\\n10.1159/000295775\\n10.3389/fimmu.2015.00493\\n10.1039/b914182k\\n10.1681/ASN.2005080859\\n10.3390/cells8070751\\n10.2337/db20-0038\\n10.1210/jc.2018-02094\\n10.1371/journal.pone.0193684\\n10.1002/humu.23639\\n10.1093/hmg/ddx262\\n10.1371/journal.pone.0234132\\n10.1159/000524849\\n10.3390/cells9030730\\n10.1016/j.jid.2017.11.038\\n10.3390/cells8020178\\n10.1038/nm1185', 'publication_date': datetime.date(2022, 11, 12), 'authors': [{'lastname': 'Elsherbini', 'firstname': 'Amira M', 'initials': 'AM', 'affiliation': 'Department of Oral Biology, Faculty of Dentistry, Mansoura University, Mansoura 35116, Egypt.'}, {'lastname': 'Alsamman', 'firstname': 'Alsamman M', 'initials': 'AM', 'affiliation': 'Agricultural Genetic Engineering Research Institute, Agricultural Research Center, Giza 12619, Egypt.'}, {'lastname': 'Elsherbiny', 'firstname': 'Nehal M', 'initials': 'NM', 'affiliation': 'Department of Pharmaceutical Chemistry, Faculty of Pharmacy, University of Tabuk, Tabuk 71491, Saudi Arabia.\\nDepartment of Biochemistry, Faculty of Pharmacy, Mansoura University, Mansoura 35116, Egypt.'}, {'lastname': 'El-Sherbiny', 'firstname': 'Mohamed', 'initials': 'M', 'affiliation': 'Department of Basic Medical Sciences, College of Medicine, AlMaarefa University, Riyadh 71666, Saudi Arabia.\\nDepartment of Anatomy, Mansoura Faculty of Medicine, Mansoura University, Mansoura 35116, Egypt.'}, {'lastname': 'Ahmed', 'firstname': 'Rehab', 'initials': 'R', 'affiliation': 'Department of Natural Products and Alternative Medicine, Faculty of Pharmacy, University of Tabuk, Tabuk 71491, Saudi Arabia.\\nDepartment of Pharmaceutics, Faculty of Pharmacy, University of Khartoum, Khartoum 11111, Sudan.'}, {'lastname': 'Ebrahim', 'firstname': 'Hasnaa Ali', 'initials': 'HA', 'affiliation': 'Department of Basic Medical Sciences, College of Medicine, Princess Nourah bint Abdulrahman University, Riyadh 11671, Saudi Arabia.'}, {'lastname': 'Bakkach', 'firstname': 'Joaira', 'initials': 'J', 'affiliation': 'Biomedical Genomics and Oncogenetics Research Laboratory, Faculty of Sciences and Techniques of Tangier, Abdelmalek Essaâdi University Morocco, Tétouan 93000, Morocco.'}]}\n",
      "\n",
      "{'pubmed_id': '36358989', 'title': 'FGF9 Promotes Expression of HAS2 in Palatal Elevation via the Wnt/β-Catenin/TCF7L2 Pathway.', 'keywords': ['FGF9', 'HAS2', 'TCF7L2', 'Wnt/β-catenin pathway', 'hyaluronic acid'], 'journal': 'Biomolecules', 'abstract': 'Using Ddx4-Cre mice, we generated the \\nFGF9 promotes HAS2 expression via Wnt/β-catenin/TCF7L2 pathway with TCF7L2 activating transcription of ', 'conclusions': None, 'methods': None, 'results': '', 'copyrights': None, 'doi': '10.3390/biom12111639\\n10.3389/fphys.2016.00067\\n10.1002/ajmg.a.31965\\n10.3109/08977199509036882\\n10.1038/ng.316\\n10.1002/(SICI)1097-0177(199909)216:1<72::AID-DVDY9>3.0.CO;2-9\\n10.1177/0022034518785336\\n10.1242/dev.128.11.2095\\n10.3389/fphys.2021.653040\\n10.1177/0022034517703580\\n10.1038/s41368-021-00122-8\\n10.1016/j.ydbio.2019.09.004\\n10.1182/blood-2014-07-590513\\n10.1080/15476278.2020.1735239\\n10.3389/fphys.2013.00010\\n10.1002/ar.24263\\n10.3389/fcell.2019.00340\\n10.1177/0022034519872478\\n10.3390/biom10111525\\n10.1530/REP-16-0240\\n10.3389/fcvm.2021.701279\\n10.1371/journal.pone.0241281\\n10.1016/j.biopha.2018.04.026\\n10.1002/bdrc.20180\\n10.1242/dev.134577\\n10.1002/lio2.214\\n10.3390/cells9030608\\n10.1016/j.mce.2016.05.023\\n10.1016/j.ydbio.2016.04.005\\n10.1007/BF00428098\\n10.1016/j.ydbio.2016.01.008\\n10.1042/bj3480029\\n10.1074/jbc.274.35.25085\\n10.3389/fcell.2020.00264\\n10.1242/dmm.037051\\n10.1242/dev.158048\\n10.1126/science.aat1178\\n10.1016/j.cell.2012.05.012\\n10.1038/s41598-018-20641-5\\n10.1126/scisignal.aat9519\\n10.1016/j.devcel.2008.04.015\\n10.1016/j.cellsig.2017.08.008\\n10.15252/embj.201798873\\n10.1186/s13098-019-0449-3\\n10.1210/me.2003-0225\\n10.1016/j.semcdb.2009.09.002\\n10.3892/etm.2019.8308\\n10.1016/j.chemosphere.2020.126889\\n10.1002/anie.200462552', 'publication_date': datetime.date(2022, 11, 12), 'authors': [{'lastname': 'Sun', 'firstname': 'Yidan', 'initials': 'Y', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Ying', 'firstname': 'Xiyu', 'initials': 'X', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Li', 'firstname': 'Ruomei', 'initials': 'R', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Weng', 'firstname': 'Mengjia', 'initials': 'M', 'affiliation': 'Department of Stomatology, Huadong Hospital, Fudan University, Shanghai 200040, China.'}, {'lastname': 'Shi', 'firstname': 'Jiajun', 'initials': 'J', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}, {'lastname': 'Chen', 'firstname': 'Zhenqi', 'initials': 'Z', 'affiliation': \"Department of Orthodontics, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, College of Stomatology, Shanghai Jiao Tong University, National Center for Stomatology, National Clinical Research Center for Oral Diseases, Shanghai Key Laboratory of Stomatology, Shanghai Research Institute of Stomatology, Shanghai 200011, China.\"}]}\n",
      "\n",
      "{'pubmed_id': '36344976', 'title': 'CHRNB2 represses pancreatic cancer migration and invasion via inhibiting β-catenin pathway.', 'keywords': ['CHRNB2', 'Lymph node ratio', 'Pancreatic cancer', 'Weighted gene co-expression network analysis', 'β-catenin pathway'], 'journal': 'Cancer cell international', 'abstract': 'Pancreatic cancer is one of the most lethal disease with highly fatal and aggressive properties. Lymph node ratio (LNR), the ratio of the number of metastatic lymph nodes to the total number of examined lymph nodes, is an important index to assess lymphatic metastasis and predict prognosis, but the molecular mechanism underlying high LNR was unclear.\\nGene expression and clinical information data of pancreatic cancer were obtained from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO). Patients in TCGA were averagely divided into low and high LNR groups. Then, Weighted Gene Co-expression Network Analysis (WGCNA) was utilized to build co-expression network to explore LNR-related modules and hub genes. GO and KEGG analysis was performed to find key pathways related to lymph node metastasis. Next, GSE101448 and the overall survival data in TCGA was employed to further select significant genes from hub genes. Considering the key role of CHRNB2 in LNR and survival, gene set enrichment analysis (GSEA) was applied to find pathways related to CHRNB2 expression in pancreatic cancer. The contribution of CHRNB2 to migrative and invasive ability of pancreatic cancer cells was confirmed by Transwell assays. We finally explored the role of CHRNB2 in EMT and β-catenin pathway via Western Blot.\\nHigh LNR was significantly related to high T stages and poor prognosis. In WGCNA, 14 hub genes (COL5A1, FN1, THBS2, etc.) were positively related to high LNR, 104 hub genes (FFAR1, SCG5, TMEM63C, etc.) were negatively related to high LNR. After taking the intersection with GSE101448, 13 genes (CDK5R2, SYT7, CACNA2D2, etc.) which might prevent lymph node metastasis were further selected. Among them, CHRNB2 showed the strongest relationship with long survival. Moreover, CHRNB2 also negatively related to the T stages and LNR. Next, knockdown of CHRNB2 expression could acetylcholine (ACh)-independently increase the migration and invasion of pancreatic cancer cells, while CHRNB2 overexpression ACh-independently decrease the migration and invasion of pancreatic cancer cells. For exploring the underlying mechanism, CHRNB2 downregulated β-catenin pathway might through controlling its upstream regulators such as SOX6, SRY, SOX17, and TCF7L2.\\nCHRNB2 negatively relates to lymph node metastasis in pancreatic cancer patients. CHRNB2 could inhibit β-catenin pathway, EMT, migration and invasion of pancreatic cancer cells via ACh-independent mechanism.', 'conclusions': None, 'methods': None, 'results': 'High LNR was significantly related to high T stages and poor prognosis. In WGCNA, 14 hub genes (COL5A1, FN1, THBS2, etc.) were positively related to high LNR, 104 hub genes (FFAR1, SCG5, TMEM63C, etc.) were negatively related to high LNR. After taking the intersection with GSE101448, 13 genes (CDK5R2, SYT7, CACNA2D2, etc.) which might prevent lymph node metastasis were further selected. Among them, CHRNB2 showed the strongest relationship with long survival. Moreover, CHRNB2 also negatively related to the T stages and LNR. Next, knockdown of CHRNB2 expression could acetylcholine (ACh)-independently increase the migration and invasion of pancreatic cancer cells, while CHRNB2 overexpression ACh-independently decrease the migration and invasion of pancreatic cancer cells. For exploring the underlying mechanism, CHRNB2 downregulated β-catenin pathway might through controlling its upstream regulators such as SOX6, SRY, SOX17, and TCF7L2.', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1186/s12935-022-02768-8\\n10.3390/ijms23031231\\n10.1172/JCI146431\\n10.3390/ijms20174242\\n10.1038/s41573-022-00410-w', 'publication_date': datetime.date(2022, 11, 8), 'authors': [{'lastname': 'Qin', 'firstname': 'Cheng', 'initials': 'C', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Li', 'firstname': 'Tianhao', 'initials': 'T', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.\\nDivision of Plastic Surgery, Department of Surgery, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Wang', 'firstname': 'Yuanyang', 'initials': 'Y', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Zhao', 'firstname': 'Bangbo', 'initials': 'B', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Li', 'firstname': 'Zeru', 'initials': 'Z', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Li', 'firstname': 'Tianyu', 'initials': 'T', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Yang', 'firstname': 'Xiaoying', 'initials': 'X', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Zhao', 'firstname': 'Yutong', 'initials': 'Y', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China.'}, {'lastname': 'Wang', 'firstname': 'Weibin', 'initials': 'W', 'affiliation': 'Department of General Surgery, State Key Laboratory of Complex Severe and Rare Diseases, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China. wwb_xh@163.com.'}]}\n",
      "\n",
      "{'pubmed_id': '36300877', 'title': 'Clinical signs of type 1 diabetes are associated with type 2 diabetes marker transcription factor 7-like 2 polymorphism.', 'keywords': ['Anti-islet autoantibodies', 'Transcription factor 7-like 2 gene', 'Type 1 diabetes'], 'journal': 'Journal of diabetes investigation', 'abstract': 'We aimed to assess the distribution of transcription factor 7-like 2 gene TCF7L2 (rs7903146) polymorphism and to find possible associations between TCF7L2 and the characteristics of type 1 diabetes.\\nWe studied 190 newly diagnosed type 1 diabetes patients (median age 12.7\\u2009years, range 2.0-72.5) and 246 controls (median age 23.8\\u2009years, range 1.4-81.5) for TCF7L2 single nucleotide polymorphism. We determined anti-islet autoantibodies, random C-peptide levels, diabetes associated HLA DR/DQ haplotypes and genotypes in all patients.\\nThere were no differences in the distribution of TCF7L2 single nucleotide polymorphism between patients and controls. However, patients with in type 1 diabetes, after adjusting for age and sex, subjects carrying C allele were at risk for a C-peptide level lower than 0.5\\u2009nmol/L (OR 5.65 [95% CI: 1.14-27.92]) and for zinc transporter 8 autoantibody positivity (5.22 [1.34-20.24]). Participants without T allele were associated with a higher level of islet antigen-2 autoantibodies (3.51 [1.49-8.27]) and zinc transporter 8 autoantibodies (2.39 [1.14-4.99]).\\nThe connection of TCF7L2 polymorphism with zinc transporter 8 and islet antigen-2 autoantibodies and C-peptide levels in patients supports the viewpoint that TCF7L2 is associated with the clinical signs and autoimmune characteristics of type 1 diabetes. The mechanisms of the interaction between the TCF7L2 risk genotype and anti-islet autoantibodies need to be studied further.', 'conclusions': None, 'methods': None, 'results': 'There were no differences in the distribution of TCF7L2 single nucleotide polymorphism between patients and controls. However, patients with in type 1 diabetes, after adjusting for age and sex, subjects carrying C allele were at risk for a C-peptide level lower than 0.5\\u2009nmol/L (OR 5.65 [95% CI: 1.14-27.92]) and for zinc transporter 8 autoantibody positivity (5.22 [1.34-20.24]). Participants without T allele were associated with a higher level of islet antigen-2 autoantibodies (3.51 [1.49-8.27]) and zinc transporter 8 autoantibodies (2.39 [1.14-4.99]).', 'copyrights': '© 2022 The Authors. Journal of Diabetes Investigation published by Asian Association for the Study of Diabetes (AASD) and John Wiley & Sons Australia, Ltd.', 'doi': '10.1111/jdi.13933', 'publication_date': datetime.date(2022, 10, 28), 'authors': [{'lastname': 'Ergür', 'firstname': 'Efe', 'initials': 'E', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Ergür', 'firstname': 'Ege', 'initials': 'E', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Alnek', 'firstname': 'Kristi', 'initials': 'K', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Metsküla', 'firstname': 'Kaja', 'initials': 'K', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}, {'lastname': 'Peet', 'firstname': 'Aleksandr', 'initials': 'A', 'affiliation': \"Department of Pediatrics, Institute of Clinical Medicine, University of Tartu, Tartu, Estonia.\\nChildren's Clinic of Tartu University Hospital, Tartu, Estonia.\"}, {'lastname': 'Lubi', 'firstname': 'Maire', 'initials': 'M', 'affiliation': 'Department of Internal Medicine, Institute of Clinical Medicine, University of Tartu, Tartu, Estonia.\\nInternal Medicine Clinic of Tartu University Hospital, Tartu, Estonia.'}, {'lastname': 'Heilman', 'firstname': 'Kaire', 'initials': 'K', 'affiliation': \"Tallinn Children's Hospital, Tallinn, Estonia.\"}, {'lastname': 'Uibo', 'firstname': 'Raivo', 'initials': 'R', 'affiliation': 'Department of Immunology, Institute of Bio- and Translational Medicine, University of Tartu, Tartu, Estonia.'}]}\n",
      "\n",
      "{'pubmed_id': '36295578', 'title': 'ITGA2 Gene Polymorphism Is Associated with Type 2 Diabetes Mellitus in the Kazakhstan Population.', 'keywords': ['BglII polymorphism in ITGA2 gene', 'diabetic retinopathy', 'type 2 diabetes mellitus', 'α2β1 integrin receptor'], 'journal': 'Medicina (Kaunas, Lithuania)', 'abstract': '', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': None, 'doi': '10.3390/medicina58101416\\n10.1038/nrendo.2017.151\\n10.1172/jci.insight.93751\\n10.1016/j.visres.2017.03.006\\n10.1016/j.preteyeres.2013.02.001\\n10.1038/nrdp.2016.12\\n10.1016/j.pop.2015.05.005\\n10.2337/diab.46.11.1829\\n10.1046/j.1464-5491.2002.00820.x\\n10.1038/s41581-020-0278-5\\n10.21037/aes.2018.08.04\\n10.1093/ndt/gfl641\\n10.1097/IIO.0b013e31819fd5d7\\n10.1155/2010/861291\\n10.12809/hkmj164869\\n10.1371/journal.pone.0094160\\n10.3109/02713683.2014.932386\\n10.1007/s00125-015-3569-9\\n10.3109/02713683.2015.1084642\\n10.1155/2016/5305282\\n10.2337/diabetes.50.6.1505\\n10.15171/jnp.2018.54\\n10.1182/blood.V92.7.2382\\n10.3132/dvdr.2005.001\\n10.1186/s12933-018-0763-3\\n10.1055/s-0035-1556731\\n10.5858/2002-126-0305-IOPCRP\\n10.1182/blood.V95.5.1560.005k43_1560_1564\\n10.1007/s10038-003-0060-0\\n10.1080/08820538.2019.1632354\\n10.2147/TACG.S31979\\n10.1182/blood.V89.6.1939\\n10.1182/blood.V97.6.1721\\n10.17513/spno.29283\\n10.1177/1099800420977685\\n10.4172/2155-6156.1000223', 'publication_date': datetime.date(2022, 10, 28), 'authors': [{'lastname': 'Magazova', 'firstname': 'Aizhan', 'initials': 'A', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.\\nDepartment of Biochemistry, Asfendiyarov Kazakh National Medical University, Almaty 050012, Kazakhstan.'}, {'lastname': 'Ashirbekov', 'firstname': 'Yeldar', 'initials': 'Y', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.'}, {'lastname': 'Abaildayev', 'firstname': 'Arman', 'initials': 'A', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.'}, {'lastname': 'Satken', 'firstname': 'Kantemir', 'initials': 'K', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.'}, {'lastname': 'Balmukhanova', 'firstname': 'Altynay', 'initials': 'A', 'affiliation': 'Department of Health Policy and Organization, Al-Farabi Kazakh National University, Almaty 050040, Kazakhstan.'}, {'lastname': 'Akanov', 'firstname': 'Zhanay', 'initials': 'Z', 'affiliation': 'City Center of Diabetes AAA Medical Clinic, Almaty 050040, Kazakhstan.'}, {'lastname': 'Jainakbayev', 'firstname': 'Nurlan', 'initials': 'N', 'affiliation': 'Department of Public Health, Kazakh-Russian Medical University, Almaty 050004, Kazakhstan.'}, {'lastname': 'Balmukhanova', 'firstname': 'Aigul', 'initials': 'A', 'affiliation': 'International Medical School, Caspian University, Almaty 050000, Kazakhstan.'}, {'lastname': 'Sharipov', 'firstname': 'Kamalidin', 'initials': 'K', 'affiliation': 'Aitkhozhin Institute of Molecular Biology and Biochemistry, Almaty 050012, Kazakhstan.\\nDepartment of Biochemistry, Asfendiyarov Kazakh National Medical University, Almaty 050012, Kazakhstan.'}]}\n",
      "\n",
      "{'pubmed_id': '36282337', 'title': 'Milder loss of insulin-containing islets in individuals with type 1 diabetes and type 2 diabetes-associated TCF7L2 genetic variants.', 'keywords': ['African American', 'Genetics', 'Heterogeneity', 'Insulin-containing cells', 'Pancreas', 'Phenotype', 'Precision medicine', 'Race', 'TCF7L2', 'Type 1 diabetes'], 'journal': 'Diabetologia', 'abstract': 'TCF7L2 variants are the strongest genetic risk factor for type 2 diabetes. In individuals with type 1 diabetes, these variants are associated with a higher C-peptide AUC, a lower glucose AUC during an OGTT, single autoantibody positivity near diagnosis, particularly in individuals older than 12 years of age, and a lower frequency of type 1 diabetes-associated HLA genotypes. Based on initial observations from clinical cohorts, we tested the hypothesis that type 2 diabetes-predisposing TCF7L2 genetic variants are associated with a higher percentage of residual insulin-containing cells (ICI%) in pancreases of donors with type 1 diabetes, by examining genomic data and pancreatic tissue samples from the Network for Pancreatic Organ donors with Diabetes (nPOD) programme.\\nWe analysed nPOD donors with type 1 diabetes (n=110; mean±SD age at type 1 diabetes onset 12.2±7.9 years, mean±SD diabetes duration 15.3±13.7 years, 53% male, 80% non-Hispanic White, 12.7% African American, 7.3% Hispanic) using data pertaining to residual beta cell number; quantified islets containing insulin-positive beta cells in pancreatic tissue sections; and expressed these values as a percentage of the total number of islets from each donor (mean ± SD ICI% 9.8±21.5, range 0-92.2).\\nDonors with a high ICI% (≥5) (n=30; 27%) vs a low ICI% (<5) (n=80; 73%) were older at onset (15.3±6.9 vs 11.1±8 years, p=0.013), had a shorter diabetes duration at donor tissue procurement (7.0±7.4 vs 18.5±14.3 years, p<0.001), a higher African ancestry score (0.2±0.3 vs 0.1±0.2, p=0.043) and a lower European ancestry score (0.7±0.3 vs 0.9±0.3, p=0.023). After adjustment for age of onset (p=0.105), diabetes duration (p<0.001), BMI z score (p=0.145), sex (p=0.351) and African American race (p=0.053), donors with the TCF7L2 rs7903146 T allele (TC or TT, 45.5%) were 2.93 times (95% CI 1.02, 8.47) more likely to have a high ICI% than those without it (CC) (p=0.047).\\nOverall, these data support the presence of a type 1 diabetes endotype associated with a genetic factor that predisposes to type 2 diabetes, with donors in this category exhibiting less severe beta cell loss. It is possible that in these individuals the disease pathogenesis may include mechanisms associated with type 2 diabetes and thus this may provide an explanation for the poor response to immunotherapies to prevent type 1 diabetes or its progression in a subset of individuals. If so, strategies that target both type 1 diabetes and type 2 diabetes-associated factors when they are present may increase the success of prevention and treatment in these individuals.', 'conclusions': None, 'methods': None, 'results': 'Donors with a high ICI% (≥5) (n=30; 27%) vs a low ICI% (<5) (n=80; 73%) were older at onset (15.3±6.9 vs 11.1±8 years, p=0.013), had a shorter diabetes duration at donor tissue procurement (7.0±7.4 vs 18.5±14.3 years, p<0.001), a higher African ancestry score (0.2±0.3 vs 0.1±0.2, p=0.043) and a lower European ancestry score (0.7±0.3 vs 0.9±0.3, p=0.023). After adjustment for age of onset (p=0.105), diabetes duration (p<0.001), BMI z score (p=0.145), sex (p=0.351) and African American race (p=0.053), donors with the TCF7L2 rs7903146 T allele (TC or TT, 45.5%) were 2.93 times (95% CI 1.02, 8.47) more likely to have a high ICI% than those without it (CC) (p=0.047).', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1007/s00125-022-05818-y', 'publication_date': datetime.date(2022, 10, 26), 'authors': [{'lastname': 'Redondo', 'firstname': 'Maria J', 'initials': 'MJ', 'affiliation': \"Department of Pediatrics, Baylor College of Medicine, Texas Children's Hospital, Houston, TX, USA. redondo@bcm.edu.\"}, {'lastname': 'Richardson', 'firstname': 'Sarah J', 'initials': 'SJ', 'affiliation': 'Islet Biology Group (IBEx), Exeter Centre of Excellence for Diabetes Research (EXCEED), University of Exeter College of Medicine and Health, Exeter, UK. s.richardson@exeter.ac.uk.'}, {'lastname': 'Perry', 'firstname': 'Daniel', 'initials': 'D', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}, {'lastname': 'Minard', 'firstname': 'Charles G', 'initials': 'CG', 'affiliation': 'Institute for Clinical and Translational Research, Baylor College of Medicine, Houston, TX, USA.'}, {'lastname': 'Carr', 'firstname': 'Alice L J', 'initials': 'ALJ', 'affiliation': 'Islet Biology Group (IBEx), Exeter Centre of Excellence for Diabetes Research (EXCEED), University of Exeter College of Medicine and Health, Exeter, UK.'}, {'lastname': 'Brusko', 'firstname': 'Todd', 'initials': 'T', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}, {'lastname': 'Kusmartseva', 'firstname': 'Irina', 'initials': 'I', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}, {'lastname': 'Pugliese', 'firstname': 'Alberto', 'initials': 'A', 'affiliation': 'Diabetes Research Institute, Department of Medicine, Division of Endocrinology, Department of Microbiology and Immunology, Miller School of Medicine, University of Miami, Miami, FL, USA.'}, {'lastname': 'Atkinson', 'firstname': 'Mark A', 'initials': 'MA', 'affiliation': 'Departments of Pathology and Pediatrics, University of Florida Diabetes Institute, Gainesville, FL, USA.'}]}\n",
      "\n",
      "{'pubmed_id': '36263318', 'title': '', 'keywords': ['TCF7L2 gene', 'adipose tissue', 'postprandial hypertriglyceridemia', 'prediabetes', 'type 2 diabetes mellitus'], 'journal': 'Frontiers in endocrinology', 'abstract': 'Postprandial lipid responses to a standardized fat challenge test were performed in 620 Asian Indian subjects (310 with NGT and 310 with T2DM/prediabetes) and compared between the risk and wild genotypes of the rs7903146 \\nT allele of rs7903146 \\nThere is significant PPTg dysmetabolism associated with the risk allele of rs7903146 polymorphism as well as adipocyte expression of ', 'conclusions': 'There is significant PPTg dysmetabolism associated with the risk allele of rs7903146 polymorphism as well as adipocyte expression of ', 'methods': None, 'results': 'T allele of rs7903146 ', 'copyrights': 'Copyright © 2022 Madhu, Mishra, Mannar, Aslam, Banerjee and Agrawal.', 'doi': '10.3389/fendo.2022.973718\\n10.1186/1471-2350-10-15\\n10.2337/db09-1386\\n10.1007/s00125-006-0502-2\\n10.1038/ng1732\\n10.2337/db11-0550\\n10.1016/j.metabol.2007.04.012\\n10.1007/s00125-006-0507-x\\n10.1007/s11892-008-0026-x\\n10.1056/NEJMoa062418\\n10.1172/JCI30706\\n10.2337/db17-0318\\n10.1016/S0065-230X(08)60783-6\\n10.1016/0092-8674(92)90630-U\\n10.1155/2011/567305\\n10.1101/003202\\n10.2337/db12-0365\\n10.1074/jbc.M411487200\\n10.1016/j.cell.2012.10.053\\n10.1371/journal.pone.0043390\\n10.1038/srep43128\\n10.3945/jn.108.096461\\n10.1007/s13410-018-0678-2\\n10.2337/diabetes.48.6.1258\\n10.7326/0003-4819-131-1-199907060-00006\\n10.1016/j.nut.2005.01.009\\n10.1016/0168-8227(94)90269-0\\n10.1016/j.jacl.2017.08.001\\n10.1371/journal.pone.0145730\\n10.1111/joim.12540\\n10.1007/s13410-018-0678-2\\n10.1111/ddg.13528\\n10.2337/dc17-S005\\n10.1016/j.diabres.2008.01.016\\n10.1006/meth.2001.1262\\n10.2337/db06-0474\\n10.1007/s10038-007-0231-5\\n10.1016/j.metabol.2007.04.012\\n10.1016/j.atherosclerosissup.2010.03.002\\n10.1111/j.1469-1809.2008.00443.x\\n10.1002/hep.22659\\n10.3967/bes2016.108\\n10.2337/db06-0338\\n10.1186/s13098-018-0338-1\\n10.1016/j.bbrc.2008.03.006\\n10.1007/s00125-007-0850-6\\n10.1016/j.metabol.2008.04.016\\n10.1007/s00125-020-05292-4\\n10.1210/jc.2016-1594\\n10.1016/j.molmet.2019.03.003\\n10.1210/jcem.86.11.7992\\n10.1371/journal.pone.0000812\\n10.1371/journal.pone.0237974\\n10.1093/jn/134.1.205\\n10.3390/biom10091230\\n10.3390/biom10040601\\n10.1080/21623945.2020.1776082\\n10.1002/oby.21377\\n10.1371/journal.pone.0018284\\n10.1210/jc.2016-1594\\n10.1016/j.jacc.2011.02.038\\n10.1155/2012/983814', 'publication_date': datetime.date(2022, 10, 21), 'authors': [{'lastname': 'Madhu', 'firstname': 'Sri Venkata', 'initials': 'SV', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Mishra', 'firstname': 'Brijesh Kumar', 'initials': 'BK', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Mannar', 'firstname': 'Velmurugan', 'initials': 'V', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Aslam', 'firstname': 'Mohd', 'initials': 'M', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Banerjee', 'firstname': 'Basudev', 'initials': 'B', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}, {'lastname': 'Agrawal', 'firstname': 'Vivek', 'initials': 'V', 'affiliation': 'University College of Medical Sciences, University of Delhi, Delhi, India.'}]}\n",
      "\n",
      "{'pubmed_id': '36217307', 'title': 'Regulation of the THRA gene, encoding the thyroid hormone nuclear receptor TRα1, in intestinal lesions.', 'keywords': ['THRA', 'TRα1', 'colon cancer', 'intestinal organoids', 'thyroid hormone nuclear receptor'], 'journal': 'Molecular oncology', 'abstract': 'The THRA gene, encoding the thyroid hormone nuclear receptor TRα1, is expressed in an increasing gradient at the bottom of intestinal crypts, overlapping with high Wnt and Notch activities. Importantly, THRA is upregulated in colorectal cancers, particularly in the high-Wnt molecular subtype. The basis of this specific and/or altered expression pattern has remained unknown. To define the mechanisms controlling THRA transcription and TRα1 expression, we used multiple in\\u2009vitro and ex\\u2009vivo approaches. Promoter analysis demonstrated that transcription factors important for crypt homeostasis and altered in colorectal cancers, such as transcription factor 7-like 2 (TCF7L2; Wnt pathway), recombining binding protein suppressor of hairless (RBPJ; Notch pathway), and homeobox protein CDX2 (epithelial cell identity), modulate THRA activity. Specifically, although TCF7L2 and CDX2 stimulated THRA, RBPJ induced its repression. In-depth analysis of the Wnt-dependent increase showed direct regulation of the THRA promoter in cells and of TRα1 expression in murine enteroids. Given our previous results on the control of the Wnt pathway by TRα1, our new results unveil a complex regulatory loop and synergy between these endocrine and epithelial-cell-intrinsic signals. Our work describes, for the first time, the regulation of the THRA gene in specific cell and tumor contexts.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': '© 2022 The Authors. Molecular Oncology published by John Wiley & Sons Ltd on behalf of Federation of European Biochemical Societies.', 'doi': '10.1002/1878-0261.13298', 'publication_date': datetime.date(2022, 10, 12), 'authors': [{'lastname': 'Giolito', 'firstname': 'Maria Virginia', 'initials': 'MV', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.\\nINSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}, {'lastname': 'La Rosa', 'firstname': 'Théo', 'initials': 'T', 'affiliation': 'INSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}, {'lastname': 'Farhat', 'firstname': 'Diana', 'initials': 'D', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.\\nINSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}, {'lastname': 'Bodoirat', 'firstname': 'Serguei', 'initials': 'S', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.'}, {'lastname': 'Guardia', 'firstname': 'Gabriela D A', 'initials': 'GDA', 'affiliation': 'Centro de Oncologia Molecular, Hospital Sírio-Libanês, São Paulo, Brazil.'}, {'lastname': 'Domon-Dell', 'firstname': 'Claire', 'initials': 'C', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.'}, {'lastname': 'Galante', 'firstname': 'Pedro A F', 'initials': 'PAF', 'affiliation': 'Centro de Oncologia Molecular, Hospital Sírio-Libanês, São Paulo, Brazil.'}, {'lastname': 'Freund', 'firstname': 'Jean-Noel', 'initials': 'JN', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.'}, {'lastname': 'Plateroti', 'firstname': 'Michelina', 'initials': 'M', 'affiliation': 'Inserm, IRFAC/UMR-S1113, FMTS, Université de Strasbourg, France.\\nINSERM U1052, CNRS UMR5286, Centre de Recherche en Cancérologie de Lyon, France.'}]}\n",
      "\n",
      "{'pubmed_id': '36183778', 'title': 'Integrated analysis of microRNA expression profiles and function network in mice testes after low dose lead exposure from early puberty.', 'keywords': ['CeRNA', 'Low dose lead', 'Puberty', 'Reproductive toxicology', 'miRNA'], 'journal': 'Toxicology and applied pharmacology', 'abstract': 'There is evidence suggesting the participation of non-coding RNAs in male reproductive dysfunction induced by lead, and the significance of microRNAs has been highlighted recently because of their essential roles in gene regulatory networks. To comprehensively understand the functions of miRNA and the regulatory networks, RNA sequencing was carried out to obtain miRNA expression profiles in mice testes exposed to low dose Pb for 90\\xa0days at the onset of puberty. In total, 44 differentially expressed miRNAs with 26 up-regulated and 18 down-regulated were identified between 200\\xa0mg/L Pb group and control group (p\\xa0<\\xa00.05). Enrichment analysis confirmed that the target genes of DE miRNAs might participate in the metabolism of testicular cells. Furthermore, a miRNA-mRNA co-expression network consisting of 19 miRNAs and 106 mRNAs and a competing endogenous RNA network of lncRNA-miRNA-mRNA including 179 genes were established. Finally, the expressions of 4 miRNAs (mmu-miR-451a, mmu-miR-133a-3p, mmu-miR-1a-3p and mmu-miR-486a-3p) and 4 mRNAs (Gramd1b, Tcf7l2, Mov10 and Srcin1) involved in regulatory networks were verified by RT-qPCR. In conclusion, our research might provide targets for the mechanism studies of miRNAs in reproductive toxicity of Pb.', 'conclusions': None, 'methods': None, 'results': None, 'copyrights': 'Copyright © 2022. Published by Elsevier Inc.', 'doi': '10.1016/j.taap.2022.116260', 'publication_date': datetime.date(2022, 10, 3), 'authors': [{'lastname': 'Ma', 'firstname': 'Haitao', 'initials': 'H', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Zhang', 'firstname': 'Haoran', 'initials': 'H', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Yu', 'firstname': 'Jun', 'initials': 'J', 'affiliation': 'Department of Preventive Medicine, School of Basic Medical Sciences, Hubei University of Science and Technology, Xianning 437100, Hubei Province, China.'}, {'lastname': 'Wang', 'firstname': 'Ziqiong', 'initials': 'Z', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Zeng', 'firstname': 'Xiangchao', 'initials': 'X', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China.'}, {'lastname': 'Ye', 'firstname': 'Jingping', 'initials': 'J', 'affiliation': 'Department of Pediatrics, Renmin Hospital of Wuhan University, Wuhan 430071, Hubei Province, China. Electronic address: jpy7012@163.com.'}, {'lastname': 'Wang', 'firstname': 'Chunhong', 'initials': 'C', 'affiliation': 'Department of Preventive Medicine, School of Public Health, Wuhan University, Wuhan 430071, Hubei Province, China. Electronic address: wchunhong027@whu.edu.cn.'}]}\n",
      "\n",
      "{'pubmed_id': '36182916', 'title': 'Leveraging family history in genetic association analyses of binary traits.', 'keywords': ['Family history', 'GWAS', 'Meta-analysis'], 'journal': 'BMC genomics', 'abstract': \"Considering relatives' health history in logistic regression for case-control genome-wide association studies (CC-GWAS) may provide new information that increases accuracy and power to detect disease associated genetic variants. We conducted simulations and analyzed type 2 diabetes (T2D) data from the Framingham Heart Study (FHS) to compare two methods, liability threshold model conditional on both case-control status and family history (LT-FH) and Fam-meta, which incorporate family history into CC-GWAS.\\nIn our simulation scenario of trait with modest T2D heritability (h\\nOverall, LT-FH and Fam-meta had higher power than CC-GWAS in simulations, especially using phenotypes that were more prevalent in older age groups, and both methods detected known genetic variants with lower P-values in real data application, highlighting the benefits of including family history in genetic association studies.\", 'conclusions': None, 'methods': None, 'results': 'In our simulation scenario of trait with modest T2D heritability (h', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1186/s12864-022-08897-8\\n10.1089/cmb.2012.0024\\n10.1371/journal.pone.0051589\\n10.1016/j.ajhg.2013.02.011\\n10.1086/521580\\n10.1038/s41588-020-0613-6\\n10.1002/gepi.21785\\n10.1093/bioinformatics/btr341\\n10.1038/ng1732\\n10.1093/nar/gkv1340\\n10.1038/ng.2653\\n10.1038/s41467-018-08000-4\\n10.1161/HYPERTENSIONAHA.116.08581\\n10.1002/em.21827\\n10.1101/2020.06.28.171561\\n10.1038/s41588-018-0205-x\\n10.1371/journal.pone.0230815\\n10.1016/j.ajhg.2022.01.009\\n10.1038/s41431-021-00980-0\\n10.1016/j.ajhg.2022.03.001', 'publication_date': datetime.date(2022, 10, 2), 'authors': [{'lastname': 'Zhang', 'firstname': 'Yixin', 'initials': 'Y', 'affiliation': 'Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA. zhangyix@bu.edu.'}, {'lastname': 'Meigs', 'firstname': 'James B', 'initials': 'JB', 'affiliation': 'Division of General Internal Medicine, Massachusetts General Hospital, Boston, MA, USA.\\nDepartment of Medicine, Harvard Medical School, Boston, MA, USA.'}, {'lastname': 'Liu', 'firstname': 'Ching-Ti', 'initials': 'CT', 'affiliation': 'Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA.'}, {'lastname': 'Dupuis', 'firstname': 'Josée', 'initials': 'J', 'affiliation': 'Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA.\\nDepartment of Epidemiology, Biostatistics and Occupational Health, McGill University, Montréal, Québec, Canada.'}, {'lastname': 'Sarnowski', 'firstname': 'Chloé', 'initials': 'C', 'affiliation': 'Department of Epidemiology, Human Genetics, and Environmental Sciences, The University of Texas Health Science Center, School of Public Health, Houston, TX, USA.'}]}\n",
      "\n",
      "{'pubmed_id': '36174902', 'title': 'Genetic risk score is associated with T2DM and diabetes complications risks.', 'keywords': ['Complications', 'Diabetes', 'Gene score', 'Polymorphism'], 'journal': 'Gene', 'abstract': \"Type 2 diabetes mellitus (T2DM) is a prototypical complex disease with polygenic architecture playing an important role in determining susceptibility to develop the disease (and its complications) in subjects exposed to modifiable lifestyle factors. A current challenge is to quantify the degree of the individual's genetic risk using genetic risk scores (GRS) capturing the results of genome-wide association studies while incorporating possible ethnicity- or population-specific differences.\\nThis study included three groups of T2DM (T2DM-I, N\\xa0=\\xa01,032; T2DM-II, N\\xa0=\\xa0353; and T2DM-III, N\\xa0=\\xa0399) patients and 2,481 diabetes-free subjects. The status of the microvascular and macrovascular diabetes complications were known for the T2DM-I patients. Overall, 21 single nucleotide polymorphisms (SNPs) were analyzed, and selected subsets were used to determine the GRS (both weighted - wGRS and unweighted - uGRS) for T2DM risk predictions (6 SNPs) and for predicting the risks of complications (7 SNPs).\\nThe strongest T2DM markers (P\\xa0<\\xa00.0001) were within the genes for TCF7L2 (transcription factor 7-like 2), FTO (fat mass and obesity associated protein) and ARAP1 (ankyrin repeat and PH domain 1). The T2DM-I subjects with uGRS values greater (Odds Ratio, 95\\xa0% Confidence Interval) than six had at least twice (2.00, 1.72-2.32) the risk of T2DM development (P\\xa0<\\xa00.0001), and these results were confirmed in the independent groups (T2DM-II 1.82, 1.45-2.27; T2DM-III 2.63, 2.11-3.27). The wGRS (>0.6) further improved (P\\xa0<\\xa00.000001) the risk estimations for all three T2DM groups. The uGRS was also a significant predictor of neuropathy (P\\xa0<\\xa00.0001), nephropathy (P\\xa0<\\xa00.005) and leg ischemia (P\\xa0<\\xa00.0005).\\nIf carefully selected and specified, GRS, both weighted and unweighted, could be significant predictors of T2DM development, as well as the diabetes complications development.\", 'conclusions': None, 'methods': None, 'results': 'The strongest T2DM markers (P\\xa0<\\xa00.0001) were within the genes for TCF7L2 (transcription factor 7-like 2), FTO (fat mass and obesity associated protein) and ARAP1 (ankyrin repeat and PH domain 1). The T2DM-I subjects with uGRS values greater (Odds Ratio, 95\\xa0% Confidence Interval) than six had at least twice (2.00, 1.72-2.32) the risk of T2DM development (P\\xa0<\\xa00.0001), and these results were confirmed in the independent groups (T2DM-II 1.82, 1.45-2.27; T2DM-III 2.63, 2.11-3.27). The wGRS (>0.6) further improved (P\\xa0<\\xa00.000001) the risk estimations for all three T2DM groups. The uGRS was also a significant predictor of neuropathy (P\\xa0<\\xa00.0001), nephropathy (P\\xa0<\\xa00.005) and leg ischemia (P\\xa0<\\xa00.0005).', 'copyrights': 'Copyright © 2022. Published by Elsevier B.V.', 'doi': '10.1016/j.gene.2022.146921', 'publication_date': datetime.date(2022, 9, 30), 'authors': [{'lastname': 'Hubacek', 'firstname': 'Jaroslav A', 'initials': 'JA', 'affiliation': 'Experimental Medicine Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic; 3rd Department of Internal Medicine, 1(st) Faculty of Medicine, Charles University, Prague, Czech Republic. Electronic address: jahb@ikem.cz.'}, {'lastname': 'Dlouha', 'firstname': 'Lucie', 'initials': 'L', 'affiliation': 'Experimental Medicine Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic; Department of Anthropology and Human Genetics, Faculty of Science, Charles University, Prague, Czech Republic.'}, {'lastname': 'Adamkova', 'firstname': 'Vera', 'initials': 'V', 'affiliation': 'Department of Preventive Cardiology, Institute for Clinical and Experimental Medicine, Prague, Czech Republic; Czech Technical University of Prague, Faculty of Biomedical Engineering, Prague, Czech Republic.'}, {'lastname': 'Dlouha', 'firstname': 'Dana', 'initials': 'D', 'affiliation': 'Experimental Medicine Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}, {'lastname': 'Pacal', 'firstname': 'Lukas', 'initials': 'L', 'affiliation': 'Department of Pathophysiology, Faculty of Medicine, Masaryk University, Brno, Czech Republic.'}, {'lastname': 'Kankova', 'firstname': 'Katerina', 'initials': 'K', 'affiliation': 'Department of Pathophysiology, Faculty of Medicine, Masaryk University, Brno, Czech Republic.'}, {'lastname': 'Galuska', 'firstname': 'David', 'initials': 'D', 'affiliation': 'Department of Pathophysiology, Faculty of Medicine, Masaryk University, Brno, Czech Republic.'}, {'lastname': 'Lanska', 'firstname': 'Vera', 'initials': 'V', 'affiliation': 'Statistical Unit, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}, {'lastname': 'Veleba', 'firstname': 'Jiri', 'initials': 'J', 'affiliation': 'Diabetes Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}, {'lastname': 'Pelikanova', 'firstname': 'Terezie', 'initials': 'T', 'affiliation': 'Diabetes Centre, Institute for Clinical and Experimental Medicine, Prague, Czech Republic.'}]}\n",
      "\n",
      "{'pubmed_id': '36163520', 'title': 'Development and validation of a model that predicts the risk of diabetic retinopathy in type 2 diabetes mellitus patients.', 'keywords': ['Diabetic retinopathy', 'Nomogram', 'Prediction model', 'Risk factors', 'Type 2 diabetes mellitus'], 'journal': 'Acta diabetologica', 'abstract': 'Diabetic retinopathy is the leading cause of blindness in people with type 2 diabetes. To enable primary care physicians to identify high-risk type 2 diabetic patients with diabetic retinopathy at an early stage, we developed a nomogram model to predict the risk of developing diabetic retinopathy in the Xinjiang type 2 diabetic population.\\nIn a retrospective study, we collected data on 834 patients with type 2 diabetes through an electronic medical record system. Stepwise regression was used to filter variables. Logistic regression was applied to build a nomogram prediction model and further validated in the training set. The c-index, forest plot, calibration plot, and clinical decision curve analysis were used to comprehensively validate the model and evaluate its accuracy and clinical validity.\\nFour predictors were selected to establish the final model: hypertension, blood urea nitrogen, duration of diabetes, and diabetic peripheral neuropathy. The model displayed medium predictive power with a C-index of 0.781(95%CI:0.741-0.822) in the training set and 0.865(95%CI:0.807-0.923)in the validation set. The calibration curve of the DR probability shows that the predicted results of the nomogram are in good agreement with the actual results. Decision curve analysis demonstrated that the novel nomogram was clinically valuable.\\nThe nomogram of the risk of developing diabetic nephropathy contains 4 characteristics. that can help primary care physicians quickly identify individuals at high risk of developing DR in patients with type 2 diabetes, to intervene as soon as possible.', 'conclusions': None, 'methods': None, 'results': 'Four predictors were selected to establish the final model: hypertension, blood urea nitrogen, duration of diabetes, and diabetic peripheral neuropathy. The model displayed medium predictive power with a C-index of 0.781(95%CI:0.741-0.822) in the training set and 0.865(95%CI:0.807-0.923)in the validation set. The calibration curve of the DR probability shows that the predicted results of the nomogram are in good agreement with the actual results. Decision curve analysis demonstrated that the novel nomogram was clinically valuable.', 'copyrights': '© 2022. Springer-Verlag Italia S.r.l., part of Springer Nature.', 'doi': '10.1007/s00592-022-01973-1\\n10.1016/j.dsx.2021.102239\\n10.3390/ijerph18147346\\n10.3346/jkms.2016.31.S2.S146\\n10.2337/dc11-1909\\n10.1001/archopht.122.4.552\\n10.1002/dmrr.1307\\n10.1097/IAE.0b013e31821c4252\\n10.1007/s00592-020-01540-6\\n10.1001/archopht.126.12.1740\\n10.1016/j.ophtha.2010.08.003\\n10.1136/bjophthalmol-2017-310316\\n10.1002/(SICI)1096-9136(199807)15:7<539::AID-DIA668>3.0.CO;2-S\\n10.1186/s12882-017-0776-2\\n10.1016/j.ebiom.2018.08.009\\n10.1097/MD.0000000000019236\\n10.1016/j.jdiacomp.2013.12.008\\n10.1016/j.ophtha.2006.10.023\\n10.1136/bmjopen-2018-023586\\n10.4103/ijo.IJO_1144_21\\n10.1371/journal.pone.0097928\\n10.1016/j.jdiacomp.2013.12.008\\n10.1016/j.atherosclerosis.2012.10.034\\n10.1002/mnfr.201900934\\n10.1016/j.gtc.2016.02.004\\n10.4103/ijo.IJO_1279_21\\n10.1507/endocrj.EJ20-0794\\n10.1172/JCI86181\\n10.2147/DMSO.S244061', 'publication_date': datetime.date(2022, 9, 28), 'authors': [{'lastname': 'Yang', 'firstname': 'Jing', 'initials': 'J', 'affiliation': 'State Key Laboratory of Pathogenesis, Prevention andTreatment of High Incidence Diseases in Central Asia, Department of Endocrinology, The First Affiliated Hospital of Xinjiang Medical University, Urumqi, 830017, China.'}, {'lastname': 'Jiang', 'firstname': 'Sheng', 'initials': 'S', 'affiliation': 'State Key Laboratory of Pathogenesis, Prevention andTreatment of High Incidence Diseases in Central Asia, Department of Endocrinology, The First Affiliated Hospital of Xinjiang Medical University, Urumqi, 830017, China. xjjiangsheng@126.com.'}]}\n",
      "\n",
      "{'pubmed_id': '36155628', 'title': 'Effect of TCF7L2 on the relationship between lifestyle factors and glycemic parameters: a systematic review.', 'keywords': ['Dietary factors', 'Glycemic parameters', 'Lifestyle factors', 'TCF7L2'], 'journal': 'Nutrition journal', 'abstract': 'Among candidate\\xa0genes related to type 2 diabetes (T2DM), one of the strongest genes is Transcription factor 7 like 2 (TCF7L2), regarding the Genome-Wide Association Studies. We aimed to conduct a systematic review of the literature on the modification effect of TCF7L2 on the relation between glycemic parameters and lifestyle factors.\\nA systematic literature search was done for relevant publications using electronic databases, including PubMed, EMBASE, Scopus, and Web of Science, from January 1, 2000, to November 2, 2021.\\nThirty-eight studies (16 observational studies, six meal test trials, and 16 randomized controlled trials (RCTs)) were included. Most observational studies had been conducted on participants with non-diabetes showing that TCF7L2 modified the association between diet (fatty acids and fiber) and insulin resistance. In addition, findings from meal test trials showed that, compared to non-risk-allele carriers, consumption of meals with different percentages of total dietary fat in healthy risk-allele carriers increased glucose concentrations and impaired insulin sensitivity. However, ten RCTs, with intervention periods of less than ten weeks and more than one year, showed that TCF7L2 did not modify glycemic parameters in response to a dietary intervention involving different macronutrients. However, two weight loss dietary RCTs with more than 1-year duration showed that serum glucose and insulin levels decreased and insulin resistance improved in non-risk allele subjects with overweight/obesity. Regarding artichoke extract supplementation (ALE), two RCTs observed that ALE supplementation significantly decreased insulin concentration and improved insulin resistance in the TT genotype of the rs7903146 variant of TCF7L2. In addition, four studies suggested that physical activity levels and smoking status modified the association between TCF7L2 and glycemic parameters. However, three studies observed no effect of TCF7L2 on glycemic parameters in participants with different levels of physical activity and smoking status.\\nThe modification effects of TCF7L2 on the relation between the lifestyle factors (diet, physical activity, and smoking status) and glycemic parameters were contradictory.\\nCRD42020196327.', 'conclusions': 'The modification effects of TCF7L2 on the relation between the lifestyle factors (diet, physical activity, and smoking status) and glycemic parameters were contradictory.', 'methods': None, 'results': 'Thirty-eight studies (16 observational studies, six meal test trials, and 16 randomized controlled trials (RCTs)) were included. Most observational studies had been conducted on participants with non-diabetes showing that TCF7L2 modified the association between diet (fatty acids and fiber) and insulin resistance. In addition, findings from meal test trials showed that, compared to non-risk-allele carriers, consumption of meals with different percentages of total dietary fat in healthy risk-allele carriers increased glucose concentrations and impaired insulin sensitivity. However, ten RCTs, with intervention periods of less than ten weeks and more than one year, showed that TCF7L2 did not modify glycemic parameters in response to a dietary intervention involving different macronutrients. However, two weight loss dietary RCTs with more than 1-year duration showed that serum glucose and insulin levels decreased and insulin resistance improved in non-risk allele subjects with overweight/obesity. Regarding artichoke extract supplementation (ALE), two RCTs observed that ALE supplementation significantly decreased insulin concentration and improved insulin resistance in the TT genotype of the rs7903146 variant of TCF7L2. In addition, four studies suggested that physical activity levels and smoking status modified the association between TCF7L2 and glycemic parameters. However, three studies observed no effect of TCF7L2 on glycemic parameters in participants with different levels of physical activity and smoking status.', 'copyrights': '© 2022. The Author(s).', 'doi': '10.1186/s12937-022-00813-w\\n10.1016/j.diabres.2019.107843\\n10.1177/2047487319878371\\n10.1016/j.cmet.2008.08.006\\n10.1038/ng1732\\n10.1038/79216\\n10.1038/nature05616\\n10.1038/ng.120\\n10.2337/dc17-0290\\n10.1007/s11010-021-04086-5\\n10.1080/07435800.2019.1573827\\n10.2337/db09-0918\\n10.2337/db09-1169\\n10.1016/j.numecd.2006.04.001\\n10.1111/obr.12921\\n10.1016/j.gde.2018.02.001\\n10.1007/s11892-016-0758-y\\n10.1055/s-0031-1285832\\n10.1007/s00109-006-0134-5\\n10.1146/annurev-nutr-071811-150648\\n10.2337/dc12-2211\\n10.3390/ijms18061188\\n10.3945/ajcn.116.150094\\n10.3389/fgene.2020.590369\\n10.1097/CM9.0000000000000617\\n10.1038/ijo.2008.146\\n10.3945/ajcn.112.038125\\n10.3945/ajcn.2009.27947\\n10.1371/journal.pone.0043390\\n10.2337/db11-0973\\n10.1371/journal.pone.0230815\\n10.1038/oby.2007.77\\n10.1056/NEJMoa062418\\n10.1371/journal.pone.0021518\\n10.1136/bmj.n71\\n10.1136/bmj.l4898\\n10.1007/s00125-010-1940-4\\n10.1210/jc.2015-1172\\n10.1186/s13098-018-0338-1\\n10.1007/s00125-009-1307-x\\n10.1016/j.diabet.2017.05.001\\n10.1017/S0007114516002798\\n10.3945/jn.111.147447\\n10.1002/ptr.5951\\n10.3945/ajcn.2009.28379\\n10.2337/db09-1050\\n10.1007/s12263-012-0284-8\\n10.1016/j.joim.2018.05.006\\n10.1007/s00125-009-1630-2\\n10.2337/dc10-1150\\n10.1016/j.atherosclerosis.2010.10.027\\n10.1007/s00125-012-2634-x\\n10.2337/dc13-0955\\n10.1186/s12863-014-0134-3\\n10.1016/j.metabol.2016.11.018\\n10.1371/journal.pone.0188382\\n10.3390/nu13061936\\n10.1016/j.jnutbio.2010.11.020\\n10.2337/db08-1623\\n10.2337/db20-0156\\n10.1002/gepi.21991\\n10.1016/j.trsl.2021.07.001\\n10.1038/s41467-020-20500-w\\n10.1172/JCI89444\\n10.1038/nrendo.2014.175', 'publication_date': datetime.date(2022, 9, 27), 'authors': [{'lastname': 'Hosseinpour-Niazi', 'firstname': 'Somayeh', 'initials': 'S', 'affiliation': 'Nutrition and Endocrine Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Mirmiran', 'firstname': 'Parvin', 'initials': 'P', 'affiliation': 'Nutrition and Endocrine Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran. mirmiran@endocrine.ac.ir.'}, {'lastname': 'Hosseini', 'firstname': 'Shabnam', 'initials': 'S', 'affiliation': 'School of Human Nutrition, Faculty of Agricultural and Environmental Sciences, McGill University, Montreal, Quebec, Canada.'}, {'lastname': 'Hadaegh', 'firstname': 'Farzad', 'initials': 'F', 'affiliation': 'Prevention of Metabolic Disorders Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Ainy', 'firstname': 'Elaheh', 'initials': 'E', 'affiliation': 'Department of Vice Chancellor Research Affairs, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Daneshpour', 'firstname': 'Maryam S', 'initials': 'MS', 'affiliation': 'Cellular and Molecular Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}, {'lastname': 'Azizi', 'firstname': 'Fereidoun', 'initials': 'F', 'affiliation': 'Endocrine Research Center, Research Institute for Endocrine Sciences, Shahid Beheshti University of Medical Sciences, Tehran, Iran.'}]}\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Generate list of dict records which will hold all article details that could be fetch from PUBMED API\n",
    "for article in articleList:\n",
    "#Sometimes article['pubmed_id'] contains list separated with comma - take first pubmedId in that list - thats article pubmedId\n",
    "    pubmedId = article['pubmed_id'].partition('\\n')[0]\n",
    "    # Append article info to dictionary \n",
    "    articleInfo.append({u'pubmed_id':pubmedId,\n",
    "                       u'title':article['title'],\n",
    "                       u'keywords':article['keywords'],\n",
    "                       u'journal':article['journal'],\n",
    "                       u'abstract':article['abstract'],\n",
    "                       u'conclusions':article['conclusions'],\n",
    "                       u'methods':article['methods'],\n",
    "                       u'results': article['results'],\n",
    "                       u'copyrights':article['copyrights'],\n",
    "                       u'doi':article['doi'],\n",
    "                       u'publication_date':article['publication_date'], \n",
    "                       u'authors':article['authors']})\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  pubmed_id                                              title  \\\n",
      "0  36479247  Genomic profiling and network-level understand...   \n",
      "1  36454469                      Genomic Editing and Diabetes.   \n",
      "2  36399471  Bioinformatic analysis and clinical diagnostic...   \n",
      "3  36382598  Identification of odontogenic ameloblast assoc...   \n",
      "4  36382190  Advances in multi-omics study of biomarkers of...   \n",
      "5  36369331  Association between TCF7L2 polymorphism and ty...   \n",
      "6  36364957  Indolepropionic Acid, a Gut Bacteria-Produced ...   \n",
      "7  36360783  Decoding Diabetes Biomarkers and Related Molec...   \n",
      "8  36358989  FGF9 Promotes Expression of HAS2 in Palatal El...   \n",
      "9  36344976  CHRNB2 represses pancreatic cancer migration a...   \n",
      "\n",
      "                                            keywords  \\\n",
      "0  [HCV and HCC, biomarkers, co-expression, gene ...   \n",
      "1  [CRISPR/Cas9, Diabetes, Genetic editing, Long ...   \n",
      "2                                                 []   \n",
      "3  [ODAM, Wnt signaling, cell cycle, colorectal c...   \n",
      "4  [1,5-AG, 1,5-anhydroglucitol, 2-AAA, 2-aminoad...   \n",
      "5  [Bangladesh, Chittagong, Comorbidities, Diabet...   \n",
      "6  [NAFLD, diet, gut microbiota, indolepropionic ...   \n",
      "7  [bioinformatics, diabetes, gene expression, pr...   \n",
      "8  [FGF9, HAS2, TCF7L2, Wnt/β-catenin pathway, hy...   \n",
      "9  [CHRNB2, Lymph node ratio, Pancreatic cancer, ...   \n",
      "\n",
      "                                             journal  \\\n",
      "0                              Frontiers in genetics   \n",
      "1      Advances in experimental medicine and biology   \n",
      "2                                           PloS one   \n",
      "3                                     Cancer science   \n",
      "4  Computational and structural biotechnology jou...   \n",
      "5                          Molecular biology reports   \n",
      "6                                          Nutrients   \n",
      "7  International journal of environmental researc...   \n",
      "8                                       Biomolecules   \n",
      "9                          Cancer cell international   \n",
      "\n",
      "                                            abstract  \\\n",
      "0  Data integration with phenotypes such as gene ...   \n",
      "1  The diabetes types and its complications have ...   \n",
      "2  This study investigates the expression and eff...   \n",
      "3  The Wnt/β-catenin signaling pathway plays a ke...   \n",
      "4  Glycolipid metabolism disorder are major threa...   \n",
      "5  Diabetes is a severe health burden for Banglad...   \n",
      "6  An intricate relationship between gut microbio...   \n",
      "7  The molecular basis of diabetes mellitus is ye...   \n",
      "8  Using Ddx4-Cre mice, we generated the \\nFGF9 p...   \n",
      "9  Pancreatic cancer is one of the most lethal di...   \n",
      "\n",
      "                                         conclusions methods  \\\n",
      "0                                               None    None   \n",
      "1                                               None    None   \n",
      "2                                               None    None   \n",
      "3                                               None    None   \n",
      "4                                               None    None   \n",
      "5  This study postulates that TCF7L2 genetic poly...    None   \n",
      "6                                               None    None   \n",
      "7                                               None    None   \n",
      "8                                               None    None   \n",
      "9                                               None    None   \n",
      "\n",
      "                                             results  \\\n",
      "0                                               None   \n",
      "1                                               None   \n",
      "2                                               None   \n",
      "3                                               None   \n",
      "4                                               None   \n",
      "5  A strong association of T2DM with polymorphism...   \n",
      "6                                               None   \n",
      "7                                               None   \n",
      "8                                                      \n",
      "9  High LNR was significantly related to high T s...   \n",
      "\n",
      "                                          copyrights  \\\n",
      "0  Copyright © 2022 El-Kafrawy, El-Daly, Bajrai, ...   \n",
      "1  © 2023. The Author(s), under exclusive license...   \n",
      "2  Copyright: © 2022 Zheng et al. This is an open...   \n",
      "3  © 2022 The Authors. Cancer Science published b...   \n",
      "4  © 2022 The Authors. Published by Elsevier B.V....   \n",
      "5  © 2022. The Author(s), under exclusive licence...   \n",
      "6                                               None   \n",
      "7                                               None   \n",
      "8                                               None   \n",
      "9                             © 2022. The Author(s).   \n",
      "\n",
      "                                                 doi publication_date  \\\n",
      "0  10.3389/fgene.2022.880440\\n10.1016/j.ajpath.20...       2022-12-09   \n",
      "1  10.1007/978-981-19-5642-3_14\\n10.1016/S0140-67...       2022-12-02   \n",
      "2  10.1371/journal.pone.0277832\\n10.1016/S0140-67...       2022-11-19   \n",
      "3                                  10.1111/cas.15657       2022-11-17   \n",
      "4  10.1016/j.csbj.2022.10.030\\n10.1038/nrendo.201...       2022-11-17   \n",
      "5  10.1007/s11033-022-08081-x\\n10.1016/j.metabol....       2022-11-13   \n",
      "6  10.3390/nu14214695\\n10.1042/bj0170907\\n10.1007...       2022-11-12   \n",
      "7  10.3390/ijerph192113890\\n10.1038/414782a\\n10.2...       2022-11-12   \n",
      "8  10.3390/biom12111639\\n10.3389/fphys.2016.00067...       2022-11-12   \n",
      "9  10.1186/s12935-022-02768-8\\n10.3390/ijms230312...       2022-11-08   \n",
      "\n",
      "                                             authors  \n",
      "0  [{'lastname': 'El-Kafrawy', 'firstname': 'Sher...  \n",
      "1  [{'lastname': 'Shah', 'firstname': 'Parth', 'i...  \n",
      "2  [{'lastname': 'Zheng', 'firstname': 'Jiqing', ...  \n",
      "3  [{'lastname': 'Yamaguchi', 'firstname': 'Kiyos...  \n",
      "4  [{'lastname': 'Fang', 'firstname': 'Xinyi', 'i...  \n",
      "5  [{'lastname': 'Salauddin', 'firstname': 'Asma'...  \n",
      "6  [{'lastname': 'Sehgal', 'firstname': 'Ratika',...  \n",
      "7  [{'lastname': 'Elsherbini', 'firstname': 'Amir...  \n",
      "8  [{'lastname': 'Sun', 'firstname': 'Yidan', 'in...  \n",
      "9  [{'lastname': 'Qin', 'firstname': 'Cheng', 'in...  \n"
     ]
    }
   ],
   "source": [
    "# Generate Pandas DataFrame from list of dictionaries\n",
    "import pandas as pd\n",
    "\n",
    "articlesPD = pd.DataFrame.from_dict(articleInfo)\n",
    "# export_csv = df.to_csv (r'C:\\Users\\YourUsernam\\Desktop\\export_dataframe.csv', index = None, header=True) \n",
    "\n",
    "#Print first 10 rows of dataframe\n",
    "print(articlesPD.head(10))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)]"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "68244b10fadec1b3e625e9157df0e88cca746a2e21e581f39bebc9f0b519388c"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
